

<!DOCTYPE html>
<!--[if IE 8]><html class="no-js lt-ie9" lang="en" > <![endif]-->
<!--[if gt IE 8]><!--> <html class="no-js" lang="en" > <!--<![endif]-->
<head>
  <meta charset="utf-8">
  <meta name="generator" content="Docutils 0.17.1: http://docutils.sourceforge.net/" />
<meta content="Jian Cheng's publications, Publications By Year" name="description" />
<meta content="jian, cheng, Jian Cheng, hompage, manifold, information geometry, information metric, Riemannian framework, medical imaging, medical image analysis, QBI, ODF, Q-ball imaging, orientation distribution function, fiber tracking, Spherical Polar Fourier Imaging, diffusion magnetic resonance imaging, diffusion mri, mri, diffusion tensor imaging, diffusion tensor, DTI, high angular resolution diffusion imaging, HARDI, publications, inria, unc, nih, computer vision, image processing" name="keywords" />

  <meta name="viewport" content="width=device-width, initial-scale=1.0">
  
  <title>Publications By Year &mdash; Jian Cheng</title>
  

  
  

  

  
  
    

  

  
  
    <link rel="stylesheet" href="_static/css/theme.css" type="text/css" />
  

  <link rel="stylesheet" href="_static/academicons.css">

  
    <link rel="stylesheet" href="_static/pygments.css" type="text/css" />
  
    <link rel="stylesheet" href="_static/css/theme.css" type="text/css" />
  
    <link rel="stylesheet" href="_static/theme_overrides.css" type="text/css" />
  

  
        <link rel="search" title="Search" href="search.html"/>
    <link rel="top" title="Jian Cheng" href="index.html"/>
        <link rel="up" title="Publications" href="publications.html"/>
        <link rel="next" title="Publications By Selected Venues" href="publications_by_venue.html"/>
        <link rel="prev" title="Publications By Type" href="publications_by_type.html"/> 

  
  <script src="_static/js/modernizr.min.js"></script>

</head>

<body class="wy-body-for-nav" role="document">

   
  <div class="wy-grid-for-nav">

    
    <nav data-toggle="wy-nav-shift" class="wy-nav-side">
      <div class="wy-side-scroll">
        <div class="wy-side-nav-search">
          

        
          <a href="index.html" class="icon icon-home"> Jian Cheng (程健)
          <img alt="Jian Cheng" src="_images/JianCheng3.png" style="width: 156.5px; height: 156.5px;"  class="icon"/>
        

        
        </a>

          
            
            
          

          
<div role="search">
  <form id="rtd-search-form" class="wy-form" action="search.html" method="get">
    <input type="text" name="q" placeholder="Search" />
    <input type="hidden" name="check_keywords" value="yes" />
    <input type="hidden" name="area" value="default" />
  </form>
</div>

          
        </div>

        <div class="wy-menu wy-menu-vertical" data-spy="affix" role="navigation" aria-label="main navigation">
          
            
            
                <ul class="current">
<li class="toctree-l1"><a class="reference internal" href="index.html">Home</a></li>
<li class="toctree-l1"><a class="reference internal" href="resume.html">Resume</a><ul>
<li class="toctree-l2"><a class="reference internal" href="resume.html#education">Education</a></li>
<li class="toctree-l2"><a class="reference internal" href="resume.html#professional-experience">Professional Experience</a></li>
<li class="toctree-l2"><a class="reference internal" href="resume.html#honors-and-awards">Honors and Awards</a></li>
<li class="toctree-l2"><a class="reference internal" href="resume.html#professional-activities">Professional Activities</a></li>
</ul>
</li>
<li class="toctree-l1 current"><a class="reference internal" href="publications.html">Publications</a><ul class="current">
<li class="toctree-l2"><a class="reference internal" href="publications_by_type.html"> Publications By Type</a><ul>
<li class="toctree-l3"><a class="reference internal" href="publications_by_type.html#book-chapters">Book Chapters</a></li>
<li class="toctree-l3"><a class="reference internal" href="publications_by_type.html#journal-articles">Journal Articles</a></li>
<li class="toctree-l3"><a class="reference internal" href="publications_by_type.html#conference-articles">Conference Articles</a></li>
<li class="toctree-l3"><a class="reference internal" href="publications_by_type.html#conference-abstracts">Conference Abstracts</a></li>
<li class="toctree-l3"><a class="reference internal" href="publications_by_type.html#theses">Theses</a></li>
</ul>
</li>
<li class="toctree-l2 current"><a class="current reference internal" href="#"> Publications By Year</a></li>
<li class="toctree-l2"><a class="reference internal" href="publications_by_venue.html"> Publications By Selected Venues</a></li>
<li class="toctree-l2"><a class="reference internal" href="publications_first_corresponding_author.html"> First- or Corresponding-Author Articles</a><ul>
<li class="toctree-l3"><a class="reference internal" href="publications_first_corresponding_author.html#book-chapters">Book Chapters</a></li>
<li class="toctree-l3"><a class="reference internal" href="publications_first_corresponding_author.html#journal-articles">Journal Articles</a></li>
<li class="toctree-l3"><a class="reference internal" href="publications_first_corresponding_author.html#conference-articles">Conference Articles</a></li>
<li class="toctree-l3"><a class="reference internal" href="publications_first_corresponding_author.html#conference-abstracts">Conference Abstracts</a></li>
<li class="toctree-l3"><a class="reference internal" href="publications_first_corresponding_author.html#theses">Theses</a></li>
</ul>
</li>
</ul>
</li>
<li class="toctree-l1"><a class="reference internal" href="software.html">Software</a><ul>
<li class="toctree-l2"><a class="reference internal" href="software.html#diffusion-mri-tool">Diffusion MRI Tool</a></li>
<li class="toctree-l2"><a class="reference internal" href="software.html#brain-age-estimation">Brain Age Estimation</a></li>
<li class="toctree-l2"><a class="reference internal" href="software.html#lrtv-for-image-superresolution">LRTV for Image Superresolution</a></li>
<li class="toctree-l2"><a class="reference internal" href="software.html#bibtex2html-py">bibtex2html.py</a></li>
</ul>
</li>
<li class="toctree-l1"><a class="reference external" href="http://shi.buaa.edu.cn/jiancheng">Chinese Homepage &#64; BUAA</a></li>
</ul>

            
          
        </div>

        &nbsp;

      <div class="mybotom">
      <!--     <a href="mailto:jian&#46;cheng&#37;&#52;&#48;nih&#46;gov"><font color="grey">[jian<span>&#46;</span>cheng [AT] nih<span>&#46;</span>gov]</font></a> <br /> -->
      <!--     <a href="https://scholar.google.com/citations?user=BARqXQ0AAAAJ&hl=en"><font  color="grey">[Google Scholar]</font></a> <br \> -->
      <!--     <a href="https://github.com/JianCheng"><font color="grey">[GitHub]</font></a> <br \>  -->
      <!--     <a href="https://www.linkedin.com/in/jian-cheng-870b0822"><font  color="grey">[LinkedIn]</font></a> <br \> -->
                <p class="social">
                <a href="https://www.linkedin.com/in/jian-cheng-870b0822">
                    <i class="fa fa-linkedin fa-2x"></i>
                </a>
                <a href="https://github.com/jiancheng">
                    <i class="fa fa-github fa-2x"></i>
                </a>
                <a href="https://scholar.google.com/citations?user=BARqXQ0AAAAJ">
                    <i class="ai ai-google-scholar-square ai-2x"></i>
                </a>
                <a href="https://www.researchgate.net/profile/Jian_Cheng2">
                    <i class="ai ai-researchgate ai-2x"></i>
                </a>
                <a href="https://arxiv.org/a/cheng_j_2">
                    <i class="ai ai-arxiv ai-2x"></i>
                </a>
                <a href="https://dblp.org/pid/14/6145-2.html">
                    <i class="ai ai-dblp ai-2x"></i>
                </a>
                <!-- <a href="mailto:jiancheng@ieee.org"> -->
                <!--     <i class="fa fa-envelope-o fa-2x"></i> -->
                <!-- </a> -->
                </p>
      </div>
        &nbsp;

    </nav>

    <section data-toggle="wy-nav-shift" class="wy-nav-content-wrap">

      
      <nav class="wy-nav-top" role="navigation" aria-label="top navigation">
        
          <i data-toggle="wy-nav-top" class="fa fa-bars"></i>
          <a href="index.html">Jian Cheng (程健)</a>
        
      </nav>


      
      <div class="wy-nav-content">
        <div class="rst-content">
          















<div role="navigation" aria-label="breadcrumbs navigation">

  <ul class="wy-breadcrumbs">
    
      <li><a href="index.html">Home</a> &raquo;</li>
        
          <li><a href="publications.html">Publications</a> &raquo;</li>
        
      <li>Publications By Year</li>
    
    
      <li class="wy-breadcrumbs-aside">
        
            
        
      </li>
    
  </ul>

  
  <hr/>
</div>
          <div role="main" class="document" itemscope="itemscope" itemtype="https://schema.org/Article">
           <div itemprop="articleBody">
            
  <section id="publications-by-year">
<h1>Publications By Year<a class="headerlink" href="#publications-by-year" title="Permalink to this headline">¶</a></h1>
<style type="text/css">

.wy-nav-content {
    max-width: 2050px
}
</style><a class="reference external" href="https://scholar.google.com/citations?user=BARqXQ0AAAAJ&amp;hl=en">
<img alt="Google Scholar" src="_static/google_scholar.png" style="width: 155.2px; height: 32.0px;" />
</a>
&nbsp;

  <div>

     <ul style="padding: 10px; border: 1px solid #333333; border-radius: 10px; text-align: justify; background-color: #F7F7F7;">

     <li> COPYRIGHT NOTICE: These materials are presented to ensure timely dissemination of scholarly and technical work. Copyright and all rights therein are retained by authors or by other copyright holders.

     <li> The publication list is automatically generated from bibtex by using <a href="https://github.com/JianCheng/bibtex2html.py">bibtex2html.py</a> that I developed.

     <li> # denotes co-first authors. * denotes corresponding authors.

     </ul>

   </div><script type="text/javascript" src="https://code.jquery.com/jquery-2.2.0.min.js"></script>
<link rel="stylesheet" href="_static/mybootstrap2.css">
<script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/js/bootstrap.min.js"></script>
<script type="text/javascript" src="scholar.js"></script>

<script type="text/javascript">
    function toggle(showHideDiv, switchTextDiv)
    {
    var ele = document.getElementById(showHideDiv);
    var text = document.getElementById(switchTextDiv);
    if(ele.style.display == "block")
    {
        ele.style.display = "none";
    }
    else
    {
        ele.style.display = "block";
    }
    }
</script>

<link rel="stylesheet" type="text/css" href="_static/jcpaperstyle.css">
<style type="text/css">
</style>





    <br>
    <p><big>&#8226;&nbsp;<b>Total Citations</b>: <a target="_blank" href='https://scholar.google.com/citations?user=BARqXQ0AAAAJ&hl=en'>2032</a> &#8226;&nbsp;  <b>H-Index</b>: <a target="_blank" href='https://scholar.google.com/citations?user=BARqXQ0AAAAJ&hl=en'>23</a></big></p>

<p><big>&#8226;&nbsp;<b>TMI</b> (5) &#8226;&nbsp;<b>MedIA</b> (2) &#8226;&nbsp;<b>eLife</b> (1) &#8226;&nbsp;<b>NeuroImage</b> (2) &#8226;&nbsp;<b>HBM</b> (5) &#8226;&nbsp;<b>CC</b> (1) &#8226;&nbsp;<b>CVPR</b> (1) &#8226;&nbsp;<b>MICCAI</b> (15) &#8226;&nbsp;<b>IPMI</b> (2) &#8226;&nbsp;</big></p>

<p><big>&#8226;&nbsp;<span style="font-size: 20px;"><a href="publications_by_year.html#year2025"><b>2025</b></a></span> &#8226;&nbsp;<span style="font-size: 20px;"><a href="publications_by_year.html#year2024"><b>2024</b></a></span> &#8226;&nbsp;<span style="font-size: 20px;"><a href="publications_by_year.html#year2023"><b>2023</b></a></span> &#8226;&nbsp;<span style="font-size: 20px;"><a href="publications_by_year.html#year2022"><b>2022</b></a></span> &#8226;&nbsp;<span style="font-size: 20px;"><a href="publications_by_year.html#year2021"><b>2021</b></a></span> &#8226;&nbsp;<span style="font-size: 20px;"><a href="publications_by_year.html#year2020"><b>2020</b></a></span> &#8226;&nbsp;<span style="font-size: 20px;"><a href="publications_by_year.html#year2018"><b>2018</b></a></span> &#8226;&nbsp;<span style="font-size: 20px;"><a href="publications_by_year.html#year2017"><b>2017</b></a></span> &#8226;&nbsp;<span style="font-size: 20px;"><a href="publications_by_year.html#year2016"><b>2016</b></a></span> &#8226;&nbsp;<span style="font-size: 20px;"><a href="publications_by_year.html#year2015"><b>2015</b></a></span> &#8226;&nbsp;<span style="font-size: 20px;"><a href="publications_by_year.html#year2014"><b>2014</b></a></span> &#8226;&nbsp;<span style="font-size: 20px;"><a href="publications_by_year.html#year2013"><b>2013</b></a></span> &#8226;&nbsp;<span style="font-size: 20px;"><a href="publications_by_year.html#year2012"><b>2012</b></a></span> &#8226;&nbsp;<span style="font-size: 20px;"><a href="publications_by_year.html#year2011"><b>2011</b></a></span> &#8226;&nbsp;<span style="font-size: 20px;"><a href="publications_by_year.html#year2010"><b>2010</b></a></span> &#8226;&nbsp;<span style="font-size: 20px;"><a href="publications_by_year.html#year2009"><b>2009</b></a></span> &#8226;&nbsp;<span style="font-size: 20px;"><a href="publications_by_year.html#year2008"><b>2008</b></a></span> &#8226;&nbsp;</big></p><h2><a name="year2025"></a>2025</h2>

<ol reversed>

<li>
<span class="author">Xiaolin Fan, Yan Wang, Yingying Zhang*, Mingkun Bao, Bosen Jia, Dong Lu, Yifan Gu, <b>Jian Cheng</b>*, Haogang Zhu*</span>,<br>
<span class="title">"AVP-AP: Self-supervised Automatic View Positioning in 3D cardiac CT via Atlas Prompting"</span>,<br>
<span class="publisher">IEEE Transactions on Medical Imaging (<b>TMI</b>)</span>, vol. 44, no. 7, pp. 2921&ndash;2932, <span class="year">2025</span>.
<br><a target="_self" href="https://arxiv.org/pdf/2504.05966.pdf"><img src="Icons/pdf.gif" alt="[pdf]" style="width: 16px; height: 16px;"></a>&nbsp;
<a target="_self" href="https://dx.doi.org/10.1109/TMI.2025.3554785"><img src="Icons/www.gif" alt="[www]" style="width: 16px; height: 16px;"></a>&nbsp;
[<a type="button" data-toggle="modal" data-target="#bib-fan-AVP-AP-TMI2025">bibtex</a>]&nbsp;
[<a type="button" data-toggle="modal" data-target="#abs-fan-AVP-AP-TMI2025">abstract</a>]&nbsp;
[<a target="_self" href="https://arxiv.org/abs/2504.05966">arxiv</a>]&nbsp;

<div class="modal fade" id="bib-fan-AVP-AP-TMI2025" role="dialog"><div class="modal-dialog"><div class="modal-content"><div class="modal-header"><button type="button" class="close" data-dismiss="modal">&times;</button><h4 class="modal-title">Bibtex</h4></div><div class="modal-body">
<pre>@article{fan:AVP-AP:TMI2025,
 author = {Xiaolin Fan and Yan Wang and Yingying Zhang and Mingkun Bao and Bosen Jia and Dong Lu and Yifan Gu and Jian Cheng and Haogang Zhu},
 doi = {10.1109/TMI.2025.3554785},
 journal = {IEEE Transactions on Medical Imaging},
 number = {7},
 pages = {2921-2932},
 pdf = {https://arxiv.org/pdf/2504.05966.pdf},
 publisher = {IEEE},
 title = {AVP-AP: Self-supervised Automatic View Positioning in 3D cardiac CT via Atlas Prompting},
 url = {https://dx.doi.org/10.1109/TMI.2025.3554785},
 volume = {44},
 year = {2025}
}</pre> </div><div class="modal-footer"><button type="button" class="btn btn-default" data-dismiss="modal">Close</button></div></div></div></div>
<div class="modal fade" id="abs-fan-AVP-AP-TMI2025" role="dialog"><div class="modal-dialog"><div class="modal-content"><div class="modal-header"><button type="button" class="close" data-dismiss="modal">&times;</button><h4 class="modal-title">Abstract</h4></div><div class="modal-body">
<pre>Automatic view positioning is crucial for cardiac computed
tomography (CT) examinations, including disease diagnosis and
surgical planning. However, it is highly challenging due to
individual variability and large 3D search space. Existing work
needs labor-intensive and time-consuming manual annotations to train
view-specific models, which are limited to predicting only a fixed
set of planes. However, in real clinical scenarios, the challenge of
positioning semantic 2D slices with any orientation into varying
coordinate space in arbitrary 3D volume remains unsolved. We thus
introduce a novel framework, AVP-AP, the first to use Atlas
Prompting for self-supervised Automatic View Positioning in the 3D
CT volume. Specifically, this paper first proposes an atlas
prompting method, which generates a 3D canonical atlas and trains a
network to map slices into their corresponding positions in the
atlas space via a self-supervised manner. Then, guided by atlas
prompts corresponding to the given query images in a reference CT,
we identify the coarse positions of slices in the target CT volume
using rigid transformation between the 3D atlas and target CT
volume, effectively reducing the search space. Finally, we refine
the coarse positions by maximizing the similarity between the
predicted slices and the query images in the feature space of a
given foundation model. Our framework is flexible and efficient
compared to other methods, outperforming other methods by 19.8%
average structural similarity (SSIM) in arbitrary view positioning
and achieving 9% SSIM in two-chamber view compared to four
radiologists. Meanwhile, experiments on a public dataset validate
our framework's generalizability.</pre> </div><div class="modal-footer"><button type="button" class="btn btn-default" data-dismiss="modal">Close</button></div></div></div></div>
</li><br>

<li>
<span class="author">Ruoyou Wu#, <b>Jian Cheng</b>#, Cheng Li, Juan Zou, Wenxin Fan, Xinrui Ma, Hua Guo, Yong Liang, Shanshan Wang*</span>,<br>
<span class="title">"Spherical Harmonics Representation Learning for High-Fidelity and Generalizable Super-Resolution in Diffusion MRI"</span>,<br>
<span class="publisher">IEEE Transactions on Biomedical Engineering (<b>TBME</b>)</span>, <span class="year">2025</span>.
<br>
<a target="_self" href="https://dx.doi.org/10.1109/TBME.2025.3607105"><img src="Icons/www.gif" alt="[www]" style="width: 16px; height: 16px;"></a>&nbsp;
[<a type="button" data-toggle="modal" data-target="#bib-wu2025spherical">bibtex</a>]&nbsp;
[<a type="button" data-toggle="modal" data-target="#abs-wu2025spherical">abstract</a>]&nbsp;

<div class="modal fade" id="bib-wu2025spherical" role="dialog"><div class="modal-dialog"><div class="modal-content"><div class="modal-header"><button type="button" class="close" data-dismiss="modal">&times;</button><h4 class="modal-title">Bibtex</h4></div><div class="modal-body">
<pre>@article{wu2025spherical,
 author = {Ruoyou Wu and Jian Cheng and Cheng Li and Juan Zou and Wenxin Fan and Xinrui Ma and Hua Guo and Yong Liang and Shanshan Wang},
 doi = {10.1109/TBME.2025.3607105},
 journal = {IEEE Transactions on Biomedical Engineering},
 publisher = {IEEE},
 title = {Spherical Harmonics Representation Learning for High-Fidelity and Generalizable Super-Resolution in Diffusion MRI},
 url = {https://dx.doi.org/10.1109/TBME.2025.3607105},
 year = {2025}
}</pre> </div><div class="modal-footer"><button type="button" class="btn btn-default" data-dismiss="modal">Close</button></div></div></div></div>
<div class="modal fade" id="abs-wu2025spherical" role="dialog"><div class="modal-dialog"><div class="modal-content"><div class="modal-header"><button type="button" class="close" data-dismiss="modal">&times;</button><h4 class="modal-title">Abstract</h4></div><div class="modal-body">
<pre>Objective: Diffusion magnetic resonance imaging (dMRI) often suffers
from low spatial and angular resolution due to inherent limitations
in imaging hardware and system noise, adversely affecting the
accurate estimation of microstructural parameters with fine
anatomical details. Deep learning-based super-resolution techniques
have shown promise in enhancing dMRI resolution without increasing
acquisition time. However, most existing methods are confined to
either spatial or angular super-resolution, disrupting the
information exchange between the two domains and limiting their
effectiveness in capturing detailed microstructural features.
Furthermore, traditional pixel-wise loss functions only consider
pixel differences, and struggle to recover intricate image details
essential for high-resolution reconstruction. Method: We propose
SHRL-dMRI, a novel Spherical Harmonics Representation Learning
framework for high-fidelity, generalizable super-resolution in dMRI
to address these challenges. SHRL-dMRI explores implicit neural
representations and spherical harmonics to model continuous spatial
and angular representations, simultaneously enhancing both spatial
and angular resolution while improving the accuracy of
microstructural parameter estimation. To further preserve image
fidelity, a data-fidelity module and wavelet-based frequency loss
are introduced, ensuring the super-resolved images preserve image
consistency and retain fine details. Results: Extensive experiments
demonstrate that, compared to five other state-of-the-art methods,
our method significantly enhances dMRI data resolution, improves the
accuracy of microstructural parameter estimation, and provides
better generalization capabilities. It maintains stable performance
even under a 45× downsampling factor. Conclusion and significance:
The proposed method can effectively improve the resolution of dMRI
data without increasing the acquisition time, providing new
possibilities for future clinical applications.</pre> </div><div class="modal-footer"><button type="button" class="btn btn-default" data-dismiss="modal">Close</button></div></div></div></div>
</li><br>

<li>
<span class="author">Shaodong Ding, Ziyang Liu, Pan Liu, Wanlin Zhu, Hong Xu, Zixiao Li, Haijun Niu, <b>Jian Cheng</b>*, Tao Liu*</span>,<br>
<span class="title">"C3R: Category contrastive adaptation and consistency regularization for cross-modality medical image segmentation"</span>,<br>
<span class="publisher">Expert Systems with Applications</span>, vol. 269, pp. 126304, <span class="year">2025</span>.
<br>
<a target="_self" href="https://dx.doi.org/10.1016/j.eswa.2024.126304"><img src="Icons/www.gif" alt="[www]" style="width: 16px; height: 16px;"></a>&nbsp;
[<a type="button" data-toggle="modal" data-target="#bib-ding2025c3r">bibtex</a>]&nbsp;
[<a type="button" data-toggle="modal" data-target="#abs-ding2025c3r">abstract</a>]&nbsp;
[citations: <a target="_blank" href="https://scholar.google.com//citations?view_op=view_citation&hl=en&oe=ASCII&user=BARqXQ0AAAAJ&pagesize=100&sortby=pubdate&citation_for_view=BARqXQ0AAAAJ:otzGkya1bYkC">6</a>]&nbsp;

<div class="modal fade" id="bib-ding2025c3r" role="dialog"><div class="modal-dialog"><div class="modal-content"><div class="modal-header"><button type="button" class="close" data-dismiss="modal">&times;</button><h4 class="modal-title">Bibtex</h4></div><div class="modal-body">
<pre>@article{ding2025c3r,
 author = {Shaodong Ding and Ziyang Liu and Pan Liu and Wanlin Zhu and Hong Xu and Zixiao Li and Haijun Niu and Jian Cheng and Tao Liu},
 doi = {10.1016/j.eswa.2024.126304},
 journal = {Expert Systems with Applications},
 pages = {126304},
 publisher = {Elsevier},
 title = {C3R: Category contrastive adaptation and consistency regularization for cross-modality medical image segmentation},
 url = {https://dx.doi.org/10.1016/j.eswa.2024.126304},
 volume = {269},
 year = {2025}
}</pre> </div><div class="modal-footer"><button type="button" class="btn btn-default" data-dismiss="modal">Close</button></div></div></div></div>
<div class="modal fade" id="abs-ding2025c3r" role="dialog"><div class="modal-dialog"><div class="modal-content"><div class="modal-header"><button type="button" class="close" data-dismiss="modal">&times;</button><h4 class="modal-title">Abstract</h4></div><div class="modal-body">
<pre>Unsupervised Domain Adaptation (UDA) has been widely used in cross-
modality medical image segmentation, where the segmentation network
is trained using both labeled images from the source domain and
unlabeled images from the target domain. A prominent research
direction in UDA involves learning domain-invariant features through
image translation between the source and target domain images,
typically utilizing cycle consistency loss and/or adversarial loss.
However, we argue that the auxiliary cycle consistency loss and
adversarial loss may interfere with the main segmentation loss
during training, despite its utility in learning domain-invariant
features. Furthermore, existing approaches in this research
direction overlook the differentiation of various categories in
domain-invariant feature learning. In this paper, we propose a novel
UDA method, named C3R, for medical image segmentation. C3R mainly
comprises four components: a shared encoder for learning domain-
invariant features, a segmenter for generating segmentation output,
and two decoders for image translation between the source and target
domains. C3R fully explores consistency regularization in training,
including image-level consistency, feature-level consistency, and
segmentation output-level consistency. Moreover, C3R employs a
detached training strategy to alleviate conflict between the main
segmentation loss and auxiliary cycle consistency loss and
adversarial loss. Last, C3R applies contrastive learning to pull
together pixels of the same category, while pushing apart pixels of
different categories, thereby enhancing the final segmentation
results. Experimental results show that C3R outperforms other state-
of-the-art methods by a considerable margin in Dice: 2.25% in
cardiac substructure segmentation, 7.98% in brain tumor
segmentation, and 1.8% in abdominal multi-organ segmentation.</pre> </div><div class="modal-footer"><button type="button" class="btn btn-default" data-dismiss="modal">Close</button></div></div></div></div>
</li><br>

<li>
<span class="author">Shanshan Wang#*, Shoujun Yu#, <b>Jian Cheng</b>#, Sen Jia, Changjun Tie, Jiayu Zhu, Haohao Peng, Yijing Dong, Jianzhong He, Fan Zhang, Yaowen Xing, Xiuqin Jia, Qi Yang, Qiyuan Tian, Hua Guo, Guobin Li, Hairong Zheng*</span>,<br>
<span class="title">"Diff5T: Benchmarking human brain diffusion MRI with an extensive 5.0 Tesla k-space and spatial dataset"</span>,<br>
<span class="publisher">Scientific Data</span>, vol. 12, no. 1, pp. 1352, <span class="year">2025</span>.
<br>
<a target="_self" href="https://dx.doi.org/10.1038/s41597-025-05640-2"><img src="Icons/www.gif" alt="[www]" style="width: 16px; height: 16px;"></a>&nbsp;
[<a type="button" data-toggle="modal" data-target="#bib-wang2025diff5t">bibtex</a>]&nbsp;
[<a type="button" data-toggle="modal" data-target="#abs-wang2025diff5t">abstract</a>]&nbsp;
[citations: <a target="_blank" href="https://scholar.google.com//citations?view_op=view_citation&hl=en&oe=ASCII&user=BARqXQ0AAAAJ&pagesize=100&sortby=pubdate&citation_for_view=BARqXQ0AAAAJ:buQ7SEKw-1sC">2</a>]&nbsp;

<div class="modal fade" id="bib-wang2025diff5t" role="dialog"><div class="modal-dialog"><div class="modal-content"><div class="modal-header"><button type="button" class="close" data-dismiss="modal">&times;</button><h4 class="modal-title">Bibtex</h4></div><div class="modal-body">
<pre>@article{wang2025diff5t,
 author = {Shanshan Wang and Shoujun Yu and Jian Cheng and Sen Jia and Changjun Tie and Jiayu Zhu and Haohao Peng and Yijing Dong and Jianzhong He and Fan Zhang and Yaowen Xing and Xiuqin Jia and Qi Yang and Qiyuan Tian and Hua Guo and Guobin Li and Hairong Zheng},
 doi = {10.1038/s41597-025-05640-2},
 journal = {Scientific Data},
 number = {1},
 pages = {1352},
 publisher = {Nature Publishing Group UK London},
 title = {Diff5T: Benchmarking human brain diffusion MRI with an extensive 5.0 Tesla k-space and spatial dataset},
 url = {https://dx.doi.org/10.1038/s41597-025-05640-2},
 volume = {12},
 year = {2025}
}</pre> </div><div class="modal-footer"><button type="button" class="btn btn-default" data-dismiss="modal">Close</button></div></div></div></div>
<div class="modal fade" id="abs-wang2025diff5t" role="dialog"><div class="modal-dialog"><div class="modal-content"><div class="modal-header"><button type="button" class="close" data-dismiss="modal">&times;</button><h4 class="modal-title">Abstract</h4></div><div class="modal-body">
<pre>Difusion magnetic resonance imaging (dMRI) provides critical
insights into the microstructural and connectional organization of
the human brain. However, the availability of high-feld, open-access
datasets that include raw k-space data for advanced research remains
limited. To address this gap, we introduce Dif5T, a frst
comprehensive 5.0 Tesla difusion MRI dataset focusing on the human
brain. This dataset includes raw k-space data and reconstructed
difusion images, acquired using a variety of imaging protocols.
Dif5T is designed to support the development and benchmarking of
innovative methods in artifact correction, image reconstruction,
image preprocessing, difusion modelling and tractography. The
dataset features a wide range of difusion parameters, including
multiple b-values and gradient directions, allowing extensive
research applications in studying human brain microstructure and
connectivity. With its emphasis on open accessibility and detailed
benchmarks, Dif5T serves as a valuable resource for advancing human
brain mapping research using difusion MRI, fostering
reproducibility, and enabling collaboration across the neuroscience
and medical imaging communities.</pre> </div><div class="modal-footer"><button type="button" class="btn btn-default" data-dismiss="modal">Close</button></div></div></div></div>
</li><br>

<li>
<span class="author">Hua Zhu#, Shuxiang Zhu#, Meixin Zhao, Ziyan Zhu, Yuxuan Shao, Xinxi Lu, Tao Liu, Haogang Zhu, Ni Shu, Hua Lin*, <b>Jian Cheng</b>*</span>,<br>
<span class="title">"Cerebellar and Brainstem White Matter Geometric Alterations in Multiple System Atrophy: A DFA-Based Biomarker for Disease Staging"</span>,<br>
<span class="publisher">CNS Neuroscience & Therapeutics</span>, vol. 31, no. 12, pp. e70623, <span class="year">2025</span>.
<br><a target="_self" href="https://onlinelibrary.wiley.com/doi/epdf/10.1111/cns.70623"><img src="Icons/pdf.gif" alt="[pdf]" style="width: 16px; height: 16px;"></a>&nbsp;
<a target="_self" href="https://dx.doi.org/10.1111/cns.70623"><img src="Icons/www.gif" alt="[www]" style="width: 16px; height: 16px;"></a>&nbsp;
[<a type="button" data-toggle="modal" data-target="#bib-zhu2025cerebellar">bibtex</a>]&nbsp;
[<a type="button" data-toggle="modal" data-target="#abs-zhu2025cerebellar">abstract</a>]&nbsp;

<div class="modal fade" id="bib-zhu2025cerebellar" role="dialog"><div class="modal-dialog"><div class="modal-content"><div class="modal-header"><button type="button" class="close" data-dismiss="modal">&times;</button><h4 class="modal-title">Bibtex</h4></div><div class="modal-body">
<pre>@article{zhu2025cerebellar,
 author = {Hua Zhu and Shuxiang Zhu and Meixin Zhao and Ziyan Zhu and Yuxuan Shao and Xinxi Lu and Tao Liu and Haogang Zhu and Ni Shu and Hua Lin and Jian Cheng},
 doi = {10.1111/cns.70623},
 journal = {CNS Neuroscience & Therapeutics},
 number = {12},
 pages = {e70623},
 pdf = {https://onlinelibrary.wiley.com/doi/epdf/10.1111/cns.70623},
 publisher = {Wiley Online Library},
 title = {Cerebellar and Brainstem White Matter Geometric Alterations in Multiple System Atrophy: A DFA-Based Biomarker for Disease Staging},
 url = {https://dx.doi.org/10.1111/cns.70623},
 volume = {31},
 year = {2025}
}</pre> </div><div class="modal-footer"><button type="button" class="btn btn-default" data-dismiss="modal">Close</button></div></div></div></div>
<div class="modal fade" id="abs-zhu2025cerebellar" role="dialog"><div class="modal-dialog"><div class="modal-content"><div class="modal-header"><button type="button" class="close" data-dismiss="modal">&times;</button><h4 class="modal-title">Abstract</h4></div><div class="modal-body">
<pre>Aims: To characterize white matter geometric pathology in cerebellar
subtype of multiple system atrophy (MSA-C) using director field
analysis (DFA) and identify stage-specific biomarkers. Methods: We
analyzed single-shell diffusion MRI (b = 1000) in 31 MSA-C patients
(15 early-, 16 late-stage) and 33 controls. DFA quantified axonal
geometry (splay/bend/twist), complemented by fixel-based analysis
(FBA) and brainstem volumetry. Group comparisons used threshold free
cluster enhancement (TFCE) (p < 0.05 FWE-corrected). DFA-altered
regions were correlated with clinical scores. AutoGluon evaluated
classification performance using different feature sets. Results:
MSA-C exhibited distinct geometric degeneration patterns: cerebellar
pathways showed reduced splay, bend, and twist (reflecting Wallerian
degeneration), whereas brainstem tracts demonstrated dissociated
geometry (increased splay/bend but decreased twist). Brainstem twist
reduction strongly differentiated early- and late-stage MSA-C (AUC =
0.95). Clinically, middle cerebellar peduncle bend correlated with
motor progression (UMSARS-II: r = 0.48), while cerebellar splay
reduction predicted ataxia severity (SARA: r =−0.43). Conclusion:
DFA captures circuit-specific white matter pathology in MSA-C, with
brainstem twist emerging as a novel biomarker associated with
disease stage. The integration of geometric metrics with automated
machine learning provides a robust framework for early diagnosis and
disease staging, highlighting distinct neurodegenerative mechanisms
in cerebellar versus brainstem pathways.</pre> </div><div class="modal-footer"><button type="button" class="btn btn-default" data-dismiss="modal">Close</button></div></div></div></div>
</li><br>

<li>
<span class="author">Taohui Xiao, <b>Jian Cheng</b>*, Wenxin Fan, Enqing Dong*, Shanshan Wang*</span>,<br>
<span class="title">"SamRobNODDI: q-space sampling-augmented continuous representation learning for robust and generalized NODDI"</span>,<br>
<span class="publisher">Physics in Medicine & Biology</span>, vol. 70, no. 17, pp. 175015, <span class="year">2025</span>.
<br><a target="_self" href="https://arxiv.org/pdf/2411.06444.pdf"><img src="Icons/pdf.gif" alt="[pdf]" style="width: 16px; height: 16px;"></a>&nbsp;
<a target="_self" href="https://dx.doi.org/10.1088/1361-6560/adf9b8"><img src="Icons/www.gif" alt="[www]" style="width: 16px; height: 16px;"></a>&nbsp;
[<a type="button" data-toggle="modal" data-target="#bib-xiao2025samrobnoddi">bibtex</a>]&nbsp;
[<a type="button" data-toggle="modal" data-target="#abs-xiao2025samrobnoddi">abstract</a>]&nbsp;
[<a target="_self" href="https://arxiv.org/abs/2411.06444">arxiv</a>]&nbsp;

<div class="modal fade" id="bib-xiao2025samrobnoddi" role="dialog"><div class="modal-dialog"><div class="modal-content"><div class="modal-header"><button type="button" class="close" data-dismiss="modal">&times;</button><h4 class="modal-title">Bibtex</h4></div><div class="modal-body">
<pre>@article{xiao2025samrobnoddi,
 author = {Taohui Xiao and Jian Cheng and Wenxin Fan and Enqing Dong and Shanshan Wang},
 doi = {10.1088/1361-6560/adf9b8},
 journal = {Physics in Medicine & Biology},
 number = {17},
 pages = {175015},
 pdf = {https://arxiv.org/pdf/2411.06444.pdf},
 publisher = {IOP Publishing},
 title = {SamRobNODDI: q-space sampling-augmented continuous representation learning for robust and generalized NODDI},
 url = {https://dx.doi.org/10.1088/1361-6560/adf9b8},
 volume = {70},
 year = {2025}
}</pre> </div><div class="modal-footer"><button type="button" class="btn btn-default" data-dismiss="modal">Close</button></div></div></div></div>
<div class="modal fade" id="abs-xiao2025samrobnoddi" role="dialog"><div class="modal-dialog"><div class="modal-content"><div class="modal-header"><button type="button" class="close" data-dismiss="modal">&times;</button><h4 class="modal-title">Abstract</h4></div><div class="modal-body">
<pre>Objective. Neurite orientation dispersion and density imaging
(NODDI) microstructure estimation from diffusion magnetic resonance
imaging (dMRI) is of great significance for the discovery and
treatment of various neurological diseases. Current deep learning-
based methods accelerate the speed of NODDI parameter estimation and
improve the accuracy. However, most methods require the number and
coordinates of gradient directions during testing and training to
remain strictly consistent, significantly limiting the
generalization and robustness of these models in NODDI parameter
estimation. Therefore, it is imperative to develop methods that can
perform robustly under varying diffusion gradient directions.
Approach. In this paper, we propose a q-space sampling augmentation-
based continuous representation learning framework (SamRobNODDI) to
achieve robust and generalized NODDI. Specifically, a continuous
representation learning method based on q-space sampling
augmentation is introduced to fully explore the information between
different gradient directions in q-space. Furthermore, we design a
sampling consistency loss to constrain the outputs of different
sampling schemes, ensuring that the outputs remain as consistent as
possible, thereby further enhancing performance and robustness to
varying q-space sampling schemes. SamRobNODDI is also a flexible
framework that can be applied to different backbone networks. Main
results. SamRobNODDI was compared against seven state-of-the-art
methods across 19 diverse q-space sampling schemes. Extensive
experimental validations have been conducted under both identical
and diverse sampling schemes for training and testing, as well as
across varying sampling rates, different loss functions, and
multiple network backbones. Significance. Results demonstrate that
the proposed SamRobNODDI has better performance, robustness,
generalization, and flexibility in the face of varying q-space
sampling schemes.</pre> </div><div class="modal-footer"><button type="button" class="btn btn-default" data-dismiss="modal">Close</button></div></div></div></div>
</li><br>

<li>
<span class="author">Miaoyan Wang#, Hua Zhu#, Tingting Huang#, Jingjing Qiao, Bo Peng, Ni Shu, Anqi Qiu, <b>Jian Cheng</b>*, Haoxiang Jiang*</span>,<br>
<span class="title">"MRI Assessment of Geometric Microstructural Changes of White Matter in Infants With Periventricular White Matter Injury and Spastic Cerebral Palsy"</span>,<br>
<span class="publisher">Journal of Magnetic Resonance Imaging</span>, vol. 62, no. 2, pp. 403&ndash;414, <span class="year">2025</span>.
<br><a target="_self" href="https://onlinelibrary.wiley.com/doi/epdf/10.1002/jmri.70043"><img src="Icons/pdf.gif" alt="[pdf]" style="width: 16px; height: 16px;"></a>&nbsp;
<a target="_self" href="https://dx.doi.org/10.1002/jmri.29730"><img src="Icons/www.gif" alt="[www]" style="width: 16px; height: 16px;"></a>&nbsp;
[<a type="button" data-toggle="modal" data-target="#bib-wang2025mri">bibtex</a>]&nbsp;
[<a type="button" data-toggle="modal" data-target="#abs-wang2025mri">abstract</a>]&nbsp;
[citations: <a target="_blank" href="https://scholar.google.com//citations?view_op=view_citation&hl=en&oe=ASCII&user=BARqXQ0AAAAJ&pagesize=100&sortby=pubdate&citation_for_view=BARqXQ0AAAAJ:BzfGm06jWhQC">1</a>]&nbsp;
(<span class="hlnote2">Cover Article</span>)&nbsp;

<div class="modal fade" id="bib-wang2025mri" role="dialog"><div class="modal-dialog"><div class="modal-content"><div class="modal-header"><button type="button" class="close" data-dismiss="modal">&times;</button><h4 class="modal-title">Bibtex</h4></div><div class="modal-body">
<pre>@article{wang2025mri,
 author = {Miaoyan Wang and Hua Zhu and Tingting Huang and Jingjing Qiao and Bo Peng and Ni Shu and Anqi Qiu and Jian Cheng and Haoxiang Jiang},
 doi = {10.1002/jmri.29730},
 journal = {Journal of Magnetic Resonance Imaging},
 number = {2},
 pages = {403-414},
 pdf = {https://onlinelibrary.wiley.com/doi/epdf/10.1002/jmri.70043},
 publisher = {Wiley Online Library},
 title = {MRI Assessment of Geometric Microstructural Changes of White Matter in Infants With Periventricular White Matter Injury and Spastic Cerebral Palsy},
 url = {https://dx.doi.org/10.1002/jmri.29730},
 volume = {62},
 year = {2025}
}</pre> </div><div class="modal-footer"><button type="button" class="btn btn-default" data-dismiss="modal">Close</button></div></div></div></div>
<div class="modal fade" id="abs-wang2025mri" role="dialog"><div class="modal-dialog"><div class="modal-content"><div class="modal-header"><button type="button" class="close" data-dismiss="modal">&times;</button><h4 class="modal-title">Abstract</h4></div><div class="modal-body">
<pre>Background: Periventricular white matter injury (PWMI) is a high-
risk factor for spastic cerebral palsy (SCP). Purpose: To
investigate the geometric microstructural changes in WM in infants
with PWMI-SCP using MRI which may facilitate early identification.
Study Type: Retrospective cohort study. Population: Twenty-three
healthy infants (aged 6.53–36 months), 25 infants with PWMI-SCP
(aged 6–33 months), and 32 infants with PWMI-nonSCP (aged 6–36
months). Field Strength/Sequence: 3.0 T, T1-weighted three-
dimensional gradient-echo sequence, and diffusion tensor imaging
(DTI) with a single-shot gradient echo planar sequence. Assessment:
The brain was automatically segmented, parcellated into major
regions of interest according to the Desikan– Killiany atlas and
volumes extracted. Fractional anisotropy (FA) and mean diffusivity
(MD) of regions were extracted from DTI data. Director field
analysis (DFA) was used to assess the geometric microstructural
properties of WM. Motor dysfunction was graded from l (mild) to 5
(severe) according to the Gross Motor Function Classification
System. Statistical Tests: Tests included analysis of variance,
correlation analysis, mediation analysis, and receiver operating
characteristic analysis. Corrected P-values <0.05 were considered
significant. Mediation analysis examined whether DFA metrics
mediated the relationship between brain morphological and motor
dysfunction. Models were constructed to identify PWMI-SCP. Results:
The PWMI-SCP group exhibited significantly elevated all four DFA
metrics (splay, bend, twist, and distortion), primarily in the
corpus callosum, posterior thalamic radiata, and corona radiata,
compared to the PWMI-nonSCP group, and was associated with enlarged
lateral ventricles, reduced deep nuclear volumes and motor
dysfunction. Mediation analysis indicated that increased splay in
the corpus callosum partially mediates (mediating effect ratio:
29.74%, 22.46%) the relationship between the lateral ventricles and
motor function. The results showed that DFA achieved a higher area
under the curve (AUC) than the FA + MD, especially in distinguishing
PWMI-nonSCP from PWMI-SCP (AUC = 0.93). Data Conclusion: Monitoring
fiber-orientational alterations may provide new insights into early
identification of PWMI-SCP.</pre> </div><div class="modal-footer"><button type="button" class="btn btn-default" data-dismiss="modal">Close</button></div></div></div></div>
</li><br>

<li>
<span class="author">Ziyang Liu, Sikang Ren, Heng Zhang, Zhiyi Liao, Zhiming Liu, Xu An, <b>Jian Cheng</b>, Chunde Li, Jian Gong, Haijun Niu, Jing Jing, Zixiao Li, Tao Liu, Yongji Tian</span>,<br>
<span class="title">"Multiparametric MRI-based machine learning system of molecular subgroups and prognosis in medulloblastoma"</span>,<br>
<span class="publisher">European Radiology</span>, vol. 35, pp. 5053–5063, <span class="year">2025</span>.
<br>
<a target="_self" href="https://dx.doi.org/10.1007/s00330-025-11385-8"><img src="Icons/www.gif" alt="[www]" style="width: 16px; height: 16px;"></a>&nbsp;
[<a type="button" data-toggle="modal" data-target="#bib-liu2025multiparametric">bibtex</a>]&nbsp;
[<a type="button" data-toggle="modal" data-target="#abs-liu2025multiparametric">abstract</a>]&nbsp;
[citations: <a target="_blank" href="https://scholar.google.com//citations?view_op=view_citation&hl=en&oe=ASCII&user=BARqXQ0AAAAJ&pagesize=100&sortby=pubdate&citation_for_view=BARqXQ0AAAAJ:jgBuDB5drN8C">4</a>]&nbsp;

<div class="modal fade" id="bib-liu2025multiparametric" role="dialog"><div class="modal-dialog"><div class="modal-content"><div class="modal-header"><button type="button" class="close" data-dismiss="modal">&times;</button><h4 class="modal-title">Bibtex</h4></div><div class="modal-body">
<pre>@article{liu2025multiparametric,
 author = {Ziyang Liu and Sikang Ren and Heng Zhang and Zhiyi Liao and Zhiming Liu and Xu An and Jian Cheng and Chunde Li and Jian Gong and Haijun Niu and Jing Jing and Zixiao Li and Tao Liu and Yongji Tian},
 doi = {10.1007/s00330-025-11385-8},
 journal = {European Radiology},
 pages = {5053–5063},
 publisher = {Springer},
 title = {Multiparametric MRI-based machine learning system of molecular subgroups and prognosis in medulloblastoma},
 url = {https://dx.doi.org/10.1007/s00330-025-11385-8},
 volume = {35},
 year = {2025}
}</pre> </div><div class="modal-footer"><button type="button" class="btn btn-default" data-dismiss="modal">Close</button></div></div></div></div>
<div class="modal fade" id="abs-liu2025multiparametric" role="dialog"><div class="modal-dialog"><div class="modal-content"><div class="modal-header"><button type="button" class="close" data-dismiss="modal">&times;</button><h4 class="modal-title">Abstract</h4></div><div class="modal-body">
<pre>Objectives We aimed to use artificial intelligence to accurately
identify molecular subgroups of medulloblastoma (MB), predict
clinical outcomes, and incorporate deep learning-based imaging
features into the risk stratification.  Methods The MRI features
were extracted for molecular subgroups by a novel multi-parameter
convolutional neural network (CNN) called Bi-ResNet-MB. Then, MR
features were used to establish a prognosis model based on XGBoost.
Finally, a novel risk stratification system to stratify the patients
based on the M2R Score (Machine learning-based Medulloblastoma Risk
Score) was proposed.  Results A total of 139 MB patients (36 female,
average age 7.27 ± 3.62 years) were treated at Beijing Tiantan
Hospital. The Bi-ResNet-MB model excelled in molecular subgroup
classification, achieving an average AUC of 0.946 (95% CI:
0.899–0.993). For prognostic prediction, our models achieved AUCs of
0.840 (95% CI: 0.792–0.888), 0.949 (95% CI: 0.899–0.999), and 0.960
(95% CI: 0.915–1.000) for OS, and 0.946 (95% CI: 0.905–0.987), 0.932
(95% CI: 0.875–0.989), and 0.964 (95% CI: 0.921–1.000) for PFS at 1,
3, and 5 years. In an independent validation dataset of 108 patients
(33 female, average age 7.11 ± 2.92 years), the average AUC of
molecular subgroup classification reached 0.894 (95% CI:
0.797–1.000). For PFS prediction at 1, 3, and 5 years, the AUCs were
0.832 (95% CI: 0.724–0.920), 0.875 (95% CI: 0.781–0.967), and 0.907
(95% CI: 0.760–1.000), respectively.  Conclusions Based on machine
learning and MRI data, models for MB molecular subgroups and
prognosis prediction and the novel risk stratification system may
significantly benefit patients.</pre> </div><div class="modal-footer"><button type="button" class="btn btn-default" data-dismiss="modal">Close</button></div></div></div></div>
</li><br>

<li>
<span class="author">Xinrui Ma, <b>Jian Cheng</b>, Wenxin Fan, Ruoyou Wu, Yongquan Ye, Shanshan Wang</span>,<br>
<span class="title">"3D Anatomical Structure-Guided Deep Learning for Accurate Diffusion Microstructure Imaging"</span>,<br>
<span class="publisher">2025 IEEE 22nd International Symposium on Biomedical Imaging (ISBI)</span>, pp. 1&ndash;4, <span class="year">2025</span>.
<br>
[<a type="button" data-toggle="modal" data-target="#bib-ma20253d">bibtex</a>]&nbsp;
[citations: <a target="_blank" href="https://scholar.google.com//citations?view_op=view_citation&hl=en&oe=ASCII&user=BARqXQ0AAAAJ&pagesize=100&sortby=pubdate&citation_for_view=BARqXQ0AAAAJ:lgwcVrK6X84C">1</a>]&nbsp;

<div class="modal fade" id="bib-ma20253d" role="dialog"><div class="modal-dialog"><div class="modal-content"><div class="modal-header"><button type="button" class="close" data-dismiss="modal">&times;</button><h4 class="modal-title">Bibtex</h4></div><div class="modal-body">
<pre>@inproceedings{ma20253d,
 author = {Xinrui Ma and Jian Cheng and Wenxin Fan and Ruoyou Wu and Yongquan Ye and Shanshan Wang},
 booktitle = {2025 IEEE 22nd International Symposium on Biomedical Imaging (ISBI)},
 organization = {IEEE},
 pages = {1-4},
 title = {3D Anatomical Structure-Guided Deep Learning for Accurate Diffusion Microstructure Imaging},
 year = {2025}
}</pre> </div><div class="modal-footer"><button type="button" class="btn btn-default" data-dismiss="modal">Close</button></div></div></div></div>
</li><br>

<li>
<span class="author">Taohui Xiao, <b>Jian Cheng</b>, Wenxin Fan, Jing Yang, Enqing Dong, Shanshan Wang</span>,<br>
<span class="title">"Estimation with Adaptive Sampling Under Continuous Representation"</span>,<br>
<span class="publisher">Computational Diffusion MRI: 15th International Workshop, CDMRI 2024, Held in Conjunction with MICCAI 2024, Marrakesh, Morocco, October 6, 2024, Proceedings</span>, vol. 15171, pp. 153, <span class="year">2025</span>.
<br>
[<a type="button" data-toggle="modal" data-target="#bib-xiao2025estimation">bibtex</a>]&nbsp;

<div class="modal fade" id="bib-xiao2025estimation" role="dialog"><div class="modal-dialog"><div class="modal-content"><div class="modal-header"><button type="button" class="close" data-dismiss="modal">&times;</button><h4 class="modal-title">Bibtex</h4></div><div class="modal-body">
<pre>@inproceedings{xiao2025estimation,
 author = {Taohui Xiao and Jian Cheng and Wenxin Fan and Jing Yang and Enqing Dong and Shanshan Wang},
 booktitle = {Computational Diffusion MRI: 15th International Workshop, CDMRI 2024, Held in Conjunction with MICCAI 2024, Marrakesh, Morocco, October 6, 2024, Proceedings},
 organization = {Springer Nature},
 pages = {153},
 title = {Estimation with Adaptive Sampling Under Continuous Representation},
 volume = {15171},
 year = {2025}
}</pre> </div><div class="modal-footer"><button type="button" class="btn btn-default" data-dismiss="modal">Close</button></div></div></div></div>
</li><br>

</ol><h2><a name="year2024"></a>2024</h2>

<ol reversed>

<li>
<span class="author">Guozheng Feng, Yiwen Wang, Weijie Huang, Haojie Chen, <b>Jian Cheng</b>, Ni Shu*</span>,<br>
<span class="title">"Spatial and temporal pattern of structure–function coupling of human brain connectome with development"</span>,<br>
<span class="publisher"><b>eLife</b></span>, vol. 13, pp. RP93325, jun, <span class="year">2024</span>.
<br>
<a target="_self" href="https://dx.doi.org/10.7554/eLife.93325"><img src="Icons/www.gif" alt="[www]" style="width: 16px; height: 16px;"></a>&nbsp;
[<a type="button" data-toggle="modal" data-target="#bib-Feng_elife2024">bibtex</a>]&nbsp;
[<a type="button" data-toggle="modal" data-target="#abs-Feng_elife2024">abstract</a>]&nbsp;
[citations: <a target="_blank" href="https://scholar.google.com//citations?view_op=view_citation&hl=en&oe=ASCII&user=BARqXQ0AAAAJ&pagesize=100&sortby=pubdate&citation_for_view=BARqXQ0AAAAJ:vDZJ-YLwNdEC">23</a>]&nbsp;

<div class="modal fade" id="bib-Feng_elife2024" role="dialog"><div class="modal-dialog"><div class="modal-content"><div class="modal-header"><button type="button" class="close" data-dismiss="modal">&times;</button><h4 class="modal-title">Bibtex</h4></div><div class="modal-body">
<pre>@article{Feng_elife2024,
 author = {Guozheng Feng and Yiwen Wang and Weijie Huang and Haojie Chen and Jian Cheng and Ni Shu},
 doi = {10.7554/eLife.93325},
 editor = {Huang, Susie Y and Roiser, Jonathan},
 journal = {eLife},
 month = {jun},
 pages = {RP93325},
 publisher = {eLife Sciences Publications, Ltd},
 title = {Spatial and temporal pattern of structure–function coupling of human brain connectome with development},
 url = {https://dx.doi.org/10.7554/eLife.93325},
 volume = {13},
 year = {2024}
}</pre> </div><div class="modal-footer"><button type="button" class="btn btn-default" data-dismiss="modal">Close</button></div></div></div></div>
<div class="modal fade" id="abs-Feng_elife2024" role="dialog"><div class="modal-dialog"><div class="modal-content"><div class="modal-header"><button type="button" class="close" data-dismiss="modal">&times;</button><h4 class="modal-title">Abstract</h4></div><div class="modal-body">
<pre>Brain structural circuitry shapes a richly patterned functional
synchronization, supporting for complex cognitive and behavioural
abilities. However, how coupling of structural connectome (SC) and
functional connectome (FC) develops and its relationships with
cognitive functions and transcriptomic architecture remain unclear.
We used multimodal magnetic resonance imaging data from 439
participants aged 5.7–21.9 years to predict functional connectivity
by incorporating intracortical and extracortical structural
connectivity, characterizing SC–FC coupling. Our findings revealed
that SC–FC coupling was strongest in the visual and somatomotor
networks, consistent with evolutionary expansion, myelin content,
and functional principal gradient. As development progressed, SC–FC
coupling exhibited heterogeneous alterations dominated by an
increase in cortical regions, broadly distributed across the
somatomotor, frontoparietal, dorsal attention, and default mode
networks. Moreover, we discovered that SC–FC coupling significantly
predicted individual variability in general intelligence, mainly
influencing frontoparietal and default mode networks. Finally, our
results demonstrated that the heterogeneous development of SC–FC
coupling is positively associated with genes in oligodendrocyte-
related pathways and negatively associated with astrocyte-related
genes. This study offers insight into the maturational principles of
SC–FC coupling in typical development.</pre> </div><div class="modal-footer"><button type="button" class="btn btn-default" data-dismiss="modal">Close</button></div></div></div></div>
</li><br>

<li>
<span class="author">Yijun Zhou, Jing Jing, Zhe Zhang, Yuesong Pan, Xueli Cai, Wanlin Zhu, Zixiao Li, Chang Liu, Hao Liu, Xia Meng, <b>Jian Cheng</b>, Yilong Wang, Hao Li, Suying Wang, Haijun Niu, Wei Wen, Perminder S Sachdev, Tiemin Wei, Tao Liu*, Yongjun Wang*</span>,<br>
<span class="title">"Disrupted pattern of rich-club organization in structural brain network from prediabetes to diabetes: A population-based study"</span>,<br>
<span class="publisher">Human Brain Mapping (<b>HBM</b>)</span>, vol. 45, no. 2, pp. e26598, <span class="year">2024</span>.
<br>
<a target="_self" href="https://dx.doi.org/10.1002/hbm.26598"><img src="Icons/www.gif" alt="[www]" style="width: 16px; height: 16px;"></a>&nbsp;
[<a type="button" data-toggle="modal" data-target="#bib-zhou2024disrupted">bibtex</a>]&nbsp;
[<a type="button" data-toggle="modal" data-target="#abs-zhou2024disrupted">abstract</a>]&nbsp;
[citations: <a target="_blank" href="https://scholar.google.com//citations?view_op=view_citation&hl=en&oe=ASCII&user=BARqXQ0AAAAJ&pagesize=100&sortby=pubdate&citation_for_view=BARqXQ0AAAAJ:2tRrZ1ZAMYUC">3</a>]&nbsp;

<div class="modal fade" id="bib-zhou2024disrupted" role="dialog"><div class="modal-dialog"><div class="modal-content"><div class="modal-header"><button type="button" class="close" data-dismiss="modal">&times;</button><h4 class="modal-title">Bibtex</h4></div><div class="modal-body">
<pre>@article{zhou2024disrupted,
 author = {Yijun Zhou and Jing Jing and Zhe Zhang and Yuesong Pan and Xueli Cai and Wanlin Zhu and Zixiao Li and Chang Liu and Hao Liu and Xia Meng and Jian Cheng and Yilong Wang and Hao Li and Suying Wang and Haijun Niu and Wei Wen and Perminder S Sachdev and Tiemin Wei and Tao Liu and Yongjun Wang},
 doi = {10.1002/hbm.26598},
 journal = {Human Brain Mapping},
 number = {2},
 pages = {e26598},
 publisher = {Wiley Online Library},
 title = {Disrupted pattern of rich-club organization in structural brain network from prediabetes to diabetes: A population-based study},
 url = {https://dx.doi.org/10.1002/hbm.26598},
 volume = {45},
 year = {2024}
}</pre> </div><div class="modal-footer"><button type="button" class="btn btn-default" data-dismiss="modal">Close</button></div></div></div></div>
<div class="modal fade" id="abs-zhou2024disrupted" role="dialog"><div class="modal-dialog"><div class="modal-content"><div class="modal-header"><button type="button" class="close" data-dismiss="modal">&times;</button><h4 class="modal-title">Abstract</h4></div><div class="modal-body">
<pre>The network nature of the brain is gradually becoming a consensus in
the neuroscience field. A set of highly connected regions in the
brain network called “rich-club” are crucial high efficiency
communication hubs in the brain. The abnormal rich-club organization
can reflect underlying abnormal brain function and metabolism, which
receives increasing attention. Diabetes is one of the risk factors
for neurological diseases, and most individuals with prediabetes
will develop overt diabetes within their lifetime. However, the
gradual impact of hyperglycemia on brain structures, including rich-
club organization, remains unclear. We hypothesized that the brain
follows a special disrupted pattern of rich-club organization in
prediabetes and diabetes. We used cross-sectional baseline data from
the population-based PolyvasculaR Evaluation for Cognitive
Impairment and vaScular Events (PRECISE) study, which included 2218
participants with a mean age of 61.3 ± 6.6 years and 54.1% females
comprising</pre> </div><div class="modal-footer"><button type="button" class="btn btn-default" data-dismiss="modal">Close</button></div></div></div></div>
</li><br>

<li>
<span class="author">Jianyu Li#, <b>Jian Cheng</b>#, Lei Yang#, Qihui Niu*, Yuanchao Zhang*, Lena Palaniyappan</span>,<br>
<span class="title">"Association of cortical gyrification, white matter microstructure, and phenotypic profile in medication-naïve obsessive–compulsive disorder"</span>,<br>
<span class="publisher">Psychological Medicine</span>, vol. 54, no. 8, pp. 1573&ndash;1579, <span class="year">2024</span>.
<br>
<a target="_self" href="https://dx.doi.org/10.1017/S0033291723003422"><img src="Icons/www.gif" alt="[www]" style="width: 16px; height: 16px;"></a>&nbsp;
[<a type="button" data-toggle="modal" data-target="#bib-li_cheng_yang_niu_zhang_palaniyappan_2023">bibtex</a>]&nbsp;
[<a type="button" data-toggle="modal" data-target="#abs-li_cheng_yang_niu_zhang_palaniyappan_2023">abstract</a>]&nbsp;
[citations: <a target="_blank" href="https://scholar.google.com//citations?view_op=view_citation&hl=en&oe=ASCII&user=BARqXQ0AAAAJ&pagesize=100&sortby=pubdate&citation_for_view=BARqXQ0AAAAJ:Ak0FvsSvgGUC">2</a>]&nbsp;

<div class="modal fade" id="bib-li_cheng_yang_niu_zhang_palaniyappan_2023" role="dialog"><div class="modal-dialog"><div class="modal-content"><div class="modal-header"><button type="button" class="close" data-dismiss="modal">&times;</button><h4 class="modal-title">Bibtex</h4></div><div class="modal-body">
<pre>@article{li_cheng_yang_niu_zhang_palaniyappan_2023,
 author = {Jianyu Li and Jian Cheng and Lei Yang and Qihui Niu and Yuanchao Zhang and Lena Palaniyappan},
 doi = {10.1017/S0033291723003422},
 journal = {Psychological Medicine},
 number = {8},
 pages = {1573-1579},
 publisher = {Cambridge University Press},
 title = {Association of cortical gyrification, white matter microstructure, and phenotypic profile in medication-naïve obsessive–compulsive disorder},
 url = {https://dx.doi.org/10.1017/S0033291723003422},
 volume = {54},
 year = {2024}
}</pre> </div><div class="modal-footer"><button type="button" class="btn btn-default" data-dismiss="modal">Close</button></div></div></div></div>
<div class="modal fade" id="abs-li_cheng_yang_niu_zhang_palaniyappan_2023" role="dialog"><div class="modal-dialog"><div class="modal-content"><div class="modal-header"><button type="button" class="close" data-dismiss="modal">&times;</button><h4 class="modal-title">Abstract</h4></div><div class="modal-body">
<pre>Background Obsessive–compulsive disorder (OCD) is thought to arise
from dysconnectivity among interlinked brain regions resulting in a
wide spectrum of clinical manifestations. Cortical gyrification, a
key morphological feature of human cerebral cortex, has been
considered associated with developmental connectivity in early life.
Monitoring cortical gyrification alterations may provide new
insights into the developmental pathogenesis of OCD.  Methods Sixty-
two medication-naive patients with OCD and 59 healthy controls (HCs)
were included in this study. Local gyrification index (LGI) was
extracted from T1-weighted MRI data to identify the gyrification
changes in OCD. Total distortion (splay, bend, or twist of fibers)
was calculated using diffusion-weighted MRI data to examine the
changes in white matter microstructure in patients with OCD.
Results Compared with HCs, patients with OCD showed significantly
increased LGI in bilateral medial frontal gyrus and the right
precuneus, where the mean LGI was positively correlated with anxiety
score. Patients with OCD also showed significantly decreased total
distortion in the body, genu, and splenium of the corpus callosum
(CC), where the average distortion was negatively correlated with
anxiety scores. Intriguingly, the mean LGI of the affected cortical
regions was significantly correlated with the mean distortion of the
affected white matter tracts in patients with OCD.  Conclusions We
demonstrated associations among increased LGI, aberrant white matter
geometry, and higher anxiety in patients with OCD. Our findings
indicate that developmental dysconnectivity-driven alterations in
cortical folding are one of the neural substrates underlying the
clinical manifestations of OCD.</pre> </div><div class="modal-footer"><button type="button" class="btn btn-default" data-dismiss="modal">Close</button></div></div></div></div>
</li><br>

<li>
<span class="author">Yuwen Wang, Mingxiu Han, Yudan Peng, Ruoqi Zhao, Dongqiong Fan, Xia Meng, Hong Xu, Haijun Niu, <b>Jian Cheng</b>*, Tao Liu*</span>,<br>
<span class="title">"LGNet: Learning local--global EEG representations for cognitive workload classification in simulated flights"</span>,<br>
<span class="publisher">Biomedical Signal Processing and Control</span>, vol. 92, pp. 106046, <span class="year">2024</span>.
<br>
<a target="_self" href="https://dx.doi.org/10.1016/j.bspc.2024.106046"><img src="Icons/www.gif" alt="[www]" style="width: 16px; height: 16px;"></a>&nbsp;
[<a type="button" data-toggle="modal" data-target="#bib-wang2024lgnet">bibtex</a>]&nbsp;
[<a type="button" data-toggle="modal" data-target="#abs-wang2024lgnet">abstract</a>]&nbsp;
[citations: <a target="_blank" href="https://scholar.google.com//citations?view_op=view_citation&hl=en&oe=ASCII&user=BARqXQ0AAAAJ&pagesize=100&sortby=pubdate&citation_for_view=BARqXQ0AAAAJ:6yz0xqPARnAC">19</a>]&nbsp;

<div class="modal fade" id="bib-wang2024lgnet" role="dialog"><div class="modal-dialog"><div class="modal-content"><div class="modal-header"><button type="button" class="close" data-dismiss="modal">&times;</button><h4 class="modal-title">Bibtex</h4></div><div class="modal-body">
<pre>@article{wang2024lgnet,
 author = {Yuwen Wang and Mingxiu Han and Yudan Peng and Ruoqi Zhao and Dongqiong Fan and Xia Meng and Hong Xu and Haijun Niu and Jian Cheng and Tao Liu},
 doi = {10.1016/j.bspc.2024.106046},
 journal = {Biomedical Signal Processing and Control},
 pages = {106046},
 publisher = {Elsevier},
 title = {LGNet: Learning local--global EEG representations for cognitive workload classification in simulated flights},
 url = {https://dx.doi.org/10.1016/j.bspc.2024.106046},
 volume = {92},
 year = {2024}
}</pre> </div><div class="modal-footer"><button type="button" class="btn btn-default" data-dismiss="modal">Close</button></div></div></div></div>
<div class="modal fade" id="abs-wang2024lgnet" role="dialog"><div class="modal-dialog"><div class="modal-content"><div class="modal-header"><button type="button" class="close" data-dismiss="modal">&times;</button><h4 class="modal-title">Abstract</h4></div><div class="modal-body">
<pre>Cognitive workload assessment is crucial for ensuring pilots' safety
during flights. Electroencephalography (EEG) is a promising tool for
monitoring cognitive workload. Convolutional neural networks (CNNs)
are effective in automatically extracting local EEG representations.
However, CNNs have limitations in global representations, because a
global representation in CNNs normally requires CNNs to be deep
enough to have a global receptive field, which normally results in
overfitting. To address this issue, we propose a local and global
network (LGNet) for assessing two levels of cognitive workload based
on EEG during simulated flight. We fuse convolutional and
Transformer layers to extract local and global representations from
the EEG signals. To enhance the learning performance, we propose a
novel SCCE loss, which combines the supervised contrastive loss with
the traditional cross-entropy loss. We collect 32-channel EEG data
from 10 subjects who perform low and high cognitive workload flight
tasks with passive auditory stimuli in a flight simulator on 3
separate days, with each participant performing the task for
approximately 345 min. The results show that LGNet with the SCCE
loss achieves a 4-fold average classification accuracy of 91.19 %
based on cross-clip data partitioning and an average classification
accuracy of 83.26 % based on cross-session data partitioning when no
target session data is added to the training data. These results
significantly outperform classifiers based on handcrafted features
and state-of-the-art deep learning methods.</pre> </div><div class="modal-footer"><button type="button" class="btn btn-default" data-dismiss="modal">Close</button></div></div></div></div>
</li><br>

<li>
<span class="author">Yuwen Wang#, Yudan Peng#, Mingxiu Han, Xinyi Liu, Haijun Niu, <b>Jian Cheng</b>*, Suhua Chang*, Tao Liu*</span>,<br>
<span class="title">"GCTNet: a graph convolutional transformer network for major depressive disorder detection based on EEG signals"</span>,<br>
<span class="publisher">Journal of Neural Engineering</span>, vol. 21, no. 3, pp. 036042, jun, <span class="year">2024</span>.
<br>
<a target="_self" href="http://iopscience.iop.org/article/10.1088/1741-2552/ad5048"><img src="Icons/www.gif" alt="[www]" style="width: 16px; height: 16px;"></a>&nbsp;
[<a type="button" data-toggle="modal" data-target="#bib-Wang_JNE2024">bibtex</a>]&nbsp;
[<a type="button" data-toggle="modal" data-target="#abs-Wang_JNE2024">abstract</a>]&nbsp;
[citations: <a target="_blank" href="https://scholar.google.com//citations?view_op=view_citation&hl=en&oe=ASCII&user=BARqXQ0AAAAJ&pagesize=100&sortby=pubdate&citation_for_view=BARqXQ0AAAAJ:0CzhzZyukY4C">16</a>]&nbsp;

<div class="modal fade" id="bib-Wang_JNE2024" role="dialog"><div class="modal-dialog"><div class="modal-content"><div class="modal-header"><button type="button" class="close" data-dismiss="modal">&times;</button><h4 class="modal-title">Bibtex</h4></div><div class="modal-body">
<pre>@article{Wang_JNE2024,
 author = {Yuwen Wang and Yudan Peng and Mingxiu Han and Xinyi Liu and Haijun Niu and Jian Cheng and Suhua Chang and Tao Liu},
 doi = {10.1088/1741-2552/ad5048},
 journal = {Journal of Neural Engineering},
 month = {jun},
 number = {3},
 pages = {036042},
 publisher = {IOP Publishing},
 title = {GCTNet: a graph convolutional transformer network for major depressive disorder detection based on EEG signals},
 url = {http://iopscience.iop.org/article/10.1088/1741-2552/ad5048},
 volume = {21},
 year = {2024}
}</pre> </div><div class="modal-footer"><button type="button" class="btn btn-default" data-dismiss="modal">Close</button></div></div></div></div>
<div class="modal fade" id="abs-Wang_JNE2024" role="dialog"><div class="modal-dialog"><div class="modal-content"><div class="modal-header"><button type="button" class="close" data-dismiss="modal">&times;</button><h4 class="modal-title">Abstract</h4></div><div class="modal-body">
<pre>Objective. Identifying major depressive disorder (MDD) using
objective physiological signals has become a pressing challenge.
Approach. Hence, this paper proposes a graph convolutional
transformer network (GCTNet) for accurate and reliable MDD detection
using electroencephalogram (EEG) signals. The developed framework
integrates a residual graph convolutional network (ResGCN) block to
capture spatial information and a Transformer block to extract
global temporal dynamics. Additionally, we introduce the contrastive
cross-entropy (CCE) loss that combines contrastive learning to
enhance the stability and discriminability of the extracted
features, thereby improving classification performance. Main
results. The effectiveness of the GCTNet model and CCE loss was
assessed using EEG data from 41 MDD patients and 44 normal controls
(NC), in addition to a publicly available dataset. Utilizing a
subject-independent data partitioning method and 10-fold cross-
validation, the proposed method demonstrated significant
performance, achieving an average Area Under the Curve (AUC) of
0.7693 and 0.9755 across both datasets, respectively. Comparative
analyses demonstrated the superiority of the GCTNet framework with
CCE loss over state-of-the-art algorithms in MDD detection tasks.
Significance. The proposed method offers an objective and effective
approach to MDD detection, providing valuable support for clinical-
assisted diagnosis.</pre> </div><div class="modal-footer"><button type="button" class="btn btn-default" data-dismiss="modal">Close</button></div></div></div></div>
</li><br>

<li>
<span class="author">Keyi He, Bo Peng, Weibo Yu, Yan Liu, Surui Liu, <b>Jian Cheng</b>*, Yakang Dai*</span>,<br>
<span class="title">"A Novel Mis-Seg-Focus Loss Function Based on a Two-Stage nnU-Net Framework for Accurate Brain Tissue Segmentation"</span>,<br>
<span class="publisher">Bioengineering</span>, vol. 11, no. 5, pp. 427, <span class="year">2024</span>.
<br>
<a target="_self" href="https://dx.doi.org/10.3390/bioengineering11050427"><img src="Icons/www.gif" alt="[www]" style="width: 16px; height: 16px;"></a>&nbsp;
[<a type="button" data-toggle="modal" data-target="#bib-he2024novel">bibtex</a>]&nbsp;
[<a type="button" data-toggle="modal" data-target="#abs-he2024novel">abstract</a>]&nbsp;
[citations: <a target="_blank" href="https://scholar.google.com//citations?view_op=view_citation&hl=en&oe=ASCII&user=BARqXQ0AAAAJ&pagesize=100&sortby=pubdate&citation_for_view=BARqXQ0AAAAJ:NyGDZy8z5eUC">1</a>]&nbsp;

<div class="modal fade" id="bib-he2024novel" role="dialog"><div class="modal-dialog"><div class="modal-content"><div class="modal-header"><button type="button" class="close" data-dismiss="modal">&times;</button><h4 class="modal-title">Bibtex</h4></div><div class="modal-body">
<pre>@article{he2024novel,
 author = {Keyi He and Bo Peng and Weibo Yu and Yan Liu and Surui Liu and Jian Cheng and Yakang Dai},
 doi = {10.3390/bioengineering11050427},
 journal = {Bioengineering},
 number = {5},
 pages = {427},
 publisher = {MDPI},
 title = {A Novel Mis-Seg-Focus Loss Function Based on a Two-Stage nnU-Net Framework for Accurate Brain Tissue Segmentation},
 url = {https://dx.doi.org/10.3390/bioengineering11050427},
 volume = {11},
 year = {2024}
}</pre> </div><div class="modal-footer"><button type="button" class="btn btn-default" data-dismiss="modal">Close</button></div></div></div></div>
<div class="modal fade" id="abs-he2024novel" role="dialog"><div class="modal-dialog"><div class="modal-content"><div class="modal-header"><button type="button" class="close" data-dismiss="modal">&times;</button><h4 class="modal-title">Abstract</h4></div><div class="modal-body">
<pre>Brain tissue segmentation plays a critical role in the diagnosis,
treatment, and study of brain diseases. Accurately identifying these
boundaries is essential for improving segmentation accuracy.
However, distinguishing boundaries between different brain tissues
can be challenging, as they often overlap. Existing deep learning
methods primarily calculate the overall segmentation results without
adequately addressing local regions, leading to error propagation
and mis-segmentation along boundaries. In this study, we propose a
novel mis-segmentation-focused loss function based on a two-stage
nnU-Net framework. Our approach aims to enhance the model’s ability
to handle ambiguous boundaries and overlapping anatomical
structures, thereby achieving more accurate brain tissue
segmentation results. Specifically, the first stage targets the
identification of mis-segmentation regions using a global loss
function, while the second stage involves defining a mis-
segmentation loss function to adaptively adjust the model, thus
improving its capability to handle ambiguous boundaries and
overlapping anatomical structures. Experimental evaluations on two
datasets demonstrate that our proposed method outperforms existing
approaches both quantitatively and qualitatively.</pre> </div><div class="modal-footer"><button type="button" class="btn btn-default" data-dismiss="modal">Close</button></div></div></div></div>
</li><br>

<li>
<span class="author">Ximing Nie, Jinxu Yang, Xinxin Li, Tianming Zhan, Dongdong Liu, Hongyi Yan, Yufei Wei, Xiran Liu, Jiaping Chen, Guoyang Gong, Zhenzhou Wu, Zhonghua Yang, Miao Wen, Weibin Gu, Yuesong Pan, Yong Jiang, Xia Meng, Tao Liu, <b>Jian Cheng</b>, Zixiao Li, Zhongrong Miao, Liping Liu*</span>,<br>
<span class="title">"Prediction of futile recanalisation after endovascular treatment in acute ischaemic stroke: development and validation of a hybrid machine learning model"</span>,<br>
<span class="publisher">Stroke and Vascular Neurology</span>, pp. svn&ndash;2023, <span class="year">2024</span>.
<br>
<a target="_self" href="https://dx.doi.org/10.1136/svn-2023-002500"><img src="Icons/www.gif" alt="[www]" style="width: 16px; height: 16px;"></a>&nbsp;
[<a type="button" data-toggle="modal" data-target="#bib-nie2024prediction">bibtex</a>]&nbsp;
[<a type="button" data-toggle="modal" data-target="#abs-nie2024prediction">abstract</a>]&nbsp;
[citations: <a target="_blank" href="https://scholar.google.com//citations?view_op=view_citation&hl=en&oe=ASCII&user=BARqXQ0AAAAJ&pagesize=100&sortby=pubdate&citation_for_view=BARqXQ0AAAAJ:sJsF-0ZLhtgC">5</a>]&nbsp;

<div class="modal fade" id="bib-nie2024prediction" role="dialog"><div class="modal-dialog"><div class="modal-content"><div class="modal-header"><button type="button" class="close" data-dismiss="modal">&times;</button><h4 class="modal-title">Bibtex</h4></div><div class="modal-body">
<pre>@article{nie2024prediction,
 author = {Ximing Nie and Jinxu Yang and Xinxin Li and Tianming Zhan and Dongdong Liu and Hongyi Yan and Yufei Wei and Xiran Liu and Jiaping Chen and Guoyang Gong and Zhenzhou Wu and Zhonghua Yang and Miao Wen and Weibin Gu and Yuesong Pan and Yong Jiang and Xia Meng and Tao Liu and Jian Cheng and Zixiao Li and Zhongrong Miao and Liping Liu},
 doi = {10.1136/svn-2023-002500},
 journal = {Stroke and Vascular Neurology},
 pages = {svn-2023},
 publisher = {BMJ Specialist Journals},
 title = {Prediction of futile recanalisation after endovascular treatment in acute ischaemic stroke: development and validation of a hybrid machine learning model},
 url = {https://dx.doi.org/10.1136/svn-2023-002500},
 year = {2024}
}</pre> </div><div class="modal-footer"><button type="button" class="btn btn-default" data-dismiss="modal">Close</button></div></div></div></div>
<div class="modal fade" id="abs-nie2024prediction" role="dialog"><div class="modal-dialog"><div class="modal-content"><div class="modal-header"><button type="button" class="close" data-dismiss="modal">&times;</button><h4 class="modal-title">Abstract</h4></div><div class="modal-body">
<pre>Background Identification of futile recanalisation following
endovascular therapy (EVT) in patients with acute ischaemic stroke
is both crucial and challenging. Here, we present a novel risk
stratification system based on hybrid machine learning method for
predicting futile recanalisation.  Methods Hybrid machine learning
models were developed to address six clinical scenarios within the
EVT and perioperative management workflow. These models were trained
on a prospective database using hybrid feature selection technique
to predict futile recanalisation following EVT. The optimal model
was validated and compared with existing models and scoring systems
in a multicentre prospective cohort to develop a hybrid machine
learning-based risk stratification system for futile recanalisation
prediction.  Results Using a hybrid feature selection approach, we
trained and tested multiple classifiers on two independent patient
cohorts (n=1122) to develop a hybrid machine learning-based
prediction model. The model demonstrated superior discriminative
ability compared with other models and scoring systems (area under
the curve=0.80, 95% CI 0.73 to 0.87) and was transformed into a web
application (RESCUE-FR Index) that provides a risk stratification
system for individual prediction (accessible online at fr-
index.biomind.cn/RESCUE-FR/).  Conclusions The proposed hybrid
machine learning approach could be used as an individualised risk
prediction model to facilitate adherence to clinical practice
guidelines and shared decision-making for optimal candidate
selection and prognosis assessment in patients undergoing EVT.</pre> </div><div class="modal-footer"><button type="button" class="btn btn-default" data-dismiss="modal">Close</button></div></div></div></div>
</li><br>

<li>
<span class="author">Luyao Luo, Pan Liu, Wanxing Ye, Fengwei Chen, Yu Liu, Ziyang Liu, Jing Jing, Yunyun Xiong, Wanlin Zhu, Yong Jiang, <b>Jian Cheng</b>, Yongjun Wang, Tao Liu</span>,<br>
<span class="title">"CT perfusion parameter estimation in stroke using neural network with transformer and physical model priors"</span>,<br>
<span class="publisher">Computers in Biology and Medicine</span>, vol. 182, pp. 109134, <span class="year">2024</span>.
<br>
<a target="_self" href="https://dx.doi.org/10.1016/j.compbiomed.2024.109134"><img src="Icons/www.gif" alt="[www]" style="width: 16px; height: 16px;"></a>&nbsp;
[<a type="button" data-toggle="modal" data-target="#bib-luo2024ct">bibtex</a>]&nbsp;
[<a type="button" data-toggle="modal" data-target="#abs-luo2024ct">abstract</a>]&nbsp;
[citations: <a target="_blank" href="https://scholar.google.com//citations?view_op=view_citation&hl=en&oe=ASCII&user=BARqXQ0AAAAJ&pagesize=100&sortby=pubdate&citation_for_view=BARqXQ0AAAAJ:jL-93Qbq4QoC">1</a>]&nbsp;

<div class="modal fade" id="bib-luo2024ct" role="dialog"><div class="modal-dialog"><div class="modal-content"><div class="modal-header"><button type="button" class="close" data-dismiss="modal">&times;</button><h4 class="modal-title">Bibtex</h4></div><div class="modal-body">
<pre>@article{luo2024ct,
 author = {Luyao Luo and Pan Liu and Wanxing Ye and Fengwei Chen and Yu Liu and Ziyang Liu and Jing Jing and Yunyun Xiong and Wanlin Zhu and Yong Jiang and Jian Cheng and Yongjun Wang and Tao Liu},
 doi = {10.1016/j.compbiomed.2024.109134},
 journal = {Computers in Biology and Medicine},
 pages = {109134},
 publisher = {Elsevier},
 title = {CT perfusion parameter estimation in stroke using neural network with transformer and physical model priors},
 url = {https://dx.doi.org/10.1016/j.compbiomed.2024.109134},
 volume = {182},
 year = {2024}
}</pre> </div><div class="modal-footer"><button type="button" class="btn btn-default" data-dismiss="modal">Close</button></div></div></div></div>
<div class="modal fade" id="abs-luo2024ct" role="dialog"><div class="modal-dialog"><div class="modal-content"><div class="modal-header"><button type="button" class="close" data-dismiss="modal">&times;</button><h4 class="modal-title">Abstract</h4></div><div class="modal-body">
<pre>Objectives CT perfusion (CTP) imaging is vital in treating acute
ischemic stroke by identifying salvageable tissue and the infarcted
core. CTP images allow quantitative estimation of CT perfusion
parameters, which can provide information on the degree of tissue
hypoperfusion and its salvage potential. Traditional methods for
estimating perfusion parameters, such as singular value
decomposition (SVD) and its variations, are known to be sensitive to
noise and inaccuracies in the arterial input function. To our
knowledge, there has been no implementation of deep learning methods
for CT perfusion parameter estimation. Materials & methods In this
work, we propose a deep learning method based on the Transformer
model, named CTPerformer-Net, for CT perfusion parameter estimation.
In addition, our method incorporates some physical priors. We
integrate physical consistency prior, smoothness prior and the
physical model prior through the design of the loss function. We
also generate a simulation dataset based on physical model prior for
training the network model. Results In the simulation dataset,
CTPerformer-Net exhibits a 23.4 % increase in correlation
coefficients, a 95.2 % decrease in system error, and a 90.7 %
reduction in random error when contrasted with block-circulant SVD.
CTPerformer-Net successfully identifies hypoperfused and infarcted
lesions in 103 real CTP images from the ISLES 2018 challenge
dataset. It achieves a mean dice score of 0.36 for the infarct core
segmentation, which is slightly higher than the commercially
available software (dice coefficient: 0.34) used as a reference
level by the challenge. Conclusion Experimental results on the
simulation dataset demonstrate that CTPerformer-Net achieves better
performance compared to block-circulant SVD. The real-world patient
dataset confirms the validity of CTPerformer-Net.</pre> </div><div class="modal-footer"><button type="button" class="btn btn-default" data-dismiss="modal">Close</button></div></div></div></div>
</li><br>

<li>
<span class="author">Xiangyu Liu, Yingying Zhang, Haogang Zhu, Xu Yang, <b>Jian Cheng</b>, Jishuo Jin, Bosen Jia, Yihua He, Hongjia Zhang</span>,<br>
<span class="title">"Diagnosis of fetal arrhythmia in echocardiography imaging using deep learning with cyclic loss"</span>,<br>
<span class="publisher">Digital Health</span>, vol. 10, pp. 20552076241286929, <span class="year">2024</span>.
<br>
<a target="_self" href="https://dx.doi.org/10.1177/20552076241286929"><img src="Icons/www.gif" alt="[www]" style="width: 16px; height: 16px;"></a>&nbsp;
[<a type="button" data-toggle="modal" data-target="#bib-liu2024diagnosis">bibtex</a>]&nbsp;
[<a type="button" data-toggle="modal" data-target="#abs-liu2024diagnosis">abstract</a>]&nbsp;

<div class="modal fade" id="bib-liu2024diagnosis" role="dialog"><div class="modal-dialog"><div class="modal-content"><div class="modal-header"><button type="button" class="close" data-dismiss="modal">&times;</button><h4 class="modal-title">Bibtex</h4></div><div class="modal-body">
<pre>@article{liu2024diagnosis,
 author = {Xiangyu Liu and Yingying Zhang and Haogang Zhu and Xu Yang and Jian Cheng and Jishuo Jin and Bosen Jia and Yihua He and Hongjia Zhang},
 doi = {10.1177/20552076241286929},
 journal = {Digital Health},
 pages = {20552076241286929},
 publisher = {SAGE Publications Sage UK: London, England},
 title = {Diagnosis of fetal arrhythmia in echocardiography imaging using deep learning with cyclic loss},
 url = {https://dx.doi.org/10.1177/20552076241286929},
 volume = {10},
 year = {2024}
}</pre> </div><div class="modal-footer"><button type="button" class="btn btn-default" data-dismiss="modal">Close</button></div></div></div></div>
<div class="modal fade" id="abs-liu2024diagnosis" role="dialog"><div class="modal-dialog"><div class="modal-content"><div class="modal-header"><button type="button" class="close" data-dismiss="modal">&times;</button><h4 class="modal-title">Abstract</h4></div><div class="modal-body">
<pre>Background Fetal arrhythmias frequently co-occur with congenital
heart disease in fetuses. The peaks observed in M-mode fetal
echocardiograms serve as pivotal diagnostic markers for fetal
arrhythmias. However, speckles, artifacts, and noise pose notable
challenges for accurate image analysis. While current deep learning
networks mainly overlook cardiac cyclic information, this study
concentrated on the integration of such features, leveraging
contextual constraints derived from cardiac cyclical features to
improve diagnostic accuracy. Methods This study proposed a novel
deep learning architecture for diagnosing fetal arrhythmias. The
architecture presented a loss function tailored to the cardiac
cyclical information and formulated a diagnostic algorithm for
classifying fetal arrhythmias. The training and validation processes
utilized a dataset comprising 4440 patches gathered from 890
participants. Results Incorporating cyclic loss significantly
enhanced the performance of deep learning networks in predicting
peak points for diagnosing fetal arrhythmia, resulting in
improvements ranging from 7.11% to 14.81% in F1-score across
different network combinations. Particularly noteworthy was the
18.2% improvement in the F1-score for the low-quality group.
Additionally, the precision of diagnosing fetal arrhythmia across
four categories exhibited improvement, with an average improvement
rate of 20.6%. Conclusion This study introduced a cyclic loss
mechanism based on the cardiac cycle information. Comparative
evaluations were conducted using baseline methods and state-of-the-
art deep learning architectures with the fetal echocardiogram
dataset. These evaluations demonstrated the proposed framework’s
superior accuracy in diagnosing fetal arrhythmias. It is also
crucial to note that further external testing is essential to assess
the model’s generalizability and clinical value.</pre> </div><div class="modal-footer"><button type="button" class="btn btn-default" data-dismiss="modal">Close</button></div></div></div></div>
</li><br>

<li>
<span class="author">Si-Miao Zhang, Jing Wang, Yi-Xuan Wang, Tao Liu, Haogang Zhu, Han Zhang, <b>Jian Cheng</b>*</span>,<br>
<span class="title">"Mixed Integer Linear Programming for Discrete Sampling Scheme Design in Diffusion MRI"</span>,<br>
<span class="publisher">International Conference on Medical Image Computing and Computer-Assisted Intervention (<b>MICCAI</b>'24)</span>, pp. 313&ndash;322, <span class="year">2024</span>.
<br>
<a target="_self" href="https://dx.doi.org/10.1007/978-3-031-72069-7_30"><img src="Icons/www.gif" alt="[www]" style="width: 16px; height: 16px;"></a>&nbsp;
[<a type="button" data-toggle="modal" data-target="#bib-zhang-MICCAI2024mixed">bibtex</a>]&nbsp;

<div class="modal fade" id="bib-zhang-MICCAI2024mixed" role="dialog"><div class="modal-dialog"><div class="modal-content"><div class="modal-header"><button type="button" class="close" data-dismiss="modal">&times;</button><h4 class="modal-title">Bibtex</h4></div><div class="modal-body">
<pre>@inproceedings{zhang:MICCAI2024mixed,
 author = {Si-Miao Zhang and Jing Wang and Yi-Xuan Wang and Tao Liu and Haogang Zhu and Han Zhang and Jian Cheng},
 booktitle = {International Conference on Medical Image Computing and Computer-Assisted Intervention (MICCAI'24)},
 doi = {10.1007/978-3-031-72069-7_30},
 organization = {Springer},
 pages = {313-322},
 title = {Mixed Integer Linear Programming for Discrete Sampling Scheme Design in Diffusion MRI},
 url = {https://dx.doi.org/10.1007/978-3-031-72069-7_30},
 year = {2024}
}</pre> </div><div class="modal-footer"><button type="button" class="btn btn-default" data-dismiss="modal">Close</button></div></div></div></div>
</li><br>

<li>
<span class="author">Mingkun Bao, Yan Wang, Xinlong Wei, Bosen Jia, Xiaolin Fan, Dong Lu, Yifan Gu, <b>Jian Cheng</b>, Yingying Zhang*, Chuanyu Wang*, Haogang Zhu*</span>,<br>
<span class="title">"Real-World Visual Navigation for Cardiac Ultrasound View Planning"</span>,<br>
<span class="publisher">International Conference on Medical Image Computing and Computer-Assisted Intervention (<b>MICCAI</b>'24)</span>, pp. 317&ndash;326, <span class="year">2024</span>.
<br>
[<a type="button" data-toggle="modal" data-target="#bib-bao-MICCAI2024real">bibtex</a>]&nbsp;
[citations: <a target="_blank" href="https://scholar.google.com//citations?view_op=view_citation&hl=en&oe=ASCII&user=BARqXQ0AAAAJ&pagesize=100&sortby=pubdate&citation_for_view=BARqXQ0AAAAJ:1lhNe0rCu4AC">4</a>]&nbsp;

<div class="modal fade" id="bib-bao-MICCAI2024real" role="dialog"><div class="modal-dialog"><div class="modal-content"><div class="modal-header"><button type="button" class="close" data-dismiss="modal">&times;</button><h4 class="modal-title">Bibtex</h4></div><div class="modal-body">
<pre>@inproceedings{bao:MICCAI2024real,
 author = {Mingkun Bao and Yan Wang and Xinlong Wei and Bosen Jia and Xiaolin Fan and Dong Lu and Yifan Gu and Jian Cheng and Yingying Zhang and Chuanyu Wang and Haogang Zhu},
 booktitle = {International Conference on Medical Image Computing and Computer-Assisted Intervention (MICCAI'24)},
 organization = {Springer},
 pages = {317-326},
 title = {Real-World Visual Navigation for Cardiac Ultrasound View Planning},
 year = {2024}
}</pre> </div><div class="modal-footer"><button type="button" class="btn btn-default" data-dismiss="modal">Close</button></div></div></div></div>
</li><br>

<li>
<span class="author">Jing Yang, <b>Jian Cheng</b>*, Cheng Li, Wenxin Fan, Juan Zou, Ruoyou Wu, Shanshan Wang*</span>,<br>
<span class="title">"Simultaneous q-Space Sampling Optimization and Reconstruction for Fast and High-fidelity Diffusion Magnetic Resonance Imaging"</span>,<br>
<span class="publisher">IEEE International Symposium on Biomedical Imaging: From Nano to Macro (ISBI'24)</span>, <span class="year">2024</span>.
<br><a target="_self" href="https://arxiv.org/pdf/2401.01662.pdf"><img src="Icons/pdf.gif" alt="[pdf]" style="width: 16px; height: 16px;"></a>&nbsp;
<a target="_self" href="https://arxiv.org/abs/2401.01662"><img src="Icons/www.gif" alt="[www]" style="width: 16px; height: 16px;"></a>&nbsp;
[<a type="button" data-toggle="modal" data-target="#bib-yang2024simultaneous">bibtex</a>]&nbsp;
[<a type="button" data-toggle="modal" data-target="#abs-yang2024simultaneous">abstract</a>]&nbsp;
[<a target="_self" href="https://arxiv.org/abs/2401.01662">arxiv</a>]&nbsp;
[citations: <a target="_blank" href="https://scholar.google.com//citations?view_op=view_citation&hl=en&oe=ASCII&user=BARqXQ0AAAAJ&pagesize=100&sortby=pubdate&citation_for_view=BARqXQ0AAAAJ:AHdEip9mkN0C">2</a>]&nbsp;

<div class="modal fade" id="bib-yang2024simultaneous" role="dialog"><div class="modal-dialog"><div class="modal-content"><div class="modal-header"><button type="button" class="close" data-dismiss="modal">&times;</button><h4 class="modal-title">Bibtex</h4></div><div class="modal-body">
<pre>@inproceedings{yang2024simultaneous,
 author = {Jing Yang and Jian Cheng and Cheng Li and Wenxin Fan and Juan Zou and Ruoyou Wu and Shanshan Wang},
 booktitle = {IEEE International Symposium on Biomedical Imaging: From Nano to Macro (ISBI'24)},
 pdf = {https://arxiv.org/pdf/2401.01662.pdf},
 title = {Simultaneous q-Space Sampling Optimization and Reconstruction for Fast and High-fidelity Diffusion Magnetic Resonance Imaging},
 url = {https://arxiv.org/abs/2401.01662},
 year = {2024}
}</pre> </div><div class="modal-footer"><button type="button" class="btn btn-default" data-dismiss="modal">Close</button></div></div></div></div>
<div class="modal fade" id="abs-yang2024simultaneous" role="dialog"><div class="modal-dialog"><div class="modal-content"><div class="modal-header"><button type="button" class="close" data-dismiss="modal">&times;</button><h4 class="modal-title">Abstract</h4></div><div class="modal-body">
<pre>Diffusion Magnetic Resonance Imaging (dMRI) plays a crucial role in
the noninvasive investigation of tissue microstructural properties
and structural connectivity in the in vivo human brain. However, to
effectively capture the intricate characteristics of water diffusion
at various directions and scales, it is important to employ
comprehensive q-space sampling. Unfortunately, this requirement
leads to long scan times, limiting the clinical applicability of
dMRI. To address this challenge, we propose SSOR, a Simultaneous
q-Space sampling Optimization and Reconstruction framework. We
jointly optimize a subset of q-space samples using a continuous
representation of spherical harmonic functions and a reconstruction
network. Additionally, we integrate the unique properties of
diffusion magnetic resonance imaging (dMRI) in both the q-space and
image domains by applying l1-norm and total-variation
regularization.The experiments conducted on HCP data demonstrate
that SSOR has promising strengths both quantitatively and
qualitatively and exhibits robustness to noise</pre> </div><div class="modal-footer"><button type="button" class="btn btn-default" data-dismiss="modal">Close</button></div></div></div></div>
</li><br>

<li>
<span class="author">Ruoyou Wu, <b>Jian Cheng</b>, Cheng Li, Juan Zou, Jing Yang, Wenxin Fan, Yong Liang, Shanshan Wang</span>,<br>
<span class="title">"CSR-dMRI: Continuous Super-Resolution of Diffusion MRI with Anatomical Structure-Assisted Implicit Neural Representation Learning"</span>,<br>
<span class="publisher">MICCAI Workshop on Machine Learning in Medical Imaging (MLMI)</span>, pp. 114&ndash;123, <span class="year">2024</span>.
<br><a target="_self" href="https://arxiv.org/pdf/2404.03209.pdf"><img src="Icons/pdf.gif" alt="[pdf]" style="width: 16px; height: 16px;"></a>&nbsp;
<a target="_self" href="https://dx.doi.org/10.1007/978-3-031-73284-3_12"><img src="Icons/www.gif" alt="[www]" style="width: 16px; height: 16px;"></a>&nbsp;
[<a type="button" data-toggle="modal" data-target="#bib-wu2024csr">bibtex</a>]&nbsp;
[<a type="button" data-toggle="modal" data-target="#abs-wu2024csr">abstract</a>]&nbsp;
[<a target="_self" href="https://arxiv.org/abs/2404.03209">arxiv</a>]&nbsp;
[citations: <a target="_blank" href="https://scholar.google.com//citations?view_op=view_citation&hl=en&oe=ASCII&user=BARqXQ0AAAAJ&pagesize=100&sortby=pubdate&citation_for_view=BARqXQ0AAAAJ:PVgj2kMGcgYC">5</a>]&nbsp;

<div class="modal fade" id="bib-wu2024csr" role="dialog"><div class="modal-dialog"><div class="modal-content"><div class="modal-header"><button type="button" class="close" data-dismiss="modal">&times;</button><h4 class="modal-title">Bibtex</h4></div><div class="modal-body">
<pre>@inproceedings{wu2024csr,
 author = {Ruoyou Wu and Jian Cheng and Cheng Li and Juan Zou and Jing Yang and Wenxin Fan and Yong Liang and Shanshan Wang},
 booktitle = {MICCAI Workshop on Machine Learning in Medical Imaging (MLMI)},
 doi = {10.1007/978-3-031-73284-3_12},
 organization = {Springer},
 pages = {114-123},
 pdf = {https://arxiv.org/pdf/2404.03209.pdf},
 title = {CSR-dMRI: Continuous Super-Resolution of Diffusion MRI with Anatomical Structure-Assisted Implicit Neural Representation Learning},
 url = {https://dx.doi.org/10.1007/978-3-031-73284-3_12},
 year = {2024}
}</pre> </div><div class="modal-footer"><button type="button" class="btn btn-default" data-dismiss="modal">Close</button></div></div></div></div>
<div class="modal fade" id="abs-wu2024csr" role="dialog"><div class="modal-dialog"><div class="modal-content"><div class="modal-header"><button type="button" class="close" data-dismiss="modal">&times;</button><h4 class="modal-title">Abstract</h4></div><div class="modal-body">
<pre>Deep learning-based dMRI super-resolution methods can effectively
enhance image resolution by leveraging the learning capabilities of
neural networks on large datasets. However, these methods tend to
learn a fixed scale mapping between low-resolution (LR) and high-
resolution (HR) images, overlooking the need for radiologists to
scale the images at arbitrary resolutions. Moreover, the pixel-wise
loss in the image domain tends to generate over-smoothed results,
losing fine textures and edge information. To address these issues,
we propose a novel continuous super-resolution method for dMRI,
called CSR-dMRI, which utilizes an anatomical structure-assisted
implicit neural representation learning approach. Specifically, the
CSR-dMRI model consists of two components. The first is the latent
feature extractor, which primarily extracts latent space feature
maps from LR dMRI and anatomical images while learning structural
prior information from the anatomical images. The second is the
implicit function network, which utilizes voxel coordinates and
latent feature vectors to generate voxel intensities at
corresponding positions. Additionally, a frequency-domain-based loss
is introduced to preserve the structural and texture information,
further enhancing the image quality. Extensive experiments on the
publicly available HCP dataset validate the effectiveness of our
approach. Furthermore, our method demonstrates superior
generalization capability and can be applied to arbitrary-scale
super-resolution, including non-integer scale factors, expanding its
applicability beyond conventional approaches.</pre> </div><div class="modal-footer"><button type="button" class="btn btn-default" data-dismiss="modal">Close</button></div></div></div></div>
</li><br>

</ol><h2><a name="year2023"></a>2023</h2>

<ol reversed>

<li>
<span class="author">Yan Wang, <b>Jian Cheng</b>*, Yixin Chen, Shuai Shao, Lanyun Zhu, Zhenzhou Wu, Tao Liu, Haogang Zhu*</span>,<br>
<span class="title">"FVP: Fourier Visual Prompting for Source-Free Unsupervised Domain Adaptation of Medical Image Segmentation"</span>,<br>
<span class="publisher">IEEE Transactions on Medical Imaging (<b>TMI</b>)</span>, vol. 42, no. 12, pp. 3738&ndash;3751, <span class="year">2023</span>.
<br><a target="_self" href="https://arxiv.org/pdf/2304.13672.pdf"><img src="Icons/pdf.gif" alt="[pdf]" style="width: 16px; height: 16px;"></a>&nbsp;
<a target="_self" href="https://dx.doi.org/10.1109/TMI.2023.3306105"><img src="Icons/www.gif" alt="[www]" style="width: 16px; height: 16px;"></a>&nbsp;
[<a type="button" data-toggle="modal" data-target="#bib-wang2023fvp">bibtex</a>]&nbsp;
[<a type="button" data-toggle="modal" data-target="#abs-wang2023fvp">abstract</a>]&nbsp;
[<a target="_self" href="https://arxiv.org/abs/2304.13672">arxiv</a>]&nbsp;
[citations: <a target="_blank" href="https://scholar.google.com//citations?view_op=view_citation&hl=en&oe=ASCII&user=BARqXQ0AAAAJ&pagesize=100&sortby=pubdate&citation_for_view=BARqXQ0AAAAJ:nVrZBo8bIpAC">56</a>]&nbsp;

<div class="modal fade" id="bib-wang2023fvp" role="dialog"><div class="modal-dialog"><div class="modal-content"><div class="modal-header"><button type="button" class="close" data-dismiss="modal">&times;</button><h4 class="modal-title">Bibtex</h4></div><div class="modal-body">
<pre>@article{wang2023fvp,
 author = {Yan Wang and Jian Cheng and Yixin Chen and Shuai Shao and Lanyun Zhu and Zhenzhou Wu and Tao Liu and Haogang Zhu},
 doi = {10.1109/TMI.2023.3306105},
 journal = {IEEE Transactions on Medical Imaging},
 number = {12},
 pages = {3738-3751},
 pdf = {https://arxiv.org/pdf/2304.13672.pdf},
 title = {FVP: Fourier Visual Prompting for Source-Free Unsupervised Domain Adaptation of Medical Image Segmentation},
 url = {https://dx.doi.org/10.1109/TMI.2023.3306105},
 volume = {42},
 year = {2023}
}</pre> </div><div class="modal-footer"><button type="button" class="btn btn-default" data-dismiss="modal">Close</button></div></div></div></div>
<div class="modal fade" id="abs-wang2023fvp" role="dialog"><div class="modal-dialog"><div class="modal-content"><div class="modal-header"><button type="button" class="close" data-dismiss="modal">&times;</button><h4 class="modal-title">Abstract</h4></div><div class="modal-body">
<pre>Medical image segmentation methods normally perform poorly when
there is a domain shift between training and testing data.
Unsupervised Domain Adaptation (UDA) addresses the domain shift
problem by training the model using both labeled data from the
source domain and unlabeled data from the target domain. Source-Free
UDA (SFUDA) was recently proposed for UDA without requiring the
source data during the adaptation, due to data privacy or data
transmission issues, which normally adapts the pre-trained deep
model in the testing stage. However, in real clinical scenarios of
medical image segmentation, the trained model is normally frozen in
the testing stage. In this paper, we propose Fourier Visual
Prompting (FVP) for SFUDA of medical image segmentation. Inspired by
prompting learning in natural language processing, FVP steers the
frozen pre-trained model to perform well in the target domain by
adding a visual prompt to the input target data. In FVP, the visual
prompt is parameterized using only a small amount of low-frequency
learnable parameters in the input frequency space, and is learned by
minimizing the segmentation loss between the predicted segmentation
of the prompted target image and reliable pseudo segmentation label
of the target image under the frozen model. To our knowledge, FVP is
the first work to apply visual prompts to SFUDA for medical image
segmentation. The proposed FVP is validated using three public
datasets, and experiments demonstrate that FVP yields better
segmentation results, compared with various existing methods.</pre> </div><div class="modal-footer"><button type="button" class="btn btn-default" data-dismiss="modal">Close</button></div></div></div></div>
</li><br>

<li>
<span class="author">Haichao Zhao, Wei Wen, <b>Jian Cheng</b>*, Jiyang Jiang, Nicole Kochan, Haijun Niu, Henry Brodaty, Perminder Sachdev, Tao Liu*</span>,<br>
<span class="title">"An accelerated degeneration of white matter microstructure and networks in the nondemented old–old"</span>,<br>
<span class="publisher">Cerebral Cortex (<b>CC</b>)</span>, vol. 33, no. 8, pp. 4688&ndash;4698, <span class="year">2023</span>.
<br><a target="_self" href="https://academic.oup.com/cercor/advance-article-pdf/doi/10.1093/cercor/bhac372/46212258/bhac372.pdf"><img src="Icons/pdf.gif" alt="[pdf]" style="width: 16px; height: 16px;"></a>&nbsp;
<a target="_self" href="https://doi.org/10.1093/cercor/bhac372"><img src="Icons/www.gif" alt="[www]" style="width: 16px; height: 16px;"></a>&nbsp;
[<a type="button" data-toggle="modal" data-target="#bib-zhao-DFA-CC2022">bibtex</a>]&nbsp;
[<a type="button" data-toggle="modal" data-target="#abs-zhao-DFA-CC2022">abstract</a>]&nbsp;
[citations: <a target="_blank" href="https://scholar.google.com//citations?view_op=view_citation&hl=en&oe=ASCII&user=BARqXQ0AAAAJ&pagesize=100&sortby=pubdate&citation_for_view=BARqXQ0AAAAJ:Aul-kAQHnToC">9</a>]&nbsp;

<div class="modal fade" id="bib-zhao-DFA-CC2022" role="dialog"><div class="modal-dialog"><div class="modal-content"><div class="modal-header"><button type="button" class="close" data-dismiss="modal">&times;</button><h4 class="modal-title">Bibtex</h4></div><div class="modal-body">
<pre>@article{zhao:DFA:CC2022,
 author = {Haichao Zhao and Wei Wen and Jian Cheng and Jiyang Jiang and Nicole Kochan and Haijun Niu and Henry Brodaty and Perminder Sachdev and Tao Liu},
 doi = {10.1093/cercor/bhac372},
 journal = {Cerebral Cortex},
 number = {8},
 pages = {4688-4698},
 pdf = {https://academic.oup.com/cercor/advance-article-pdf/doi/10.1093/cercor/bhac372/46212258/bhac372.pdf},
 publisher = {Oxford University Press},
 title = {An accelerated degeneration of white matter microstructure and networks in the nondemented old–old},
 url = {https://doi.org/10.1093/cercor/bhac372},
 volume = {33},
 year = {2023}
}</pre> </div><div class="modal-footer"><button type="button" class="btn btn-default" data-dismiss="modal">Close</button></div></div></div></div>
<div class="modal fade" id="abs-zhao-DFA-CC2022" role="dialog"><div class="modal-dialog"><div class="modal-content"><div class="modal-header"><button type="button" class="close" data-dismiss="modal">&times;</button><h4 class="modal-title">Abstract</h4></div><div class="modal-body">
<pre>{The nondemented old–old over the age of 80 comprise a rapidly
increasing population group; they can be regarded as exemplars of
successful aging. However, our current understanding of successful
aging in advanced age and its neural underpinnings is limited. In
this study, we measured the microstructural and network-based
topological properties of brain white matter using diffusion-
weighted imaging scans of 419 community-dwelling nondemented older
participants. The participants were further divided into 230
young–old (between 72 and 79, mean = 76.25 ± 2.00) and 219 old–old
(between 80 and 92, mean = 83.98 ± 2.97). Results showed that white
matter connectivity in microstructure and brain networks
significantly declined with increased age and that the declined
rates were faster in the old–old compared with young–old. Mediation
models indicated that cognitive decline was in part through the age
effect on the white matter connectivity in the old–old but not in
the young–old. Machine learning predictive models further supported
the crucial role of declines in white matter connectivity as a
neural substrate of cognitive aging in the nondemented older
population. Our findings shed new light on white matter connectivity
in the nondemented aging brains and may contribute to uncovering the
neural substrates of successful brain aging.}</pre> </div><div class="modal-footer"><button type="button" class="btn btn-default" data-dismiss="modal">Close</button></div></div></div></div>
</li><br>

<li>
<span class="author">Jing Jing, Chang Liu, Wanlin Zhu, Yuesong Pan, Jiyang Jiang, Xueli Cai, Zhe Zhang, Zixiao Li, Yijun Zhou, Xia Meng, <b>Jian Cheng</b>, Yilong Wang, Hao Li, Yong Jiang, Huaguang Zheng, Suying Wang, Haijun Niu, Wei Wen, Perminder S. Sachdev, Tiemin Wei, Tao Liu*, Yongjun Wang*</span>,<br>
<span class="title">"Increased Resting-State Functional Connectivity as a Compensatory Mechanism for Reduced Brain Volume in Prediabetes and Type 2 Diabetes"</span>,<br>
<span class="publisher"><b>Diabetes Care</b></span>, vol. 46, no. 4, pp. 819&ndash;827, 02, <span class="year">2023</span>.
<br>
<a target="_self" href="https://doi.org/10.2337/dc22-1998"><img src="Icons/www.gif" alt="[www]" style="width: 16px; height: 16px;"></a>&nbsp;
[<a type="button" data-toggle="modal" data-target="#bib-jing2023increased">bibtex</a>]&nbsp;
[<a type="button" data-toggle="modal" data-target="#abs-jing2023increased">abstract</a>]&nbsp;
[citations: <a target="_blank" href="https://scholar.google.com//citations?view_op=view_citation&hl=en&oe=ASCII&user=BARqXQ0AAAAJ&pagesize=100&sortby=pubdate&citation_for_view=BARqXQ0AAAAJ:lvd772isFD0C">28</a>]&nbsp;

<div class="modal fade" id="bib-jing2023increased" role="dialog"><div class="modal-dialog"><div class="modal-content"><div class="modal-header"><button type="button" class="close" data-dismiss="modal">&times;</button><h4 class="modal-title">Bibtex</h4></div><div class="modal-body">
<pre>@article{jing2023increased,
 author = {Jing Jing and Chang Liu and Wanlin Zhu and Yuesong Pan and Jiyang Jiang and Xueli Cai and Zhe Zhang and Zixiao Li and Yijun Zhou and Xia Meng and Jian Cheng and Yilong Wang and Hao Li and Yong Jiang and Huaguang Zheng and Suying Wang and Haijun Niu and Wei Wen and Perminder S. Sachdev and Tiemin Wei and Tao Liu and Yongjun Wang},
 doi = {10.2337/dc22-1998},
 journal = {Diabetes Care},
 month = {02},
 number = {4},
 pages = {819-827},
 publisher = {Am Diabetes Assoc},
 title = {Increased Resting-State Functional Connectivity as a Compensatory Mechanism for Reduced Brain Volume in Prediabetes and Type 2 Diabetes},
 url = {https://doi.org/10.2337/dc22-1998},
 volume = {46},
 year = {2023}
}</pre> </div><div class="modal-footer"><button type="button" class="btn btn-default" data-dismiss="modal">Close</button></div></div></div></div>
<div class="modal fade" id="abs-jing2023increased" role="dialog"><div class="modal-dialog"><div class="modal-content"><div class="modal-header"><button type="button" class="close" data-dismiss="modal">&times;</button><h4 class="modal-title">Abstract</h4></div><div class="modal-body">
<pre>{To investigate the contribution of alterations in brain structure
and function to cognitive function and their interactions in
individuals with diabetes and patients with type 2 diabetes mellitus
(T2DM).This population-based study included 2,483 participants who
underwent structural MRI (n = 569 with normal glucose metabolism
[NGM], n = 1,353 with prediabetes, and n = 561 with T2DM) and
cognitive testing. Of these, 2145 participants also underwent
functional MRI (n = 496 NGM, n = 1,170 prediabetes, and n = 479
T2DM). Multivariate linear regression models were used to assess the
association of brain volume and functional connectivity with
cognition, as well as the association of brain volume and functional
connectivity.Compared with NGM participants, those with T2DM had
lower brain volume in a wide range of brain regions and stronger
functional connectivity between the bilateral thalamus and brain
functional network (visual network and default mode network), and
those with prediabetes had lower brain volume in specific local
regions (subcortical gray matter volume and subcortical subregions
[bilateral thalamus, bilateral nucleus accumbens, and right
putamen]) and stronger functional connectivity between the right
thalamus and visual network. Cognition was associated with greater
right thalamus volume and lower functional connectivity between the
right thalamus and visual network. Functional connectivity between
the right thalamus and visual network was associated with lower
right thalamus volume.Cognition was associated with greater brain
volume and lower functional connectivity in T2DM. Increased
functional connectivity may indicate a compensatory mechanism for
reduced brain volume that begins in the prediabetic phase.}</pre> </div><div class="modal-footer"><button type="button" class="btn btn-default" data-dismiss="modal">Close</button></div></div></div></div>
</li><br>

<li>
<span class="author">Yingying Zhang, Haogang Zhu*, <b>Jian Cheng</b>, Jingyi Wang, Xiaoyan Gu, Jiancheng Han, Ye Zhang, Ying Zhao, Yihua He*, Hongjia Zhang*</span>,<br>
<span class="title">"Improving the Quality of Fetal Heart Ultrasound Imaging with Multihead Enhanced Self-Attention and Contrastive Learning"</span>,<br>
<span class="publisher">IEEE Journal of Biomedical and Health Informatics (<b>JBHI</b>)</span>, <span class="year">2023</span>.
<br><a target="_self" href="https://ieeexplore.ieee.org/stamp/stamp.jsp?tp=&arnumber=10213997"><img src="Icons/pdf.gif" alt="[pdf]" style="width: 16px; height: 16px;"></a>&nbsp;
<a target="_self" href="https://dx.doi.org/10.1109/JBHI.2023.3303573"><img src="Icons/www.gif" alt="[www]" style="width: 16px; height: 16px;"></a>&nbsp;
[<a type="button" data-toggle="modal" data-target="#bib-zhang2023improving">bibtex</a>]&nbsp;
[<a type="button" data-toggle="modal" data-target="#abs-zhang2023improving">abstract</a>]&nbsp;
[citations: <a target="_blank" href="https://scholar.google.com//citations?view_op=view_citation&hl=en&oe=ASCII&user=BARqXQ0AAAAJ&pagesize=100&sortby=pubdate&citation_for_view=BARqXQ0AAAAJ:5icHVeHT4IsC">16</a>]&nbsp;

<div class="modal fade" id="bib-zhang2023improving" role="dialog"><div class="modal-dialog"><div class="modal-content"><div class="modal-header"><button type="button" class="close" data-dismiss="modal">&times;</button><h4 class="modal-title">Bibtex</h4></div><div class="modal-body">
<pre>@article{zhang2023improving,
 author = {Yingying Zhang and Haogang Zhu and Jian Cheng and Jingyi Wang and Xiaoyan Gu and Jiancheng Han and Ye Zhang and Ying Zhao and Yihua He and Hongjia Zhang},
 doi = {10.1109/JBHI.2023.3303573},
 journal = {IEEE Journal of Biomedical and Health Informatics},
 pdf = {https://ieeexplore.ieee.org/stamp/stamp.jsp?tp=&arnumber=10213997},
 publisher = {IEEE},
 title = {Improving the Quality of Fetal Heart Ultrasound Imaging with Multihead Enhanced Self-Attention and Contrastive Learning},
 url = {https://dx.doi.org/10.1109/JBHI.2023.3303573},
 year = {2023}
}</pre> </div><div class="modal-footer"><button type="button" class="btn btn-default" data-dismiss="modal">Close</button></div></div></div></div>
<div class="modal fade" id="abs-zhang2023improving" role="dialog"><div class="modal-dialog"><div class="modal-content"><div class="modal-header"><button type="button" class="close" data-dismiss="modal">&times;</button><h4 class="modal-title">Abstract</h4></div><div class="modal-body">
<pre>Fetal congenital heart disease (FCHD) is a common, serious birth
defect affecting ∼1% of newborns annually. Fetal echocardiography is
the most effective and important technique for prenatal FCHD
diagnosis. The prerequisites for accurate ultrasound FCHD diagnosis
are accurate view recognition and high-quality diagnostic view
extraction. However, these manual clinical procedures have drawbacks
such as, varying technical capabilities and inefficiency. Therefore,
the automatic identification of high-quality multiview fetal heart
scan images is highly desirable to improve prenatal diagnosis
efficiency and accuracy of FCHD. Here, we present a framework for
multiview fetal heart ultrasound image recognition and quality
assessment that comprises two parts: a multiview classification and
localization network (MCLN) and an improved contrastive learning
network (ICLN). In the MCLN, a multihead enhanced self-attention
mechanism is applied to construct the classification network and
identify six accurate and interpretable views of the fetal heart. In
the ICLN, anatomical structure standardization and image clarity are
considered. With contrastive learning, the absolute loss, feature
relative loss and predicted value relative loss are combined to
achieve favorable quality assessment results. Experiments show that
the MCLN outperforms other state-of-the-art networks by 1.52–13.61%
when determining the F1 score in six standard view recognition
tasks, and the ICLN is comparable to the performance of expert
cardiologists in the quality assessment of fetal heart ultrasound
images, reaching 97% on a test set within 2 points for the four-
chamber view task. Thus, our architecture offers great potential in
helping cardiologists improve quality control for fetal
echocardiographic images in clinical practice.</pre> </div><div class="modal-footer"><button type="button" class="btn btn-default" data-dismiss="modal">Close</button></div></div></div></div>
</li><br>

<li>
<span class="author">Ke Fang, Zejun Wang, Qi Xia, Yingchao Liu, Bao Wang, Zhaowei Cheng, <b>Jian Cheng</b>, Xinyu Jin*, Ruiliang Bai*, Lanjuan Li*</span>,<br>
<span class="title">"Normalizing Flow-based Distribution Estimation of Pharmacokinetic Parameters in Dynamic Contrast-Enhanced Magnetic Resonance Imaging"</span>,<br>
<span class="publisher">IEEE Transactions on Biomedical Engineering (<b>TBME</b>)</span>, <span class="year">2023</span>.
<br>
<a target="_self" href="https://dx.doi.org/10.1109/TBME.2023.3318087"><img src="Icons/www.gif" alt="[www]" style="width: 16px; height: 16px;"></a>&nbsp;
[<a type="button" data-toggle="modal" data-target="#bib-fang2023normalizing">bibtex</a>]&nbsp;
[<a type="button" data-toggle="modal" data-target="#abs-fang2023normalizing">abstract</a>]&nbsp;
[citations: <a target="_blank" href="https://scholar.google.com//citations?view_op=view_citation&hl=en&oe=ASCII&user=BARqXQ0AAAAJ&pagesize=100&sortby=pubdate&citation_for_view=BARqXQ0AAAAJ:BJbdYPG6LGMC">4</a>]&nbsp;

<div class="modal fade" id="bib-fang2023normalizing" role="dialog"><div class="modal-dialog"><div class="modal-content"><div class="modal-header"><button type="button" class="close" data-dismiss="modal">&times;</button><h4 class="modal-title">Bibtex</h4></div><div class="modal-body">
<pre>@article{fang2023normalizing,
 author = {Ke Fang and Zejun Wang and Qi Xia and Yingchao Liu and Bao Wang and Zhaowei Cheng and Jian Cheng and Xinyu Jin and Ruiliang Bai and Lanjuan Li},
 doi = {10.1109/TBME.2023.3318087},
 journal = {IEEE Transactions on Biomedical Engineering},
 publisher = {IEEE},
 title = {Normalizing Flow-based Distribution Estimation of Pharmacokinetic Parameters in Dynamic Contrast-Enhanced Magnetic Resonance Imaging},
 url = {https://dx.doi.org/10.1109/TBME.2023.3318087},
 year = {2023}
}</pre> </div><div class="modal-footer"><button type="button" class="btn btn-default" data-dismiss="modal">Close</button></div></div></div></div>
<div class="modal fade" id="abs-fang2023normalizing" role="dialog"><div class="modal-dialog"><div class="modal-content"><div class="modal-header"><button type="button" class="close" data-dismiss="modal">&times;</button><h4 class="modal-title">Abstract</h4></div><div class="modal-body">
<pre>Objective: The pharmacokinetic (PK) parameters estimated from
dynamic contrast-enhanced magnetic resonance imaging (DCE-MRI)
provide valuable information for clinical research and diagnosis.
However, these estimated PK parameters suffer from many sources of
variability. Thus, the estimation of the posterior distributions of
these PK parameters could provide a way to simultaneously quantify
the values and uncertainties of the PK parameters. Our objective is
to develop an efficient and flexible method to more closely
approximate and estimate the underlying posterior distributions of
the PK parameters. Methods: The normalizing flow model-based
parameters distribution estimation neural network (FPDEN) is
proposed to adaptively learn and estimate the posterior
distributions of the PK parameters. The maximum likelihood
estimation (MLE) loss is directly constructed based on the parameter
distributions learned by the normalizing flow model, rather than
pre-defined distributions. Results: Experimental analysis shows that
the proposed method can improve parameter estimation accuracy.
Moreover, the uncertainty derived from the parameter distribution
constitutes an effective indicator to exclude unreliable parametric
results. A successful demonstration is the improved classification
performance of the glioma World Health Organization (WHO) grading
task, specifically in terms of distinguishing between low and high
grades, as well as between Grade III and Grade IV. Conclusion: The
FPDEN method offers improved accuracy for estimation of PK
parameters and boosts the performance of the glioma grading task.
Significance: By enhancing the precision and reliability of DCE-MRI,
the proposed method promotes its further applications in clinical
practice.</pre> </div><div class="modal-footer"><button type="button" class="btn btn-default" data-dismiss="modal">Close</button></div></div></div></div>
</li><br>

<li>
<span class="author">Ruoqi Zhao, Yuwen Wang, Xiangxin Cheng, Wanlin Zhu, Xia Meng, Haijun Niu, <b>Jian Cheng</b>*, Tao Liu*</span>,<br>
<span class="title">"A mutli-scale spatial-temporal convolutional neural network with contrastive learning for motor imagery EEG classification"</span>,<br>
<span class="publisher">Medicine in Novel Technology and Devices</span>, vol. 17, pp. 100215, <span class="year">2023</span>.
<br>
<a target="_self" href="https://www.sciencedirect.com/science/article/pii/S2590093523000103"><img src="Icons/www.gif" alt="[www]" style="width: 16px; height: 16px;"></a>&nbsp;
[<a type="button" data-toggle="modal" data-target="#bib-ZHAO2023100215">bibtex</a>]&nbsp;
[<a type="button" data-toggle="modal" data-target="#abs-ZHAO2023100215">abstract</a>]&nbsp;
[citations: <a target="_blank" href="https://scholar.google.com//citations?view_op=view_citation&hl=en&oe=ASCII&user=BARqXQ0AAAAJ&pagesize=100&sortby=pubdate&citation_for_view=BARqXQ0AAAAJ:gVv57TyPmFsC">20</a>]&nbsp;

<div class="modal fade" id="bib-ZHAO2023100215" role="dialog"><div class="modal-dialog"><div class="modal-content"><div class="modal-header"><button type="button" class="close" data-dismiss="modal">&times;</button><h4 class="modal-title">Bibtex</h4></div><div class="modal-body">
<pre>@article{ZHAO2023100215,
 author = {Ruoqi Zhao and Yuwen Wang and Xiangxin Cheng and Wanlin Zhu and Xia Meng and Haijun Niu and Jian Cheng and Tao Liu},
 doi = {https://doi.org/10.1016/j.medntd.2023.100215},
 journal = {Medicine in Novel Technology and Devices},
 pages = {100215},
 title = {A mutli-scale spatial-temporal convolutional neural network with contrastive learning for motor imagery EEG classification},
 url = {https://www.sciencedirect.com/science/article/pii/S2590093523000103},
 volume = {17},
 year = {2023}
}</pre> </div><div class="modal-footer"><button type="button" class="btn btn-default" data-dismiss="modal">Close</button></div></div></div></div>
<div class="modal fade" id="abs-ZHAO2023100215" role="dialog"><div class="modal-dialog"><div class="modal-content"><div class="modal-header"><button type="button" class="close" data-dismiss="modal">&times;</button><h4 class="modal-title">Abstract</h4></div><div class="modal-body">
<pre>Motor imagery (MI) based Brain-computer interfaces (BCIs) have a
wide range of applications in the stroke rehabilitation field.
However, due to the low signal-to-noise ratio and high cross-subject
variation of the electroencephalogram (EEG) signals generated by
motor imagery, the classification performance of the existing
methods still needs to be improved to meet the need of real
practice. To overcome this problem, we propose a multi-scale
spatial-temporal convolutional neural network called MSCNet. We
introduce the contrastive learning into a multi-temporal convolution
scale backbone to further improve the robustness and discrimination
of embedding vectors. Experimental results of binary classification
show that MSCNet outperforms the state-of-the-art methods, achieving
accuracy improvement of 6.04%, 3.98%, and 8.15% on BCIC IV 2a, SMR-
BCI, and OpenBMI datasets in subject-dependent manner, respectively.
The results show that the contrastive learning method can
significantly improve the classification accuracy of motor imagery
EEG signals, which provides an important reference for the design of
motor imagery classification algorithms.</pre> </div><div class="modal-footer"><button type="button" class="btn btn-default" data-dismiss="modal">Close</button></div></div></div></div>
</li><br>

<li>
<span class="author">Jing Jing, Ziyang Liu, Hao Guan, Wanlin Zhu, Zhe Zhang, Xia Meng, <b>Jian Cheng</b>, Yuesong Pan, Yong Jiang, Yilong Wang, Haijun Niu, Xingquan Zhao, Wei Wen, Jinxi Lin, Wei Li, Hao Li, Perminder S. Sachdev, Tao Liu, Zixiao Li, Dacheng Tao, Yongjun Wang</span>,<br>
<span class="title">"A Deep Learning System to Predict Recurrence and Disability Outcomes in Patients with Transient Ischemic Attack or Ischemic Stroke"</span>,<br>
<span class="publisher">Advanced Intelligent Systems</span>, vol. 5, no. 4, pp. 2200240, <span class="year">2023</span>.
<br>
<a target="_self" href="https://onlinelibrary.wiley.com/doi/abs/10.1002/aisy.202200240"><img src="Icons/www.gif" alt="[www]" style="width: 16px; height: 16px;"></a>&nbsp;
[<a type="button" data-toggle="modal" data-target="#bib-jing2023_AIS">bibtex</a>]&nbsp;
[<a type="button" data-toggle="modal" data-target="#abs-jing2023_AIS">abstract</a>]&nbsp;
[citations: <a target="_blank" href="https://scholar.google.com//citations?view_op=view_citation&hl=en&oe=ASCII&user=BARqXQ0AAAAJ&pagesize=100&sortby=pubdate&citation_for_view=BARqXQ0AAAAJ:5MTHONV0fEkC">10</a>]&nbsp;

<div class="modal fade" id="bib-jing2023_AIS" role="dialog"><div class="modal-dialog"><div class="modal-content"><div class="modal-header"><button type="button" class="close" data-dismiss="modal">&times;</button><h4 class="modal-title">Bibtex</h4></div><div class="modal-body">
<pre>@article{jing2023_AIS,
 author = {Jing Jing and Ziyang Liu and Hao Guan and Wanlin Zhu and Zhe Zhang and Xia Meng and Jian Cheng and Yuesong Pan and Yong Jiang and Yilong Wang and Haijun Niu and Xingquan Zhao and Wei Wen and Jinxi Lin and Wei Li and Hao Li and Perminder S. Sachdev and Tao Liu and Zixiao Li and Dacheng Tao and Yongjun Wang},
 doi = {https://doi.org/10.1002/aisy.202200240},
 journal = {Advanced Intelligent Systems},
 number = {4},
 pages = {2200240},
 title = {A Deep Learning System to Predict Recurrence and Disability Outcomes in Patients with Transient Ischemic Attack or Ischemic Stroke},
 url = {https://onlinelibrary.wiley.com/doi/abs/10.1002/aisy.202200240},
 volume = {5},
 year = {2023}
}</pre> </div><div class="modal-footer"><button type="button" class="btn btn-default" data-dismiss="modal">Close</button></div></div></div></div>
<div class="modal fade" id="abs-jing2023_AIS" role="dialog"><div class="modal-dialog"><div class="modal-content"><div class="modal-header"><button type="button" class="close" data-dismiss="modal">&times;</button><h4 class="modal-title">Abstract</h4></div><div class="modal-body">
<pre>Ischemic strokes (IS) and transient ischemic attacks (TIA) account
for approximately 80\% of all strokes and are leading causes of
death worldwide. Assessing the risk of recurrence or functional
impairment in IS and TIA patients is essential to both acute phase
treatment and secondary prevention. Current risk prediction systems
that rely on clinical parameters alone without leveraging imaging
data have only modest performance. Herein, a deep learning-based
risk prediction system (RPS) is developed to predict the probability
of stroke recurrence or disability (i.e., deep-learning stroke
recurrence risk score, SRR score). Then, Kaplan–Meier analysis to
evaluate the ability of SRR score to stratify patients at stroke
recurrence risk is discussed. Using 15 166 Third China National
Stroke Registry (CNSR-III) cases, the RPS's receiver operating
characteristic curve (AUC) values of 0.850 for 14 day TIA recurrence
prediction and 0.837 for 3 month IS disability prediction are used.
Among patients deemed high risk by SRR score, 22.9\% and 24.4\% of
individuals with TIA and IS respectively have stroke recurrence
within 1 year, which are significantly higher than the rates in low-
risk individuals. Deep learning-based RPS can outperform
conventional risk scores and has the potential to assist accurate
prognostication in stroke patients to optimize management.</pre> </div><div class="modal-footer"><button type="button" class="btn btn-default" data-dismiss="modal">Close</button></div></div></div></div>
</li><br>

<li>
<span class="author">Yuxin Gong, Haogang Zhu*, Jixing Li, Jingchun Yang*, <b>Jian Cheng</b>, Ying Chang, Xiao Bai, Xunming Ji</span>,<br>
<span class="title">"SCCNet: Self-correction boundary preservation with a dynamic class prior filter for high-variability ultrasound image segmentation"</span>,<br>
<span class="publisher">Computerized Medical Imaging and Graphics</span>, vol. 104, pp. 102183, <span class="year">2023</span>.
<br>
<a target="_self" href="https://dx.doi.org/10.1016/j.compmedimag.2023.102183"><img src="Icons/www.gif" alt="[www]" style="width: 16px; height: 16px;"></a>&nbsp;
[<a type="button" data-toggle="modal" data-target="#bib-gong2023sccnet">bibtex</a>]&nbsp;
[<a type="button" data-toggle="modal" data-target="#abs-gong2023sccnet">abstract</a>]&nbsp;
[citations: <a target="_blank" href="https://scholar.google.com//citations?view_op=view_citation&hl=en&oe=ASCII&user=BARqXQ0AAAAJ&pagesize=100&sortby=pubdate&citation_for_view=BARqXQ0AAAAJ:fbc8zXXH2BUC">10</a>]&nbsp;

<div class="modal fade" id="bib-gong2023sccnet" role="dialog"><div class="modal-dialog"><div class="modal-content"><div class="modal-header"><button type="button" class="close" data-dismiss="modal">&times;</button><h4 class="modal-title">Bibtex</h4></div><div class="modal-body">
<pre>@article{gong2023sccnet,
 author = {Yuxin Gong and Haogang Zhu and Jixing Li and Jingchun Yang and Jian Cheng and Ying Chang and Xiao Bai and Xunming Ji},
 doi = {10.1016/j.compmedimag.2023.102183},
 journal = {Computerized Medical Imaging and Graphics},
 pages = {102183},
 publisher = {Elsevier},
 title = {SCCNet: Self-correction boundary preservation with a dynamic class prior filter for high-variability ultrasound image segmentation},
 url = {https://dx.doi.org/10.1016/j.compmedimag.2023.102183},
 volume = {104},
 year = {2023}
}</pre> </div><div class="modal-footer"><button type="button" class="btn btn-default" data-dismiss="modal">Close</button></div></div></div></div>
<div class="modal fade" id="abs-gong2023sccnet" role="dialog"><div class="modal-dialog"><div class="modal-content"><div class="modal-header"><button type="button" class="close" data-dismiss="modal">&times;</button><h4 class="modal-title">Abstract</h4></div><div class="modal-body">
<pre>The highly ambiguous nature of boundaries and similar objects is
difficult to address in some ultrasound image segmentation tasks,
such as neck muscle segmentation, leading to unsatisfactory
performance. Thus, this paper proposes a two-stage network called
SCCNet (self-correction context network) using a self-correction
boundary preservation module and class-context filter to alleviate
these problems. The proposed self-correction boundary preservation
module uses a dynamic key boundary point (KBP) map to increase the
capability of iteratively discriminating ambiguous boundary points
segments, and the predicted segmentation map from one stage is used
to obtain a dynamic class prior filter to improve the segmentation
performance at Stage 2. Finally, three datasets, Neck Muscle, CAMUS
and Thyroid, are used to demonstrate that our proposed SCCNet
outperforms other state-of-the art methods, such as BPBnet, DSNnet,
and RAGCnet. Our proposed network shows at least a 1.2–3.7%
improvement on the three datasets, Neck Muscle, Thyroid, and CAMUS.
The source code is available at
https://github.com/lijixing0425/SCCNet.</pre> </div><div class="modal-footer"><button type="button" class="btn btn-default" data-dismiss="modal">Close</button></div></div></div></div>
</li><br>

<li>
<span class="author">Guozheng Feng, Rui Chen, Rui Zhao, Yuanyuan Li, Leilei Ma, Yanpei Wang, Weiwei Men, Jiahong Gao, Shuping Tan, <b>Jian Cheng</b>, Yong He, Shaozheng Qin, Qi Dong, Sha Tao*, Ni Shu*</span>,<br>
<span class="title">"Longitudinal development of the human white matter structural connectome and its association with brain transcriptomic and cellular architecture"</span>,<br>
<span class="publisher">Communications Biology</span>, vol. 6, no. 1, pp. 1257, <span class="year">2023</span>.
<br>
<a target="_self" href="https://dx.doi.org/10.1038/s42003-024-05771-z"><img src="Icons/www.gif" alt="[www]" style="width: 16px; height: 16px;"></a>&nbsp;
[<a type="button" data-toggle="modal" data-target="#bib-feng2023longitudinal">bibtex</a>]&nbsp;
[<a type="button" data-toggle="modal" data-target="#abs-feng2023longitudinal">abstract</a>]&nbsp;
[citations: <a target="_blank" href="https://scholar.google.com//citations?view_op=view_citation&hl=en&oe=ASCII&user=BARqXQ0AAAAJ&pagesize=100&sortby=pubdate&citation_for_view=BARqXQ0AAAAJ:4X0JR2_MtJMC">9</a>]&nbsp;

<div class="modal fade" id="bib-feng2023longitudinal" role="dialog"><div class="modal-dialog"><div class="modal-content"><div class="modal-header"><button type="button" class="close" data-dismiss="modal">&times;</button><h4 class="modal-title">Bibtex</h4></div><div class="modal-body">
<pre>@article{feng2023longitudinal,
 author = {Guozheng Feng and Rui Chen and Rui Zhao and Yuanyuan Li and Leilei Ma and Yanpei Wang and Weiwei Men and Jiahong Gao and Shuping Tan and Jian Cheng and Yong He and Shaozheng Qin and Qi Dong and Sha Tao and Ni Shu},
 doi = {10.1038/s42003-024-05771-z},
 journal = {Communications Biology},
 number = {1},
 pages = {1257},
 publisher = {Nature Publishing Group UK London},
 title = {Longitudinal development of the human white matter structural connectome and its association with brain transcriptomic and cellular architecture},
 url = {https://dx.doi.org/10.1038/s42003-024-05771-z},
 volume = {6},
 year = {2023}
}</pre> </div><div class="modal-footer"><button type="button" class="btn btn-default" data-dismiss="modal">Close</button></div></div></div></div>
<div class="modal fade" id="abs-feng2023longitudinal" role="dialog"><div class="modal-dialog"><div class="modal-content"><div class="modal-header"><button type="button" class="close" data-dismiss="modal">&times;</button><h4 class="modal-title">Abstract</h4></div><div class="modal-body">
<pre>From childhood to adolescence, the spatiotemporal development
pattern of the human brain white matter connectome and its
underlying transcriptomic and cellular mechanisms remain largely
unknown. With a longitudinal diffusion MRI cohort of 604
participants, we map the developmental trajectory of the white
matter connectome from global to regional levels and identify that
most brain network properties followed a linear developmental
trajectory. Importantly, connectome-transcriptomic analysis reveals
that the spatial development pattern of white matter connectome is
potentially regulated by the transcriptomic architecture, with
positively correlated genes involve in ion transport- and
development-related pathways expressed in excitatory and inhibitory
neurons, and negatively correlated genes enriches in synapse- and
development-related pathways expressed in astrocytes, inhibitory
neurons and microglia. Additionally, the macroscale developmental
pattern is also associated with myelin content and thicknesses of
specific laminas. These findings offer insights into the underlying
genetics and neural mechanisms of macroscale white matter connectome
development from childhood to adolescence.</pre> </div><div class="modal-footer"><button type="button" class="btn btn-default" data-dismiss="modal">Close</button></div></div></div></div>
</li><br>

</ol><h2><a name="year2022"></a>2022</h2>

<ol reversed>

<li>
<span class="author">Shengke Xue#, Zhaowei Cheng#, Guangxu Han, Chaoliang Sun, Ke Fang, Yingchao Liu, <b>Jian Cheng</b>, Xinyu Jin, Ruiliang Bai*</span>,<br>
<span class="title">"2D probabilistic undersampling pattern optimization for MR image reconstruction"</span>,<br>
<span class="publisher">Medical Image Analysis (<b>MedIA</b>)</span>, vol. 77, pp. 102346, <span class="year">2022</span>.
<br>
<a target="_self" href="https://www.sciencedirect.com/science/article/pii/S1361841521003911"><img src="Icons/www.gif" alt="[www]" style="width: 16px; height: 16px;"></a>&nbsp;
[<a type="button" data-toggle="modal" data-target="#bib-xue-MIA2022-sampling">bibtex</a>]&nbsp;
[<a type="button" data-toggle="modal" data-target="#abs-xue-MIA2022-sampling">abstract</a>]&nbsp;
[citations: <a target="_blank" href="https://scholar.google.com//citations?view_op=view_citation&hl=en&oe=ASCII&user=BARqXQ0AAAAJ&pagesize=100&sortby=pubdate&citation_for_view=BARqXQ0AAAAJ:_OXeSy2IsFwC">12</a>]&nbsp;

<div class="modal fade" id="bib-xue-MIA2022-sampling" role="dialog"><div class="modal-dialog"><div class="modal-content"><div class="modal-header"><button type="button" class="close" data-dismiss="modal">&times;</button><h4 class="modal-title">Bibtex</h4></div><div class="modal-body">
<pre>@article{xue:MIA2022:sampling,
 author = {Shengke Xue and Zhaowei Cheng and Guangxu Han and Chaoliang Sun and Ke Fang and Yingchao Liu and Jian Cheng and Xinyu Jin and Ruiliang Bai},
 doi = {https://doi.org/10.1016/j.media.2021.102346},
 journal = {Medical Image Analysis},
 pages = {102346},
 title = {2D probabilistic undersampling pattern optimization for MR image reconstruction},
 url = {https://www.sciencedirect.com/science/article/pii/S1361841521003911},
 volume = {77},
 year = {2022}
}</pre> </div><div class="modal-footer"><button type="button" class="btn btn-default" data-dismiss="modal">Close</button></div></div></div></div>
<div class="modal fade" id="abs-xue-MIA2022-sampling" role="dialog"><div class="modal-dialog"><div class="modal-content"><div class="modal-header"><button type="button" class="close" data-dismiss="modal">&times;</button><h4 class="modal-title">Abstract</h4></div><div class="modal-body">
<pre>With 3D magnetic resonance imaging (MRI), a tradeoff exists between
higher image quality and shorter scan time. One way to solve this
problem is to reconstruct high-quality MRI images from undersampled
k-space. There have been many recent studies exploring effective
k-space undersampling patterns and designing MRI reconstruction
methods from undersampled k-space, which are two necessary steps.
Most studies separately considered these two steps, although in
theory, their performance is dependent on each other. In this study,
we propose a joint optimization model, trained end-to-end, to
simultaneously optimize the undersampling pattern in the Fourier
domain and the reconstruction model in the image domain. A 2D
probabilistic undersampling layer was designed to optimize the
undersampling pattern and probability distribution in a
differentiable manner. A 2D inverse Fourier transform layer was
implemented to connect the Fourier domain and the image domain
during the forward and back propagation. Finally, we discovered an
optimized relationship between the probability distribution of the
undersampling pattern and its corresponding sampling rate. Further
testing was performed using 3D T1-weighted MR images of the brain
from the MICCAI 2013 Grand Challenge on Multi-Atlas Labeling dataset
and locally acquired brain 3D T1-weighted MR images of healthy
volunteers and contrast-enhanced 3D T1-weighted MR images of high-
grade glioma patients. The results showed that the recovered MR
images using our 2D probabilistic undersampling pattern (with or
without the reconstruction network) significantly outperformed those
using the existing start-of-the-art undersampling strategies for
both qualitative and quantitative comparison, suggesting the
advantages and some extent of the generalization of our proposed
method.</pre> </div><div class="modal-footer"><button type="button" class="btn btn-default" data-dismiss="modal">Close</button></div></div></div></div>
</li><br>

<li>
<span class="author">Jing Jing#, Yijun Zhou#, Yuesong Pan#, Xueli Cai#, Wanlin Zhu, Zhe Zhang, Zixiao Li, Chang Liu, Xia Meng, <b>Jian Cheng</b>, Yilong Wang, Hao Li, Zhenzhou Wu, Suying Wang, Haijun Niu, Wei Wen, Tao Liu*, Tiemin Wei*, Yongjun Wang*, Perminder S. Sachdev</span>,<br>
<span class="title">"Reduced white matter microstructural integrity in prediabetes and diabetes: A population-based study"</span>,<br>
<span class="publisher"><b>eBioMedicine</b></span>, vol. 82, pp. 104144, <span class="year">2022</span>.
<br><a target="_self" href="https://www.sciencedirect.com/science/article/pii/S2352396422003255/pdfft?md5=cd11719d27845497ef250123b8479b1f&pid=1-s2.0-S2352396422003255-main.pdf"><img src="Icons/pdf.gif" alt="[pdf]" style="width: 16px; height: 16px;"></a>&nbsp;
<a target="_self" href="https://www.sciencedirect.com/science/article/pii/S2352396422003255"><img src="Icons/www.gif" alt="[www]" style="width: 16px; height: 16px;"></a>&nbsp;
[<a type="button" data-toggle="modal" data-target="#bib-jing-eBioMedicine2022">bibtex</a>]&nbsp;
[<a type="button" data-toggle="modal" data-target="#abs-jing-eBioMedicine2022">abstract</a>]&nbsp;
[citations: <a target="_blank" href="https://scholar.google.com//citations?view_op=view_citation&hl=en&oe=ASCII&user=BARqXQ0AAAAJ&pagesize=100&sortby=pubdate&citation_for_view=BARqXQ0AAAAJ:ZzlSgRqYykMC">23</a>]&nbsp;

<div class="modal fade" id="bib-jing-eBioMedicine2022" role="dialog"><div class="modal-dialog"><div class="modal-content"><div class="modal-header"><button type="button" class="close" data-dismiss="modal">&times;</button><h4 class="modal-title">Bibtex</h4></div><div class="modal-body">
<pre>@article{jing:eBioMedicine2022,
 author = {Jing Jing and Yijun Zhou and Yuesong Pan and Xueli Cai and Wanlin Zhu and Zhe Zhang and Zixiao Li and Chang Liu and Xia Meng and Jian Cheng and Yilong Wang and Hao Li and Zhenzhou Wu and Suying Wang and Haijun Niu and Wei Wen and Tao Liu and Tiemin Wei and Yongjun Wang and Perminder S. Sachdev},
 doi = {https://doi.org/10.1016/j.ebiom.2022.104144},
 journal = {eBioMedicine},
 pages = {104144},
 pdf = {https://www.sciencedirect.com/science/article/pii/S2352396422003255/pdfft?md5=cd11719d27845497ef250123b8479b1f&pid=1-s2.0-S2352396422003255-main.pdf},
 title = {Reduced white matter microstructural integrity in prediabetes and diabetes: A population-based study},
 url = {https://www.sciencedirect.com/science/article/pii/S2352396422003255},
 volume = {82},
 year = {2022}
}</pre> </div><div class="modal-footer"><button type="button" class="btn btn-default" data-dismiss="modal">Close</button></div></div></div></div>
<div class="modal fade" id="abs-jing-eBioMedicine2022" role="dialog"><div class="modal-dialog"><div class="modal-content"><div class="modal-header"><button type="button" class="close" data-dismiss="modal">&times;</button><h4 class="modal-title">Abstract</h4></div><div class="modal-body">
<pre>Summary Background White matter (WM) microstructural abnormalities
have been observed in diabetes. However, evidence of prediabetes is
currently lacking. This study aims to investigate the WM integrity
in prediabetes and diabetes. We also assess the association of WM
abnormalities with glucose metabolism status and continuous glucose
measures. Methods The WM integrity was analyzed using cross-
sectional baseline data from a population-based PolyvasculaR
Evaluation for Cognitive Impairment and vaScular Events (PRECISE)
study. The cohort, including a total of 2218 cases with the mean age
of 61.3 ± 6.6 years and 54.1% female, consisted of 1205 prediabetes
which are categorized into two subgroups (a group of 254 prediabetes
with combined impaired fasting glucose (IFG) and impaired glucose
tolerance (IGT) and the other group of 951 prediabetes without
combined IFG/IGT), 504 diabetes, and 509 normal control subjects.
Alterations of WM integrity were determined by diffusion tensor
imaging along with tract-based spatial statistics analysis to
compare diffusion metrics on WM skeletons between groups. The mixed-
effects multivariate linear regression models were used to assess
the association between WM microstructural alterations and glucose
status. Findings Microstructural abnormalities distributed in local
WM tracts in prediabetes with combined IFG/IGT and spread widely in
diabetes. These WM abnormalities are associated with higher glucose
measures. Interpretation Our findings suggest that WM
microstructural abnormalities are already present at the prediabetes
with combined IFG/IGT stage. Preventative strategies should begin
early to maintain normal glucose metabolism and avert further
destruction of WM integrity. Funding Partially supported by National
Key R&D Program of China (2016YFC0901002).</pre> </div><div class="modal-footer"><button type="button" class="btn btn-default" data-dismiss="modal">Close</button></div></div></div></div>
</li><br>

<li>
<span class="author">Hua Zhu, Lijun Zuo, Wanlin Zhu, Jing Jing, Zhe Zhang, Lingling Ding, Fengjuan Wang, <b>Jian Cheng</b>, Zhenzhou Wu, Yongjun Wang, Tao Liu*, Zixiao Li*</span>,<br>
<span class="title">"The distinct disrupted plasticity in structural and functional network in mild stroke with basal ganglia region infarcts"</span>,<br>
<span class="publisher">Brain Imaging and Behavior</span>, pp. 1&ndash;21, <span class="year">2022</span>.
<br>
<a target="_self" href="https://dx.doi.org/10.1007/s11682-022-00689-8"><img src="Icons/www.gif" alt="[www]" style="width: 16px; height: 16px;"></a>&nbsp;
[<a type="button" data-toggle="modal" data-target="#bib-zhu2022distinct">bibtex</a>]&nbsp;
[<a type="button" data-toggle="modal" data-target="#abs-zhu2022distinct">abstract</a>]&nbsp;
[citations: <a target="_blank" href="https://scholar.google.com//citations?view_op=view_citation&hl=en&oe=ASCII&user=BARqXQ0AAAAJ&pagesize=100&sortby=pubdate&citation_for_view=BARqXQ0AAAAJ:8xutWZnSdmoC">17</a>]&nbsp;

<div class="modal fade" id="bib-zhu2022distinct" role="dialog"><div class="modal-dialog"><div class="modal-content"><div class="modal-header"><button type="button" class="close" data-dismiss="modal">&times;</button><h4 class="modal-title">Bibtex</h4></div><div class="modal-body">
<pre>@article{zhu2022distinct,
 author = {Hua Zhu and Lijun Zuo and Wanlin Zhu and Jing Jing and Zhe Zhang and Lingling Ding and Fengjuan Wang and Jian Cheng and Zhenzhou Wu and Yongjun Wang and Tao Liu and Zixiao Li},
 doi = {10.1007/s11682-022-00689-8},
 journal = {Brain Imaging and Behavior},
 pages = {1-21},
 publisher = {Springer},
 title = {The distinct disrupted plasticity in structural and functional network in mild stroke with basal ganglia region infarcts},
 url = {https://dx.doi.org/10.1007/s11682-022-00689-8},
 year = {2022}
}</pre> </div><div class="modal-footer"><button type="button" class="btn btn-default" data-dismiss="modal">Close</button></div></div></div></div>
<div class="modal fade" id="abs-zhu2022distinct" role="dialog"><div class="modal-dialog"><div class="modal-content"><div class="modal-header"><button type="button" class="close" data-dismiss="modal">&times;</button><h4 class="modal-title">Abstract</h4></div><div class="modal-body">
<pre>Stroke induced by basal ganglia infarction often impair cognitive
function. The exploration of topological patterns in structural and
functional networks associated cognitive impairment after stroke may
contribute to understand the pathological mechanism of cognitive
impairment caused by stroke. In this paper, graph theory analysis
was applied to diffusion-weighted imaging (DWI) data and resting-
state functional MRI (fMRI) data from 23 post-stroke patients with
cognitive impairment (PSCI), 17 post-stroke patients without
cognitive impairment (NPSCI), and 29 healthy controls (HC).
Structural and functional connectivity between 90 cortical and
subcortical brain regions was estimated and set threshold to
construct a set of undirected graphs. Network-based statistics (NBS)
was used to characterize altered connectivity patterns among the
three groups. Compared to HC, the PSCI group demonstrated
substantial reductions in all three types of connections—rich club,
feeder, and local—in structural and functional networks.
Specifically, in structural network analysis, reduced connections
were observed within basal ganglia and basal ganglia-frontal
networks, whereas in the functional network analysis, reduced
connections were observed in fronto-parietal network (FPN) and
cingulo-opercular networks (CON). Meanwhile, compared to HC, the
NPSCI group demonstrated reductions in both feeder and local
connections only within occipital area and occipital-temporal
structural networks. The findings of reduced structural connectivity
in regions stemming from a basal ganglia core and reduced functional
connectivity in FPN and CON may indicate a bottom-up cognitive
impairment induced by stroke. Graph analysis and connectomics may
aid clinical diagnosis and serve as potential imaging biomarkers for
post-stroke patients with cognitive impairment.</pre> </div><div class="modal-footer"><button type="button" class="btn btn-default" data-dismiss="modal">Close</button></div></div></div></div>
</li><br>

</ol><h2><a name="year2021"></a>2021</h2>

<ol reversed>

<li>
<span class="author"><b>Jian Cheng</b>#, Ziyang Liu#, Hao Guan, Zhenzhou Wu, Haogang Zhu, Jiyang Jiang, Wei Wen, Dacheng Tao, Tao Liu*</span>,<br>
<span class="title">"Brain Age Estimation From MRI Using Cascade Networks with Ranking Loss"</span>,<br>
<span class="publisher">IEEE Transactions on Medical Imaging (<b>TMI</b>)</span>, vol. 40, no. 12, pp. 3400&ndash;3412, <span class="year">2021</span>.
<br><a target="_self" href="https://arxiv.org/pdf/2106.03052.pdf"><img src="Icons/pdf.gif" alt="[pdf]" style="width: 16px; height: 16px;"></a>&nbsp;
<a target="_self" href="https://dx.doi.org/10.1109/TMI.2021.3085948"><img src="Icons/www.gif" alt="[www]" style="width: 16px; height: 16px;"></a>&nbsp;
[<a type="button" data-toggle="modal" data-target="#bib-cheng-TMI2021">bibtex</a>]&nbsp;
[<a type="button" data-toggle="modal" data-target="#abs-cheng-TMI2021">abstract</a>]&nbsp;
[<a target="_self" href="https://arxiv.org/abs/2106.03052">arxiv</a>]&nbsp;
[<a target="_self" href="https://github.com/Milan-BUAA/TSAN-brain-age-estimation">code</a>]&nbsp;
[citations: <a target="_blank" href="https://scholar.google.com//citations?view_op=view_citation&hl=en&oe=ASCII&user=BARqXQ0AAAAJ&pagesize=100&sortby=pubdate&citation_for_view=BARqXQ0AAAAJ:yqoGN6RLRZoC">94</a>]&nbsp;
(<span class="hlnote0">Jian Cheng and Ziyang Liu contributed equally.</span>)&nbsp;

<div class="modal fade" id="bib-cheng-TMI2021" role="dialog"><div class="modal-dialog"><div class="modal-content"><div class="modal-header"><button type="button" class="close" data-dismiss="modal">&times;</button><h4 class="modal-title">Bibtex</h4></div><div class="modal-body">
<pre>@article{cheng:TMI2021,
 author = {Jian Cheng and Ziyang Liu and Hao Guan and Zhenzhou Wu and Haogang Zhu and Jiyang Jiang and Wei Wen and Dacheng Tao and Tao Liu},
 doi = {10.1109/TMI.2021.3085948},
 journal = {IEEE Transactions on Medical Imaging},
 number = {12},
 pages = {3400-3412},
 pdf = {https://arxiv.org/pdf/2106.03052.pdf},
 publisher = {IEEE},
 title = {Brain Age Estimation From MRI Using Cascade Networks with Ranking Loss},
 url = {https://dx.doi.org/10.1109/TMI.2021.3085948},
 volume = {40},
 year = {2021}
}</pre> </div><div class="modal-footer"><button type="button" class="btn btn-default" data-dismiss="modal">Close</button></div></div></div></div>
<div class="modal fade" id="abs-cheng-TMI2021" role="dialog"><div class="modal-dialog"><div class="modal-content"><div class="modal-header"><button type="button" class="close" data-dismiss="modal">&times;</button><h4 class="modal-title">Abstract</h4></div><div class="modal-body">
<pre>Chronological age of healthy people is able to be predicted
accurately using deep neural networks from neuroimaging data,  and
the predicted brain age could serve as a biomarker for detecting
aging-related diseases.  In this paper, a novel 3D convolutional
network, called two-stage-age-network (TSAN), is proposed to
estimate brain age from T1-weighted MRI data.  Compared with
existing methods, TSAN has the following improvements.  First, TSAN
uses a two-stage cascade network architecture, where the first-stage
network estimates a rough brain age,  then the second-stage network
estimates the brain age more accurately from the discretized brain
age by the first-stage network.  Second, to our knowledge, TSAN is
the first work to apply novel ranking losses in brain age
estimation, together with the traditional mean square error (MSE)
loss.  Third, densely connected paths are used to combine feature
maps with different scales.  The experiments with $6586$ MRIs showed
that TSAN could provide accurate brain age estimation,  yielding
mean absolute error (MAE) of $2.428$ and Pearson's correlation
coefficient (PCC) of $0.985$, between the estimated and
chronological ages.  Furthermore, using the brain age gap between
brain age and chronological age as a biomarker,  Alzheimer's disease
(AD) and Mild Cognitive Impairment (MCI) can be distinguished from
healthy control (HC) subjects by support vector machine (SVM).
Classification AUC in AD/HC and MCI/HC was $0.904$ and $0.823$,
respectively.  It showed that brain age gap is an effective
biomarker associated with risk of dementia, and has potential for
early-stage dementia risk screening.  The codes and trained models
have been released on GitHub: https://github.com/Milan-BUAA/TSAN-
brain-age-estimation.</pre> </div><div class="modal-footer"><button type="button" class="btn btn-default" data-dismiss="modal">Close</button></div></div></div></div>
</li><br>

<li>
<span class="author">Hui Tang, Tao Liu*, Hao Liu, Jiyang Jiang, <b>Jian Cheng</b>, Haijun Niu, Shuyu Li, Henry Brodaty, Perminder Sachdev, Wei Wen</span>,<br>
<span class="title">"A slower rate of sulcal widening in the brains of the nondemented oldest old"</span>,<br>
<span class="publisher"><b>NeuroImage</b></span>, vol. 229, pp. 117740, <span class="year">2021</span>.
<br><a target="_self" href="https://www.sciencedirect.com/science/article/pii/S1053811921000173/pdfft?md5=21c91728b2c65b222185ad07040f3a6f&pid=1-s2.0-S1053811921000173-main.pdf"><img src="Icons/pdf.gif" alt="[pdf]" style="width: 16px; height: 16px;"></a>&nbsp;
<a target="_self" href="https://dx.doi.org/10.1016/j.neuroimage.2021.117740"><img src="Icons/www.gif" alt="[www]" style="width: 16px; height: 16px;"></a>&nbsp;
[<a type="button" data-toggle="modal" data-target="#bib-tang-NI2021">bibtex</a>]&nbsp;
[<a type="button" data-toggle="modal" data-target="#abs-tang-NI2021">abstract</a>]&nbsp;
[citations: <a target="_blank" href="https://scholar.google.com//citations?view_op=view_citation&hl=en&oe=ASCII&user=BARqXQ0AAAAJ&pagesize=100&sortby=pubdate&citation_for_view=BARqXQ0AAAAJ:LhH-TYMQEocC">18</a>]&nbsp;

<div class="modal fade" id="bib-tang-NI2021" role="dialog"><div class="modal-dialog"><div class="modal-content"><div class="modal-header"><button type="button" class="close" data-dismiss="modal">&times;</button><h4 class="modal-title">Bibtex</h4></div><div class="modal-body">
<pre>@article{tang:NI2021,
 author = {Hui Tang and Tao Liu and Hao Liu and Jiyang Jiang and Jian Cheng and Haijun Niu and Shuyu Li and Henry Brodaty and Perminder Sachdev and Wei Wen},
 doi = {10.1016/j.neuroimage.2021.117740},
 journal = {NeuroImage},
 pages = {117740},
 pdf = {https://www.sciencedirect.com/science/article/pii/S1053811921000173/pdfft?md5=21c91728b2c65b222185ad07040f3a6f&pid=1-s2.0-S1053811921000173-main.pdf},
 publisher = {Elsevier},
 title = {A slower rate of sulcal widening in the brains of the nondemented oldest old},
 url = {https://dx.doi.org/10.1016/j.neuroimage.2021.117740},
 volume = {229},
 year = {2021}
}</pre> </div><div class="modal-footer"><button type="button" class="btn btn-default" data-dismiss="modal">Close</button></div></div></div></div>
<div class="modal fade" id="abs-tang-NI2021" role="dialog"><div class="modal-dialog"><div class="modal-content"><div class="modal-header"><button type="button" class="close" data-dismiss="modal">&times;</button><h4 class="modal-title">Abstract</h4></div><div class="modal-body">
<pre>The relationships between aging and brain morphology have been
reported in many previous structural brain studies. However, the
trajectories of successful brain aging in the extremely old remain
underexplored. In the limited research on the oldest old, covering
individuals aged 85 years and older, there are very few studies that
have focused on the cortical morphology, especially cortical sulcal
features. In this paper, we measured sulcal width and depth as well
as cortical thickness from T1-weighted scans of 290 nondemented
community-dwelling participants aged between 76 and 103 years. We
divided the participants into young old (between 76 and 84; mean =
80.35±2.44; male/female = 76/88) and oldest old (between 85 and 103;
mean = 91.74±5.11; male/female = 60/66) groups. The results showed
that most of the examined sulci significantly widened with increased
age and that the rates of sulcal widening were lower in the oldest
old. The spatial pattern of the cortical thinning partly
corresponded with that of sulcal widening. Compared to females,
males had significantly wider sulci, especially in the oldest old.
This study builds a foundation for future investigations of
neurocognitive disorders and neurodegenerative diseases in the
oldest old, including centenarians.</pre> </div><div class="modal-footer"><button type="button" class="btn btn-default" data-dismiss="modal">Close</button></div></div></div></div>
</li><br>

<li>
<span class="author">Haichao Zhao#, <b>Jian Cheng</b>#, Tao Liu*, Jiyang Jiang, Forrest Koch, Perminder S. Sachdev, Peter J. Basser, Wei Wen, for the Alzheimer's Disease Neuroimaging Initiative</span>,<br>
<span class="title">"Orientational changes of white matter fibers in Alzheimer's disease and amnestic mild cognitive impairment"</span>,<br>
<span class="publisher">Human Brain Mapping (<b>HBM</b>)</span>, <span class="year">2021</span>.
<br><a target="_self" href="https://onlinelibrary.wiley.com/doi/pdfdirect/10.1002/hbm.25628"><img src="Icons/pdf.gif" alt="[pdf]" style="width: 16px; height: 16px;"></a>&nbsp;
<a target="_self" href="https://onlinelibrary.wiley.com/doi/abs/10.1002/hbm.25628"><img src="Icons/www.gif" alt="[www]" style="width: 16px; height: 16px;"></a>&nbsp;
[<a type="button" data-toggle="modal" data-target="#bib-zhao-DFA-HBM2021">bibtex</a>]&nbsp;
[<a type="button" data-toggle="modal" data-target="#abs-zhao-DFA-HBM2021">abstract</a>]&nbsp;
[citations: <a target="_blank" href="https://scholar.google.com//citations?view_op=view_citation&hl=en&oe=ASCII&user=BARqXQ0AAAAJ&pagesize=100&sortby=pubdate&citation_for_view=BARqXQ0AAAAJ:OP4eGU-M3BUC">18</a>]&nbsp;
(<span class="hlnote0">Haichao Zhao and Jian Cheng contributed equally.</span>)&nbsp;

<div class="modal fade" id="bib-zhao-DFA-HBM2021" role="dialog"><div class="modal-dialog"><div class="modal-content"><div class="modal-header"><button type="button" class="close" data-dismiss="modal">&times;</button><h4 class="modal-title">Bibtex</h4></div><div class="modal-body">
<pre>@article{zhao:DFA:HBM2021,
 author = {Haichao Zhao and Jian Cheng and Tao Liu and Jiyang Jiang and Forrest Koch and Perminder S. Sachdev and Peter J. Basser and Wei Wen and for the Alzheimer's Disease Neuroimaging Initiative},
 doi = {10.1002/hbm.25628},
 journal = {Human Brain Mapping},
 pdf = {https://onlinelibrary.wiley.com/doi/pdfdirect/10.1002/hbm.25628},
 title = {Orientational changes of white matter fibers in Alzheimer's disease and amnestic mild cognitive impairment},
 url = {https://onlinelibrary.wiley.com/doi/abs/10.1002/hbm.25628},
 year = {2021}
}</pre> </div><div class="modal-footer"><button type="button" class="btn btn-default" data-dismiss="modal">Close</button></div></div></div></div>
<div class="modal fade" id="abs-zhao-DFA-HBM2021" role="dialog"><div class="modal-dialog"><div class="modal-content"><div class="modal-header"><button type="button" class="close" data-dismiss="modal">&times;</button><h4 class="modal-title">Abstract</h4></div><div class="modal-body">
<pre>Abstract White matter abnormalities represent early
neuropathological events in neurodegenerative diseases such as
Alzheimer's disease (AD), investigating these white matter
alterations would likely provide valuable insights into pathological
changes over the course of AD. Using a novel mathematical framework
called “Director Field Analysis” (DFA), we investigated the
geometric microstructural properties (i.e., splay, bend, twist, and
total distortion) in the orientation of white matter fibers in AD,
amnestic mild cognitive impairment (aMCI), and cognitively normal
(CN) individuals from the Alzheimer's Disease Neuroimaging
Initiative 2 database. Results revealed that AD patients had
extensive orientational changes in the bilateral anterior thalamic
radiation, corticospinal tract, inferior and superior longitudinal
fasciculus, inferior fronto-occipital fasciculus, and uncinate
fasciculus in comparison with CN. We postulate that these
orientational changes of white matter fibers may be partially caused
by the expansion of lateral ventricle, white matter atrophy, and
gray matter atrophy in AD. In contrast, aMCI individuals showed
subtle orientational changes in the left inferior longitudinal
fasciculus and right uncinate fasciculus, which showed a significant
association with the cognitive performance, suggesting that these
regions may be preferential vulnerable to breakdown by
neurodegenerative brain disorders, thereby resulting in the
patients' cognitive impairment. To our knowledge, this article is
the first to examine geometric microstructural changes in the
orientation of white matter fibers in AD and aMCI. Our findings
demonstrate that the orientational information of white matter
fibers could provide novel insight into the underlying biological
and pathological changes in AD and aMCI.</pre> </div><div class="modal-footer"><button type="button" class="btn btn-default" data-dismiss="modal">Close</button></div></div></div></div>
</li><br>

<li>
<span class="author">Hao Guan, Chaoyue Wang, <b>Jian Cheng</b>, Jing Jing, Tao Liu*</span>,<br>
<span class="title">"A parallel attention-augmented bilinear network for early magnetic resonance imaging-based diagnosis of Alzheimer's disease"</span>,<br>
<span class="publisher">Human Brain Mapping (<b>HBM</b>)</span>, <span class="year">2021</span>.
<br><a target="_self" href="https://onlinelibrary.wiley.com/doi/pdf/10.1002/hbm.25685"><img src="Icons/pdf.gif" alt="[pdf]" style="width: 16px; height: 16px;"></a>&nbsp;
<a target="_self" href="https://dx.doi.org/10.1002/hbm.25685"><img src="Icons/www.gif" alt="[www]" style="width: 16px; height: 16px;"></a>&nbsp;
[<a type="button" data-toggle="modal" data-target="#bib-guan-HBM2021">bibtex</a>]&nbsp;
[<a type="button" data-toggle="modal" data-target="#abs-guan-HBM2021">abstract</a>]&nbsp;
[citations: <a target="_blank" href="https://scholar.google.com//citations?view_op=view_citation&hl=en&oe=ASCII&user=BARqXQ0AAAAJ&pagesize=100&sortby=pubdate&citation_for_view=BARqXQ0AAAAJ:P7Ujq4OLJYoC">38</a>]&nbsp;

<div class="modal fade" id="bib-guan-HBM2021" role="dialog"><div class="modal-dialog"><div class="modal-content"><div class="modal-header"><button type="button" class="close" data-dismiss="modal">&times;</button><h4 class="modal-title">Bibtex</h4></div><div class="modal-body">
<pre>@article{guan:HBM2021,
 author = {Hao Guan and Chaoyue Wang and Jian Cheng and Jing Jing and Tao Liu},
 doi = {10.1002/hbm.25685},
 journal = {Human Brain Mapping},
 pdf = {https://onlinelibrary.wiley.com/doi/pdf/10.1002/hbm.25685},
 publisher = {Wiley Online Library},
 title = {A parallel attention-augmented bilinear network for early magnetic resonance imaging-based diagnosis of Alzheimer's disease},
 url = {https://dx.doi.org/10.1002/hbm.25685},
 year = {2021}
}</pre> </div><div class="modal-footer"><button type="button" class="btn btn-default" data-dismiss="modal">Close</button></div></div></div></div>
<div class="modal fade" id="abs-guan-HBM2021" role="dialog"><div class="modal-dialog"><div class="modal-content"><div class="modal-header"><button type="button" class="close" data-dismiss="modal">&times;</button><h4 class="modal-title">Abstract</h4></div><div class="modal-body">
<pre>Structural magnetic resonance imaging (sMRI) can capture the spatial
patterns of brain atrophy in Alzheimer's disease (AD) and incipient
dementia. Recently, many sMRI-based deep learning methods have been
developed for AD diagnosis. Some of these methods utilize neural
networks to extract high-level representations on the basis of
handcrafted features, while others attempt to learn useful features
from brain regions proposed by a separate module. However, these
methods require considerable manual engineering. Their stepwise
training procedures would introduce cascading errors. Here, we
propose the parallel attention-augmented bilinear network, a novel
deep learning framework for AD diagnosis. Based on a 3D
convolutional neural network, the framework directly learns both
global and local features from sMRI scans without any prior
knowledge. The framework is lightweight and suitable for end-to-end
training. We evaluate the framework on two public datasets (ADNI-1
and ADNI-2) containing 1,340 subjects. On both the AD classification
and mild cognitive impairment conversion prediction tasks, our
framework achieves competitive results. Furthermore, we generate
heat maps that highlight discriminative areas for visual
interpretation. Experiments demonstrate the effectiveness of the
proposed framework when medical priors are unavailable or the
computing resources are limited. The proposed framework is general
for 3D medical image analysis with both efficiency and
interpretability.</pre> </div><div class="modal-footer"><button type="button" class="btn btn-default" data-dismiss="modal">Close</button></div></div></div></div>
</li><br>

<li>
<span class="author">Xinxin Li, Yu Zhao, Jiyang Jiang, <b>Jian Cheng</b>, Wanlin Zhu, Zhenzhou Wu, Jing Jing, Zhe Zhang, Wei Wen, Perminder S. Sachdev, Yongjun Wang, Tao Liu*, Zixiao Li*</span>,<br>
<span class="title">"White matter hyperintensities segmentation using an ensemble of neural networks"</span>,<br>
<span class="publisher">Human Brain Mapping (<b>HBM</b>)</span>, <span class="year">2021</span>.
<br><a target="_self" href="https://onlinelibrary.wiley.com/doi/pdf/10.1002/hbm.25695"><img src="Icons/pdf.gif" alt="[pdf]" style="width: 16px; height: 16px;"></a>&nbsp;
<a target="_self" href="https://onlinelibrary.wiley.com/doi/abs/10.1002/hbm.25695"><img src="Icons/www.gif" alt="[www]" style="width: 16px; height: 16px;"></a>&nbsp;
[<a type="button" data-toggle="modal" data-target="#bib-li-HBM2021">bibtex</a>]&nbsp;
[<a type="button" data-toggle="modal" data-target="#abs-li-HBM2021">abstract</a>]&nbsp;
[citations: <a target="_blank" href="https://scholar.google.com//citations?view_op=view_citation&hl=en&oe=ASCII&user=BARqXQ0AAAAJ&pagesize=100&sortby=pubdate&citation_for_view=BARqXQ0AAAAJ:nZcligLrVowC">37</a>]&nbsp;

<div class="modal fade" id="bib-li-HBM2021" role="dialog"><div class="modal-dialog"><div class="modal-content"><div class="modal-header"><button type="button" class="close" data-dismiss="modal">&times;</button><h4 class="modal-title">Bibtex</h4></div><div class="modal-body">
<pre>@article{li:HBM2021,
 author = {Xinxin Li and Yu Zhao and Jiyang Jiang and Jian Cheng and Wanlin Zhu and Zhenzhou Wu and Jing Jing and Zhe Zhang and Wei Wen and Perminder S. Sachdev and Yongjun Wang and Tao Liu and Zixiao Li},
 doi = {10.1002/hbm.25695},
 journal = {Human Brain Mapping},
 pdf = {https://onlinelibrary.wiley.com/doi/pdf/10.1002/hbm.25695},
 title = {White matter hyperintensities segmentation using an ensemble of neural networks},
 url = {https://onlinelibrary.wiley.com/doi/abs/10.1002/hbm.25695},
 year = {2021}
}</pre> </div><div class="modal-footer"><button type="button" class="btn btn-default" data-dismiss="modal">Close</button></div></div></div></div>
<div class="modal fade" id="abs-li-HBM2021" role="dialog"><div class="modal-dialog"><div class="modal-content"><div class="modal-header"><button type="button" class="close" data-dismiss="modal">&times;</button><h4 class="modal-title">Abstract</h4></div><div class="modal-body">
<pre>Abstract White matter hyperintensities (WMHs) represent the most
common neuroimaging marker of cerebral small vessel disease (CSVD).
The volume and location of WMHs are important clinical measures. We
present a pipeline using deep fully convolutional network and
ensemble models, combining U-Net, SE-Net, and multi-scale features,
to automatically segment WMHs and estimate their volumes and
locations. We evaluated our method in two datasets: a clinical
routine dataset comprising 60 patients (selected from Chinese
National Stroke Registry, CNSR) and a research dataset composed of
60 patients (selected from MICCAI WMH Challenge, MWC). The
performance of our pipeline was compared with four freely available
methods: LGA, LPA, UBO detector, and U-Net, in terms of a variety of
metrics. Additionally, to access the model generalization ability,
another research dataset comprising 40 patients (from Older
Australian Twins Study and Sydney Memory and Aging Study, OSM), was
selected and tested. The pipeline achieved the best performance in
both research dataset and the clinical routine dataset with DSC
being significantly higher than other methods (p < .001), reaching
.833 and .783, respectively. The results of model generalization
ability showed that the model trained on the research dataset
(DSC = 0.736) performed higher than that trained on the clinical
dataset (DSC = 0.622). Our method outperformed widely used pipelines
in WMHs segmentation. This system could generate both image and text
outputs for whole brain, lobar and anatomical automatic labeling
WMHs. Additionally, software and models of our method are made
publicly available at https://www.nitrc.org/projects/what\_v1.</pre> </div><div class="modal-footer"><button type="button" class="btn btn-default" data-dismiss="modal">Close</button></div></div></div></div>
</li><br>

<li>
<span class="author">Haichao Zhao, <b>Jian Cheng</b>, Jiyang Jiang, Lijun Zuo, Wanlin Zhu, Wei Wen, Perminder Sachdev, Yongjun Wang, Tao Liu*, Zixiao Li*</span>,<br>
<span class="title">"Geometric microstructural damage of white matter with functional compensation in post-stroke"</span>,<br>
<span class="publisher">Neuropsychologia</span>, pp. 107980, <span class="year">2021</span>.
<br>
<a target="_self" href="https://dx.doi.org/j.neuropsychologia.2021.107980"><img src="Icons/www.gif" alt="[www]" style="width: 16px; height: 16px;"></a>&nbsp;
[<a type="button" data-toggle="modal" data-target="#bib-zhao-DFA-2021">bibtex</a>]&nbsp;
[<a type="button" data-toggle="modal" data-target="#abs-zhao-DFA-2021">abstract</a>]&nbsp;
[citations: <a target="_blank" href="https://scholar.google.com//citations?view_op=view_citation&hl=en&oe=ASCII&user=BARqXQ0AAAAJ&pagesize=100&sortby=pubdate&citation_for_view=BARqXQ0AAAAJ:F9fV5C73w3QC">13</a>]&nbsp;

<div class="modal fade" id="bib-zhao-DFA-2021" role="dialog"><div class="modal-dialog"><div class="modal-content"><div class="modal-header"><button type="button" class="close" data-dismiss="modal">&times;</button><h4 class="modal-title">Bibtex</h4></div><div class="modal-body">
<pre>@article{zhao:DFA:2021,
 author = {Haichao Zhao and Jian Cheng and Jiyang Jiang and Lijun Zuo and Wanlin Zhu and Wei Wen and Perminder Sachdev and Yongjun Wang and Tao Liu and Zixiao Li},
 doi = {j.neuropsychologia.2021.107980},
 journal = {Neuropsychologia},
 pages = {107980},
 publisher = {Elsevier},
 title = {Geometric microstructural damage of white matter with functional compensation in post-stroke},
 url = {https://dx.doi.org/j.neuropsychologia.2021.107980},
 year = {2021}
}</pre> </div><div class="modal-footer"><button type="button" class="btn btn-default" data-dismiss="modal">Close</button></div></div></div></div>
<div class="modal fade" id="abs-zhao-DFA-2021" role="dialog"><div class="modal-dialog"><div class="modal-content"><div class="modal-header"><button type="button" class="close" data-dismiss="modal">&times;</button><h4 class="modal-title">Abstract</h4></div><div class="modal-body">
<pre>Background and purpose  Subcortical ischemic stroke usually leads to
the geometric microstructural changes in the orientation of peri-
infarct white matter fiber. We conducted the study to determine the
microstructural changes in the white matter fiber orientation in
post stroke patients with and without cognitive impairment (PSCI,
NPSCI), and to investigate the impact of peri-infarct white matter
damage on the morphology and functional connectivity of their
projective cerebral regions. Methods  A novel mathematical framework
called Director Field Analysis (DFA) was applied to study the
microstructural changes in the orientation of white matter fiber in
PSCI (n = 23), NPSCI (n = 17), and cognitively normal (CN, n = 29)
individuals. Results  PSCI patients had extensive abnormalities in
the orientation of white matter fiber in the corpus callosum,
bilateral internal capsule, external capsule, forceps major, forceps
minor, and corticospinal tract in comparison with NPSCI and CN.
NPSCI patients also showed significant increases in bend and twist
of white matter fiber orientation in the internal capsule in
comparison with CN. Seed-based functional connectivity analysis
showed that peri-infarct white matter deficits indicate a
significant impact on functional connectivity with related cortical
regions, suggesting the coexistence of impairment and compensation
in post-stroke. In addition, these peri-infarct white matter damages
and abnormal functional connectivity were significantly correlated
with cognitive scores. Machine learning model also indicated that
these changes in white matter fiber orientation and functional
connectivity can predict the cognitive status in post-stroke.
Conclusions  Post-stroke patients experienced pathological damage in
the orientation of peri-infarct white matter fiber. The peri-infarct
white matter damage may further induce the abnormal functional
connectivity in projective cerebral regions. These degenerations of
peri-infarct white matter fiber and associated functional
connectivity changes may mediate the cognitive impairment in post-
stroke.</pre> </div><div class="modal-footer"><button type="button" class="btn btn-default" data-dismiss="modal">Close</button></div></div></div></div>
</li><br>

</ol><h2><a name="year2020"></a>2020</h2>

<ol reversed>

<li>
<span class="author">Hao Liu, Tao Liu*, Jiyang Jiang, <b>Jian Cheng</b>, Yan Liu, Daqing Li, Chao Dong, Haijun Niu, Shuyu Li, Jicong Zhang, Henry Brodaty, Perminder Sachdev, Wei Wen</span>,<br>
<span class="title">"Differential longitudinal changes in structural complexity and volumetric measures in community-dwelling older individuals"</span>,<br>
<span class="publisher">Neurobiology of Aging</span>, vol. 91, pp. 26&ndash;35, <span class="year">2020</span>.
<br>
<a target="_self" href="https://dx.doi.org/10.1016/j.neurobiolaging.2020.02.023"><img src="Icons/www.gif" alt="[www]" style="width: 16px; height: 16px;"></a>&nbsp;
[<a type="button" data-toggle="modal" data-target="#bib-liu-NoA2020">bibtex</a>]&nbsp;
[<a type="button" data-toggle="modal" data-target="#abs-liu-NoA2020">abstract</a>]&nbsp;
[citations: <a target="_blank" href="https://scholar.google.com//citations?view_op=view_citation&hl=en&oe=ASCII&user=BARqXQ0AAAAJ&pagesize=100&sortby=pubdate&citation_for_view=BARqXQ0AAAAJ:HeT0ZceujKMC">19</a>]&nbsp;

<div class="modal fade" id="bib-liu-NoA2020" role="dialog"><div class="modal-dialog"><div class="modal-content"><div class="modal-header"><button type="button" class="close" data-dismiss="modal">&times;</button><h4 class="modal-title">Bibtex</h4></div><div class="modal-body">
<pre>@article{liu:NoA2020,
 author = {Hao Liu and Tao Liu and Jiyang Jiang and Jian Cheng and Yan Liu and Daqing Li and Chao Dong and Haijun Niu and Shuyu Li and Jicong Zhang and Henry Brodaty and Perminder Sachdev and Wei Wen},
 doi = {10.1016/j.neurobiolaging.2020.02.023},
 journal = {Neurobiology of Aging},
 pages = {26-35},
 publisher = {Elsevier},
 title = {Differential longitudinal changes in structural complexity and volumetric measures in community-dwelling older individuals},
 url = {https://dx.doi.org/10.1016/j.neurobiolaging.2020.02.023},
 volume = {91},
 year = {2020}
}</pre> </div><div class="modal-footer"><button type="button" class="btn btn-default" data-dismiss="modal">Close</button></div></div></div></div>
<div class="modal fade" id="abs-liu-NoA2020" role="dialog"><div class="modal-dialog"><div class="modal-content"><div class="modal-header"><button type="button" class="close" data-dismiss="modal">&times;</button><h4 class="modal-title">Abstract</h4></div><div class="modal-body">
<pre>Fractal geometry provides a method of analyzing natural and
especially biological morphologies. To investigate the relationship
between the complexity measure, which is indexed as fractal
dimensionality (FD), and the traditional Euclidean metrics, such as
the volume and thickness, of the brain in older age, we analyzed 483
MRI scans of 161 community-dwelling, nondemented individuals aged
70–90 years at the baseline and their 2-year and 6-year follow-ups.
We quantified changes in neuroimaging metrics in cortical lobes and
subcortical structures and investigated the effects of age, sex,
hemisphere, and education on FD. We also analyzed the mediating
effects of these metrics for further investigation. FD showed
significant age-related decline in all structures, and its
trajectory was best modeled quadratically in the bilateral frontal,
parietal, and occipital lobes, as well as in the bilateral caudate,
putamen, hippocampus, amygdala, and accumbens. FD showed specific
mediating effects on aging and cognitive decline in some regions.
Our findings suggest that FD is reliable yet shows a different
pattern of decline compared with volumetric measures.</pre> </div><div class="modal-footer"><button type="button" class="btn btn-default" data-dismiss="modal">Close</button></div></div></div></div>
</li><br>

<li>
<span class="author">Qiongge Li#, Chao Dong#, Tao Liu*, Xiaodan Chen, Alistair Perry, Jiyang Jiang, <b>Jian Cheng</b>, Haijun Niu, Nicole A. Kochan, Henry Brodaty, Perminder S. Sachdev, Wei Wen</span>,<br>
<span class="title">"Longitudinal Changes in Whole-Brain Functional Connectivity Strength Patterns and the Relationship With the Global Cognitive Decline in Older Adults"</span>,<br>
<span class="publisher">Frontiers in Aging Neuroscience</span>, vol. 12, pp. 71, <span class="year">2020</span>.
<br><a target="_self" href="https://www.frontiersin.org/articles/10.3389/fnagi.2020.00071/pdf"><img src="Icons/pdf.gif" alt="[pdf]" style="width: 16px; height: 16px;"></a>&nbsp;
<a target="_self" href="https://dx.doi.org/10.3389/fnagi.2020.00071"><img src="Icons/www.gif" alt="[www]" style="width: 16px; height: 16px;"></a>&nbsp;
[<a type="button" data-toggle="modal" data-target="#bib-li-fan2020">bibtex</a>]&nbsp;
[<a type="button" data-toggle="modal" data-target="#abs-li-fan2020">abstract</a>]&nbsp;
[citations: <a target="_blank" href="https://scholar.google.com//citations?view_op=view_citation&hl=en&oe=ASCII&user=BARqXQ0AAAAJ&pagesize=100&sortby=pubdate&citation_for_view=BARqXQ0AAAAJ:yMeIxYmEMEAC">30</a>]&nbsp;

<div class="modal fade" id="bib-li-fan2020" role="dialog"><div class="modal-dialog"><div class="modal-content"><div class="modal-header"><button type="button" class="close" data-dismiss="modal">&times;</button><h4 class="modal-title">Bibtex</h4></div><div class="modal-body">
<pre>@article{li:fan2020,
 author = {Qiongge Li and Chao Dong and Tao Liu and Xiaodan Chen and Alistair Perry and Jiyang Jiang and Jian Cheng and Haijun Niu and Nicole A. Kochan and Henry Brodaty and Perminder S. Sachdev and Wei Wen},
 doi = {10.3389/fnagi.2020.00071},
 journal = {Frontiers in Aging Neuroscience},
 pages = {71},
 pdf = {https://www.frontiersin.org/articles/10.3389/fnagi.2020.00071/pdf},
 publisher = {Frontiers},
 title = {Longitudinal Changes in Whole-Brain Functional Connectivity Strength Patterns and the Relationship With the Global Cognitive Decline in Older Adults},
 url = {https://dx.doi.org/10.3389/fnagi.2020.00071},
 volume = {12},
 year = {2020}
}</pre> </div><div class="modal-footer"><button type="button" class="btn btn-default" data-dismiss="modal">Close</button></div></div></div></div>
<div class="modal fade" id="abs-li-fan2020" role="dialog"><div class="modal-dialog"><div class="modal-content"><div class="modal-header"><button type="button" class="close" data-dismiss="modal">&times;</button><h4 class="modal-title">Abstract</h4></div><div class="modal-body">
<pre>Aging is associated with changes in brain functional patterns as
well as cognition. The present research sought to investigate
longitudinal changes in whole brain functional connectivity strength
(FCS) and cognitive performance scores in very old cognitively
unimpaired individuals. We studied 34 cognitively normal elderly
individuals at both baseline and 4-year follow-up (baseline age = 78
± 3.14 years) with resting-state functional magnetic resonance
imaging (r-fMRI), structural MRI scans, and neuropsychological
assessments conducted. Voxel-based whole brain FCS was calculated
and we found that bilateral superior parietal and medial frontal
regions showed decreased FCS, while the supplementary motor area
(SMA) and insula showed increased FCS with age, along with a
decrease in bilateral prefrontal cortical thickness. The changes of
FCS in left precuneus were associated with an aging-related decline
in global cognition. Taken together, our results suggest changes in
FCS with aging with the precuneus as a hub and this may underlie
changes in global cognition that accompany aging. These findings
help better understand the normal aging mechanism.</pre> </div><div class="modal-footer"><button type="button" class="btn btn-default" data-dismiss="modal">Close</button></div></div></div></div>
</li><br>

<li>
<span class="author">Jing Zhang, Zixiao Li, Xingxing Cao, Lijun Zuo, Wei Wen, Wanlin Zhu, Jiyang Jiang, <b>Jian Cheng</b>, Perminder Sachdev, Tao Liu*, Yongjun Wang*</span>,<br>
<span class="title">"Altered Prefrontal--Basal Ganglia Effective Connectivity in Patients With Poststroke Cognitive Impairment"</span>,<br>
<span class="publisher">Frontiers in Neurology</span>, vol. 11, <span class="year">2020</span>.
<br><a target="_self" href="https://www.frontiersin.org/articles/10.3389/fneur.2020.577482/pdf"><img src="Icons/pdf.gif" alt="[pdf]" style="width: 16px; height: 16px;"></a>&nbsp;
<a target="_self" href="https://dx.doi.org/10.3389/fneur.2020.577482"><img src="Icons/www.gif" alt="[www]" style="width: 16px; height: 16px;"></a>&nbsp;
[<a type="button" data-toggle="modal" data-target="#bib-zhang-FN2020">bibtex</a>]&nbsp;
[<a type="button" data-toggle="modal" data-target="#abs-zhang-FN2020">abstract</a>]&nbsp;
[citations: <a target="_blank" href="https://scholar.google.com//citations?view_op=view_citation&hl=en&oe=ASCII&user=BARqXQ0AAAAJ&pagesize=100&sortby=pubdate&citation_for_view=BARqXQ0AAAAJ:a3BOlSfXSfwC">14</a>]&nbsp;

<div class="modal fade" id="bib-zhang-FN2020" role="dialog"><div class="modal-dialog"><div class="modal-content"><div class="modal-header"><button type="button" class="close" data-dismiss="modal">&times;</button><h4 class="modal-title">Bibtex</h4></div><div class="modal-body">
<pre>@article{zhang:FN2020,
 author = {Jing Zhang and Zixiao Li and Xingxing Cao and Lijun Zuo and Wei Wen and Wanlin Zhu and Jiyang Jiang and Jian Cheng and Perminder Sachdev and Tao Liu and Yongjun Wang},
 doi = {10.3389/fneur.2020.577482},
 journal = {Frontiers in Neurology},
 pdf = {https://www.frontiersin.org/articles/10.3389/fneur.2020.577482/pdf},
 publisher = {Frontiers Media SA},
 title = {Altered Prefrontal--Basal Ganglia Effective Connectivity in Patients With Poststroke Cognitive Impairment},
 url = {https://dx.doi.org/10.3389/fneur.2020.577482},
 volume = {11},
 year = {2020}
}</pre> </div><div class="modal-footer"><button type="button" class="btn btn-default" data-dismiss="modal">Close</button></div></div></div></div>
<div class="modal fade" id="abs-zhang-FN2020" role="dialog"><div class="modal-dialog"><div class="modal-content"><div class="modal-header"><button type="button" class="close" data-dismiss="modal">&times;</button><h4 class="modal-title">Abstract</h4></div><div class="modal-body">
<pre>We investigated the association between poststroke cognitive
impairment and a specific effective network connectivity in the
prefrontal–basal ganglia circuit. The resting-state effective
connectivity of this circuit was modeled by employing spectral
dynamic causal modeling in 11 poststroke patients with cognitive
impairment (PSCI), 8 poststroke patients without cognitive
impairment (non-PSCI) at baseline and 3-month follow-up, and 28
healthy controls. Our results showed that different neuronal models
of effective connectivity in the prefrontal–basal ganglia circuit
were observed among healthy controls, non-PSCI, and PSCI patients.
Additional connected paths (extra paths) appeared in the neuronal
models of stroke patients compared with healthy controls. Moreover,
changes were detected in the extra paths of non-PSCI between
baseline and 3-month follow-up poststroke, indicating reorganization
in the ipsilesional hemisphere and suggesting potential compensatory
changes in the contralesional hemisphere. Furthermore, the
connectivity strengths of the extra paths from the contralesional
ventral anterior nucleus of thalamus to caudate correlated
significantly with cognitive scores in non-PSCI and PSCI patients.
These suggest that the neuronal model of effective connectivity of
the prefrontal–basal ganglia circuit may be sensitive to stroke-
induced cognitive decline, and it could be a biomarker for
poststroke cognitive impairment 3 months poststroke. Importantly,
contralesional brain regions may play an important role in
functional compensation of cognitive decline.</pre> </div><div class="modal-footer"><button type="button" class="btn btn-default" data-dismiss="modal">Close</button></div></div></div></div>
</li><br>

<li>
<span class="author">Ziyang Liu#, <b>Jian Cheng</b>#, Haogang Zhu, Jicong Zhang, Tao Liu</span>,<br>
<span class="title">"Brain Age Estimation from MRI Using a Two-Stage Cascade Network with Ranking Loss"</span>,<br>
<span class="publisher">International Conference on Medical Image Computing and Computer-Assisted Intervention (<b>MICCAI</b>'20)</span>, pp. 198&ndash;207, <span class="year">2020</span>.
<br>
<a target="_self" href="https://dx.doi.org/10.1007/978-3-030-59728-3_20"><img src="Icons/www.gif" alt="[www]" style="width: 16px; height: 16px;"></a>&nbsp;
[<a type="button" data-toggle="modal" data-target="#bib-liu-MICCAI2020">bibtex</a>]&nbsp;
[<a type="button" data-toggle="modal" data-target="#abs-liu-MICCAI2020">abstract</a>]&nbsp;
[<a target="_self" href="https://github.com/Milan-BUAA/TSAN-brain-age-estimation">code</a>]&nbsp;
[citations: <a target="_blank" href="https://scholar.google.com//citations?view_op=view_citation&hl=en&oe=ASCII&user=BARqXQ0AAAAJ&pagesize=100&sortby=pubdate&citation_for_view=BARqXQ0AAAAJ:r_AWSJRzSzQC">9</a>]&nbsp;
(<span class="hlnote0">Ziyang Liu and Jian Cheng contributed equally.</span>)&nbsp;

<div class="modal fade" id="bib-liu-MICCAI2020" role="dialog"><div class="modal-dialog"><div class="modal-content"><div class="modal-header"><button type="button" class="close" data-dismiss="modal">&times;</button><h4 class="modal-title">Bibtex</h4></div><div class="modal-body">
<pre>@inproceedings{liu:MICCAI2020,
 author = {Ziyang Liu and Jian Cheng and Haogang Zhu and Jicong Zhang and Tao Liu},
 booktitle = {International Conference on Medical Image Computing and Computer-Assisted Intervention (MICCAI'20)},
 doi = {10.1007/978-3-030-59728-3_20},
 organization = {Springer},
 pages = {198-207},
 title = {Brain Age Estimation from MRI Using a Two-Stage Cascade Network with Ranking Loss},
 url = {https://dx.doi.org/10.1007/978-3-030-59728-3_20},
 year = {2020}
}</pre> </div><div class="modal-footer"><button type="button" class="btn btn-default" data-dismiss="modal">Close</button></div></div></div></div>
<div class="modal fade" id="abs-liu-MICCAI2020" role="dialog"><div class="modal-dialog"><div class="modal-content"><div class="modal-header"><button type="button" class="close" data-dismiss="modal">&times;</button><h4 class="modal-title">Abstract</h4></div><div class="modal-body">
<pre>As age increases, human brains will be aged, and people tend to
experience cognitive decline with a higher risk of neuro-
degenerative disease and dementia. Recently, it was reported that
deep neural networks, e.g., 3D convolutional neural networks (CNN),
are able to predict chronological age accurately in healthy people
from their T1-weighted magnetic resonance images (MRI). The
predicted age, called as “brain age” or “brain predicted age”, could
be a biomarker of the brain ageing process. In this paper, we
propose a novel 3D convolutional network, called as two-stage-age-
net (TSAN), for brain age estimation from T1-weighted MRI data.
Compared with the state-of-the-art CNN by Cole et al., TSAN has
several improvements: 1) TSAN uses a two-stage cascade architecture,
where the first network is to estimate a discretized age range, then
the second network is to further estimate the brain age more
accurately; 2) Besides using the traditional mean square error (MSE)
loss between chronological and estimated ages, TSAN considers two
additional novel ranking losses, based on paired samples and a batch
of samples, for regularizing the training process; 3) TSAN uses
densely connected paths to combine feature maps with different
scales; 4) TSAN considers gender labels as input features for the
network, considering brains of male and female age differently. The
proposed TSAN was validated in three public datasets. The
experiments showed that TSAN could provide accurate brain age
estimation in healthy subjects, yielding a mean absolute error (MAE)
of 2.428, and a Pearson’s correlation coefficient (PCC) of 0.985,
between the estimated and the chronological ages.</pre> </div><div class="modal-footer"><button type="button" class="btn btn-default" data-dismiss="modal">Close</button></div></div></div></div>
</li><br>

</ol><h2><a name="year2018"></a>2018</h2>

<ol reversed>

<li>
<span class="author"><b>Jian Cheng</b>*, Dinggang Shen, Pew-Thian Yap*, Peter J. Basser*</span>,<br>
<span class="title">"Single- and Multiple-Shell Uniform Sampling Schemes for Diffusion MRI Using Spherical Codes"</span>,<br>
<span class="publisher">IEEE Transactions on Medical Imaging (<b>TMI</b>)</span>, vol. 37, no. 1, pp. 185&ndash;199, <span class="year">2018</span>.
<br><a target="_self" href="https://arxiv.org/pdf/1706.06682.pdf"><img src="Icons/pdf.gif" alt="[pdf]" style="width: 16px; height: 16px;"></a>&nbsp;
<a target="_self" href="https://dx.doi.org/10.1109/TMI.2017.2756072"><img src="Icons/www.gif" alt="[www]" style="width: 16px; height: 16px;"></a>&nbsp;
[<a type="button" data-toggle="modal" data-target="#bib-cheng-TMI2017">bibtex</a>]&nbsp;
[<a type="button" data-toggle="modal" data-target="#abs-cheng-TMI2017">abstract</a>]&nbsp;
[<a target="_self" href="https://arxiv.org/abs/1706.06682">arxiv</a>]&nbsp;
[<a target="_self" href="https://diffusionmritool.github.io/tutorial_qspacesampling.html">project</a>]&nbsp;
[citations: <a target="_blank" href="https://scholar.google.com//citations?view_op=view_citation&hl=en&oe=ASCII&user=BARqXQ0AAAAJ&pagesize=100&sortby=pubdate&citation_for_view=BARqXQ0AAAAJ:ODE9OILHJdcC">42</a>]&nbsp;

<div class="modal fade" id="bib-cheng-TMI2017" role="dialog"><div class="modal-dialog"><div class="modal-content"><div class="modal-header"><button type="button" class="close" data-dismiss="modal">&times;</button><h4 class="modal-title">Bibtex</h4></div><div class="modal-body">
<pre>@article{cheng:TMI2017,
 author = {Jian Cheng and Dinggang Shen and Pew-Thian Yap and Peter J. Basser},
 doi = {10.1109/TMI.2017.2756072},
 journal = {IEEE Transactions on Medical Imaging},
 number = {1},
 pages = {185-199},
 pdf = {https://arxiv.org/pdf/1706.06682.pdf},
 publisher = {IEEE},
 title = {Single- and Multiple-Shell Uniform Sampling Schemes for Diffusion MRI Using Spherical Codes},
 url = {https://dx.doi.org/10.1109/TMI.2017.2756072},
 volume = {37},
 year = {2018}
}</pre> </div><div class="modal-footer"><button type="button" class="btn btn-default" data-dismiss="modal">Close</button></div></div></div></div>
<div class="modal fade" id="abs-cheng-TMI2017" role="dialog"><div class="modal-dialog"><div class="modal-content"><div class="modal-header"><button type="button" class="close" data-dismiss="modal">&times;</button><h4 class="modal-title">Abstract</h4></div><div class="modal-body">
<pre>In diffusion MRI (dMRI), a good sampling scheme is important for
efficient acquisition and robust reconstruction. Diffusion weighted
signal is normally acquired on single or multiple shells in q-space.
Signal samples are typically distributed uniformly on different
shells to make them invariant to the orientation of structures
within tissue, or the laboratory coordinate frame. The Electrostatic
Energy Minimization (EEM) method, originally proposed for single
shell sampling scheme in dMRI, was recently generalized to multi-
shell schemes, called Generalized EEM (GEEM). GEEM has been
successfully used in the Human Connectome Project (HCP). However,
EEM does not directly address the goal of optimal sampling, i.e.,
achieving large angular separation between sampling points. In this
paper, we propose a more natural formulation, called Spherical Code
(SC), to directly maximize the minimal angle between different
samples in single or multiple shells. We consider not only
continuous problems to design single or multiple shell sampling
schemes, but also discrete problems to uniformly extract sub-sampled
schemes from an existing single or multiple shell scheme, and to
order samples in an existing scheme. We propose five algorithms to
solve the above problems, including an incremental SC (ISC), a
sophisticated greedy algorithm called Iterative Maximum Overlap
Construction (IMOC), an 1-Opt greedy method, a Mixed Integer Linear
Programming (MILP) method, and a Constrained Non-Linear Optimization
(CNLO) method. To our knowledge, this is the first work to use the
SC formulation for single or multiple shell sampling schemes in
dMRI. Experimental results indicate that SC methods obtain larger
angular separation and better rotational invariance than the state-
of-the-art EEM and GEEM. The related codes and a tutorial have been
released in DMRITool.</pre> </div><div class="modal-footer"><button type="button" class="btn btn-default" data-dismiss="modal">Close</button></div></div></div></div>
</li><br>

<li>
<span class="author"><b>Jian Cheng</b>*, Peter J. Basser*</span>,<br>
<span class="title">"Director Field Analysis (DFA): Exploring Local White Matter Geometric Structure in diffusion MRI"</span>,<br>
<span class="publisher">Medical Image Analysis (<b>MedIA</b>)</span>, vol. 43, pp. 112&ndash;128, <span class="year">2018</span>.
<br><a target="_self" href="https://arxiv.org/pdf/1706.01862.pdf"><img src="Icons/pdf.gif" alt="[pdf]" style="width: 16px; height: 16px;"></a>&nbsp;
<a target="_self" href="https://dx.doi.org/10.1016/j.media.2017.10.003"><img src="Icons/www.gif" alt="[www]" style="width: 16px; height: 16px;"></a>&nbsp;
[<a type="button" data-toggle="modal" data-target="#bib-cheng-MedIA2017">bibtex</a>]&nbsp;
[<a type="button" data-toggle="modal" data-target="#abs-cheng-MedIA2017">abstract</a>]&nbsp;
[<a target="_self" href="https://arxiv.org/abs/1706.01862">arxiv</a>]&nbsp;
[citations: <a target="_blank" href="https://scholar.google.com//citations?view_op=view_citation&hl=en&oe=ASCII&user=BARqXQ0AAAAJ&pagesize=100&sortby=pubdate&citation_for_view=BARqXQ0AAAAJ:DJbcl8HfkQkC">20</a>]&nbsp;

<div class="modal fade" id="bib-cheng-MedIA2017" role="dialog"><div class="modal-dialog"><div class="modal-content"><div class="modal-header"><button type="button" class="close" data-dismiss="modal">&times;</button><h4 class="modal-title">Bibtex</h4></div><div class="modal-body">
<pre>@article{cheng:MedIA2017,
 author = {Jian Cheng and Peter J. Basser},
 doi = {10.1016/j.media.2017.10.003},
 journal = {Medical Image Analysis},
 pages = {112-128},
 pdf = {https://arxiv.org/pdf/1706.01862.pdf},
 publisher = {Elsevier},
 title = {Director Field Analysis (DFA): Exploring Local White Matter Geometric Structure in diffusion MRI},
 url = {https://dx.doi.org/10.1016/j.media.2017.10.003},
 volume = {43},
 year = {2018}
}</pre> </div><div class="modal-footer"><button type="button" class="btn btn-default" data-dismiss="modal">Close</button></div></div></div></div>
<div class="modal fade" id="abs-cheng-MedIA2017" role="dialog"><div class="modal-dialog"><div class="modal-content"><div class="modal-header"><button type="button" class="close" data-dismiss="modal">&times;</button><h4 class="modal-title">Abstract</h4></div><div class="modal-body">
<pre>In Diffusion Tensor Imaging (DTI) or High Angular Resolution
Diffusion Imaging (HARDI), a tensor field or a spherical function
field (e.g., an orientation distribution function field), can be
estimated from measured diffusion weighted images. In this paper,
inspired by the microscopic theoretical treatment of phases in
liquid crystals, we introduce a novel mathematical framework, called
Director Field Analysis (DFA), to study local geometric structural
information of white matter based on the reconstructed tensor field
or spherical function field: 1) We propose a set of mathematical
tools to process general director data, which consists of dyadic
tensors that have orientations but no direction. 2) We propose
Orientational Order (OO) and Orientational Dispersion (OD) indices
to describe the degree of alignment and dispersion of a spherical
function in a single voxel or in a region, respectively; 3) We also
show how to construct a local orthogonal coordinate frame in each
voxel exhibiting anisotropic diffusion; 4) Finally, we define three
indices to describe three types of orientational distortion (splay,
bend, and twist) in a local spatial neighborhood, and a total
distortion index to describe distortions of all three types. To our
knowledge, this is the first work to quantitatively describe
orientational distortion (splay, bend, and twist) in general
spherical function fields from DTI or HARDI data. The proposed DFA
and its related mathematical tools can be used to process not only
diffusion MRI data but also general director field data, and the
proposed scalar indices are useful for detecting local geometric
changes of white matter for voxel-based or tract-based analysis in
both DTI and HARDI acquisitions. The related codes and a tutorial
for DFA will be released in DMRITool.</pre> </div><div class="modal-footer"><button type="button" class="btn btn-default" data-dismiss="modal">Close</button></div></div></div></div>
</li><br>

<li>
<span class="author">Yongqin Zhang, Feng Shi, <b>Jian Cheng</b>, Li Wang, Pew-Thian Yap, Dinggang Shen</span>,<br>
<span class="title">"Longitudinally Guided Super-Resolution of Neonatal Brain Magnetic Resonance Images"</span>,<br>
<span class="publisher">IEEE Transactions on Cybernetics (<b>TCYB</b>)</span>, <span class="year">2018</span>.
<br><a target="_self" href="https://www.researchgate.net/profile/Yongqin_Zhang2/publication/322348018_Longitudinally_Guided_Super-Resolution_of_Neonatal_Brain_Magnetic_Resonance_Images/links/5a846d6b4585159152b7fffe/Longitudinally-Guided-Super-Resolution-of-Neonatal-Brain-Magnetic-Resonance-Images.pdf"><img src="Icons/pdf.gif" alt="[pdf]" style="width: 16px; height: 16px;"></a>&nbsp;
<a target="_self" href="https://dx.doi.org/10.1109/TCYB.2017.2786161"><img src="Icons/www.gif" alt="[www]" style="width: 16px; height: 16px;"></a>&nbsp;
[<a type="button" data-toggle="modal" data-target="#bib-zhang-TCYB2018">bibtex</a>]&nbsp;
[<a type="button" data-toggle="modal" data-target="#abs-zhang-TCYB2018">abstract</a>]&nbsp;
[citations: <a target="_blank" href="https://scholar.google.com//citations?view_op=view_citation&hl=en&oe=ASCII&user=BARqXQ0AAAAJ&pagesize=100&sortby=pubdate&citation_for_view=BARqXQ0AAAAJ:fFSKOagxvKUC">43</a>]&nbsp;

<div class="modal fade" id="bib-zhang-TCYB2018" role="dialog"><div class="modal-dialog"><div class="modal-content"><div class="modal-header"><button type="button" class="close" data-dismiss="modal">&times;</button><h4 class="modal-title">Bibtex</h4></div><div class="modal-body">
<pre>@article{zhang:TCYB2018,
 author = {Yongqin Zhang and Feng Shi and Jian Cheng and Li Wang and Pew-Thian Yap and Dinggang Shen},
 doi = {10.1109/TCYB.2017.2786161},
 journal = {IEEE Transactions on Cybernetics},
 pdf = {https://www.researchgate.net/profile/Yongqin_Zhang2/publication/322348018_Longitudinally_Guided_Super-Resolution_of_Neonatal_Brain_Magnetic_Resonance_Images/links/5a846d6b4585159152b7fffe/Longitudinally-Guided-Super-Resolution-of-Neonatal-Brain-Magnetic-Resonance-Images.pdf},
 title = {Longitudinally Guided Super-Resolution of Neonatal Brain Magnetic Resonance Images},
 url = {https://dx.doi.org/10.1109/TCYB.2017.2786161},
 year = {2018}
}</pre> </div><div class="modal-footer"><button type="button" class="btn btn-default" data-dismiss="modal">Close</button></div></div></div></div>
<div class="modal fade" id="abs-zhang-TCYB2018" role="dialog"><div class="modal-dialog"><div class="modal-content"><div class="modal-header"><button type="button" class="close" data-dismiss="modal">&times;</button><h4 class="modal-title">Abstract</h4></div><div class="modal-body">
<pre>Neonatal magnetic resonance (MR) images typically have low spatial
resolution and insufficient tissue contrast. Interpolation methods
are commonly used to upsample the images for the subsequent
analysis. However, the resulting images are often blurry and
susceptible to partial volume effects. In this paper, we propose a
novel longitudinally guided super-resolution algorithm for neonatal
images. This is motivated by the fact that anatomical structures
evolve slowly and smoothly as the brain develops after birth. We
propose a strategy involving longitudinal regularization, similar to
bilateral filtering, in combination with lowrank and total variation
constraints to solve the ill-posed inverse problem associated with
image super-resolution. Experimental results on neonatal MR images
demonstrate that the proposed algorithm recovers clear structural
details and outperforms stateof-the-art methods both qualitatively
and quantitatively.</pre> </div><div class="modal-footer"><button type="button" class="btn btn-default" data-dismiss="modal">Close</button></div></div></div></div>
</li><br>

<li>
<span class="author"><b>Jian Cheng</b>, Tao Liu, Feng Shi, Ruiliang Bai, Jicong Zhang, Haogang Zhu, Dacheng Tao, Peter J Basser</span>,<br>
<span class="title">"On Quantifying Local Geometric Structures of Fiber Tracts"</span>,<br>
<span class="publisher">Medical Image Computing and Computer-Assisted Intervention (<b>MICCAI</b>'18)</span>, pp. 392&ndash;400, <span class="year">2018</span>.
<br><a target="_self" href="https://hal.archives-ouvertes.fr/hal-02098668/document"><img src="Icons/pdf.gif" alt="[pdf]" style="width: 16px; height: 16px;"></a>&nbsp;
<a target="_self" href="https://dx.doi.org/10.1007/978-3-030-00931-1_45"><img src="Icons/www.gif" alt="[www]" style="width: 16px; height: 16px;"></a>&nbsp;
[<a type="button" data-toggle="modal" data-target="#bib-cheng-MICCAI2018">bibtex</a>]&nbsp;
[<a type="button" data-toggle="modal" data-target="#abs-cheng-MICCAI2018">abstract</a>]&nbsp;
[citations: <a target="_blank" href="https://scholar.google.com//citations?view_op=view_citation&hl=en&oe=ASCII&user=BARqXQ0AAAAJ&pagesize=100&sortby=pubdate&citation_for_view=BARqXQ0AAAAJ:KbBQZpvPDL4C">2</a>]&nbsp;

<div class="modal fade" id="bib-cheng-MICCAI2018" role="dialog"><div class="modal-dialog"><div class="modal-content"><div class="modal-header"><button type="button" class="close" data-dismiss="modal">&times;</button><h4 class="modal-title">Bibtex</h4></div><div class="modal-body">
<pre>@inproceedings{cheng:MICCAI2018,
 author = {Jian Cheng and Tao Liu and Feng Shi and Ruiliang Bai and Jicong Zhang and Haogang Zhu and Dacheng Tao and Peter J Basser},
 booktitle = {Medical Image Computing and Computer-Assisted Intervention (MICCAI'18)},
 doi = {10.1007/978-3-030-00931-1_45},
 hal_id = {hal-02098668},
 organization = {Springer},
 pages = {392-400},
 pdf = {https://hal.archives-ouvertes.fr/hal-02098668/document},
 title = {On Quantifying Local Geometric Structures of Fiber Tracts},
 url = {https://dx.doi.org/10.1007/978-3-030-00931-1_45},
 year = {2018}
}</pre> </div><div class="modal-footer"><button type="button" class="btn btn-default" data-dismiss="modal">Close</button></div></div></div></div>
<div class="modal fade" id="abs-cheng-MICCAI2018" role="dialog"><div class="modal-dialog"><div class="modal-content"><div class="modal-header"><button type="button" class="close" data-dismiss="modal">&times;</button><h4 class="modal-title">Abstract</h4></div><div class="modal-body">
<pre>In diffusion MRI, fiber tracts, represented by densely distributed
3D curves, can be estimated from diffusion weighted images using
tractography. The spatial geometric structure of white matter fiber
tracts is known to be complex in human brain, but it carries
intrinsic information of human brain. In this paper, inspired by
studies of liquid crystals, we propose tract-based director field
analysis (tDFA) with total six rotationally invariant scalar indices
to quantify local geometric structures of fiber tracts. The
contributions of tDFA include: (1) We propose orientational order
(OO) and orientational dispersion (OD) indices to quantify the
degree of alignment and dispersion of fiber tracts; (2) We define
the local orthogonal frame for a set of unoriented curves, which is
proved to be a generalization of the Frenet frame defined for a
single oriented curve; (3) With the local orthogonal frame, we
propose splay, bend, and twist indices to quantify three types of
orientational distortion of local fiber tracts, and a total
distortion index to describe distortions of all three types. The
proposed tDFA for fiber tracts is a generalization of the voxel-
based DFA (vDFA) which was recently proposed for a spherical
function field (i.e., an ODF field). To our knowledge, this is the
first work to quantify orientational distortion (splay, bend, twist,
and total distortion) of fiber tracts. Experiments show that the
proposed scalar indices are useful descriptors of local geometric
structures to visualize and analyze fiber tracts.</pre> </div><div class="modal-footer"><button type="button" class="btn btn-default" data-dismiss="modal">Close</button></div></div></div></div>
</li><br>

</ol><h2><a name="year2017"></a>2017</h2>

<ol reversed>

<li>
<span class="author">Hao Guan, Tao Liu*, Jiyang Jiang, Dacheng Tao, Jicong Zhang, Haijun Niu, Wanlin Zhu, Yilong Wang, <b>Jian Cheng</b>*, Nicole A. Kochan, Henry Brodaty, Perminder Sachdev, Wei Wen</span>,<br>
<span class="title">"Classifying MCI subtypes in community-dwelling elderly using cross-sectional and longitudinal MRI-based biomarkers"</span>,<br>
<span class="publisher">Frontiers in Aging Neuroscience</span>, vol. 9, pp. 309, <span class="year">2017</span>.
<br><a target="_self" href="http://journal.frontiersin.org/article/10.3389/fnagi.2017.00309/pdf"><img src="Icons/pdf.gif" alt="[pdf]" style="width: 16px; height: 16px;"></a>&nbsp;
<a target="_self" href="https://dx.doi.org/10.3389/fnagi.2017.00309"><img src="Icons/www.gif" alt="[www]" style="width: 16px; height: 16px;"></a>&nbsp;
[<a type="button" data-toggle="modal" data-target="#bib-tao-cheng-FNAGI2017">bibtex</a>]&nbsp;
[<a type="button" data-toggle="modal" data-target="#abs-tao-cheng-FNAGI2017">abstract</a>]&nbsp;
[citations: <a target="_blank" href="https://scholar.google.com//citations?view_op=view_citation&hl=en&oe=ASCII&user=BARqXQ0AAAAJ&pagesize=100&sortby=pubdate&citation_for_view=BARqXQ0AAAAJ:OR75R8vi5nAC">29</a>]&nbsp;

<div class="modal fade" id="bib-tao-cheng-FNAGI2017" role="dialog"><div class="modal-dialog"><div class="modal-content"><div class="modal-header"><button type="button" class="close" data-dismiss="modal">&times;</button><h4 class="modal-title">Bibtex</h4></div><div class="modal-body">
<pre>@article{tao:cheng:FNAGI2017,
 author = {Hao Guan and Tao Liu and Jiyang Jiang and Dacheng Tao and Jicong Zhang and Haijun Niu and Wanlin Zhu and Yilong Wang and Jian Cheng and Nicole A. Kochan and Henry Brodaty and Perminder Sachdev and Wei Wen},
 doi = {10.3389/fnagi.2017.00309},
 journal = {Frontiers in Aging Neuroscience},
 pages = {309},
 pdf = {http://journal.frontiersin.org/article/10.3389/fnagi.2017.00309/pdf},
 title = {Classifying MCI subtypes in community-dwelling elderly using cross-sectional and longitudinal MRI-based biomarkers},
 url = {https://dx.doi.org/10.3389/fnagi.2017.00309},
 volume = {9},
 year = {2017}
}</pre> </div><div class="modal-footer"><button type="button" class="btn btn-default" data-dismiss="modal">Close</button></div></div></div></div>
<div class="modal fade" id="abs-tao-cheng-FNAGI2017" role="dialog"><div class="modal-dialog"><div class="modal-content"><div class="modal-header"><button type="button" class="close" data-dismiss="modal">&times;</button><h4 class="modal-title">Abstract</h4></div><div class="modal-body">
<pre>Amnestic MCI (aMCI) and non-amnestic MCI (naMCI) are considered to
differ in etiology and outcome. Accurately classifying MCI into
meaningful subtypes would enable early intervention with targeted
treatment. In this study, we employed structural magnetic resonance
imaging (MRI) for MCI subtype classification. This was carried out
in a sample of 184 community-dwelling individuals (aged 73–85
years). Cortical surface based measurements were computed from
longitudinal and cross-sectional scans. By introducing a feature
selection algorithm, we identified a set of discriminative features,
and further investigated the temporal patterns of these features. A
voting classifier was trained and evaluated via 10 iterations of
cross-validation. The best classification accuracies achieved were:
77% (naMCI vs. aMCI), 81% (aMCI vs. cognitively normal (CN)) and 70%
(naMCI vs. CN). The best results for differentiating aMCI from naMCI
were achieved with baseline features. Hippocampus, amygdala and
frontal pole were found to be most discriminative for classifying
MCI subtypes. Additionally, we observed the dynamics of
classification of several MRI biomarkers. Learning the dynamics of
atrophy may aid in the development of better biomarkers, as it may
track the progression of cognitive impairment.</pre> </div><div class="modal-footer"><button type="button" class="btn btn-default" data-dismiss="modal">Close</button></div></div></div></div>
</li><br>

<li>
<span class="author"><b>Jian Cheng</b>, Peter J. Basser</span>,<br>
<span class="title">"Director Field Analysis to Explore Local White Matter Geometric Structure in diffusion MRI"</span>,<br>
<span class="publisher">International Conference on Information Processing in Medical Imaging (<b>IPMI</b>'17)</span>, vol. 10265, pp. 427&ndash;439, <span class="year">2017</span>.
<br><a target="_self" href="https://hal.archives-ouvertes.fr/hal-01511573/document"><img src="Icons/pdf.gif" alt="[pdf]" style="width: 16px; height: 16px;"></a>&nbsp;
<a target="_self" href="https://dx.doi.org/10.1007/978-3-319-59050-9_34"><img src="Icons/www.gif" alt="[www]" style="width: 16px; height: 16px;"></a>&nbsp;
[<a type="button" data-toggle="modal" data-target="#bib-cheng-IPMI2017">bibtex</a>]&nbsp;
[<a type="button" data-toggle="modal" data-target="#abs-cheng-IPMI2017">abstract</a>]&nbsp;
(<span class="hlnote">Oral presentation; Turned to be a poster due to conflicting travel plans.</span>)&nbsp;

<div class="modal fade" id="bib-cheng-IPMI2017" role="dialog"><div class="modal-dialog"><div class="modal-content"><div class="modal-header"><button type="button" class="close" data-dismiss="modal">&times;</button><h4 class="modal-title">Bibtex</h4></div><div class="modal-body">
<pre>@inproceedings{cheng:IPMI2017,
 author = {Jian Cheng and Peter J. Basser},
 booktitle = {International Conference on Information Processing in Medical Imaging (IPMI'17)},
 doi = {10.1007/978-3-319-59050-9_34},
 hal_id = {hal-01511573},
 pages = {427-439},
 pdf = {https://hal.archives-ouvertes.fr/hal-01511573/document},
 title = {Director Field Analysis to Explore Local White Matter Geometric Structure in diffusion MRI},
 url = {https://dx.doi.org/10.1007/978-3-319-59050-9_34},
 volume = {10265},
 year = {2017}
}</pre> </div><div class="modal-footer"><button type="button" class="btn btn-default" data-dismiss="modal">Close</button></div></div></div></div>
<div class="modal fade" id="abs-cheng-IPMI2017" role="dialog"><div class="modal-dialog"><div class="modal-content"><div class="modal-header"><button type="button" class="close" data-dismiss="modal">&times;</button><h4 class="modal-title">Abstract</h4></div><div class="modal-body">
<pre>In diffusion MRI, a tensor field or a spherical function field,
e.g., an Orientation Distribution Function (ODF) field, are
estimated from measured diffusion weighted images.  In this paper,
inspired by microscopic theoretical treatment of phases in liquid
crystals,  we introduce a novel mathematical framework, called
Director Field Analysis (DFA), to study local geometric structural
information of white matter from the estimated tensor field or
spherical function field.   1) We propose Orientational Order (OO)
and Orientational Dispersion (OD) indices to describe the degree of
alignment and dispersion of a spherical function in each voxel;  2)
We estimate a local orthogonal coordinate frame in each voxel with
anisotropic diffusion;  3) Finally, we define three indices to
describe three types of orientational distortion (splay, bend, and
twist) in a local spatial neighborhood, and a total distortion index
to describe distortions of all three types.  To our knowledge, this
is the first work to \emph{quantitatively} describe orientational
distortion (splay, bend, and twist) in diffusion MRI.  The proposed
scalar indices are useful to detect local geometric changes of white
matter for voxel-based or tract-based analysis in both DTI and HARDI
acquisitions.</pre> </div><div class="modal-footer"><button type="button" class="btn btn-default" data-dismiss="modal">Close</button></div></div></div></div>
</li><br>

<li>
<span class="author"><b>Jian Cheng</b>, Peter J. Basser</span>,<br>
<span class="title">"Exploring Local White Matter Geometric Structure in diffusion MRI Using Director Field Analysis"</span>,<br>
<span class="publisher">Scientific Meeting and Exhibition of the ISMRM (ISMRM'17)</span>, <span class="year">2017</span>.
<br><a target="_self" href="https://hal.archives-ouvertes.fr/hal-01511564/document"><img src="Icons/pdf.gif" alt="[pdf]" style="width: 16px; height: 16px;"></a>&nbsp;
<a target="_self" href="http://submissions.mirasmart.com/ISMRM2017/ViewSubmissionPublic.aspx?sei=YLBxyBRee"><img src="Icons/www.gif" alt="[www]" style="width: 16px; height: 16px;"></a>&nbsp;
[<a type="button" data-toggle="modal" data-target="#bib-cheng-ISMRM2017">bibtex</a>]&nbsp;

<div class="modal fade" id="bib-cheng-ISMRM2017" role="dialog"><div class="modal-dialog"><div class="modal-content"><div class="modal-header"><button type="button" class="close" data-dismiss="modal">&times;</button><h4 class="modal-title">Bibtex</h4></div><div class="modal-body">
<pre>@conference{cheng:ISMRM2017,
 author = {Jian Cheng and Peter J. Basser},
 booktitle = {Scientific Meeting and Exhibition of the ISMRM (ISMRM'17)},
 hal_id = {hal-01511564},
 pdf = {https://hal.archives-ouvertes.fr/hal-01511564/document},
 title = {Exploring Local White Matter Geometric Structure in diffusion MRI Using Director Field Analysis},
 url = {http://submissions.mirasmart.com/ISMRM2017/ViewSubmissionPublic.aspx?sei=YLBxyBRee},
 year = {2017}
}</pre> </div><div class="modal-footer"><button type="button" class="btn btn-default" data-dismiss="modal">Close</button></div></div></div></div>
</li><br>

</ol><h2><a name="year2016"></a>2016</h2>

<ol reversed>

<li>
<span class="author"><b>Jian Cheng</b>, Hongtu Zhu</span>,<br>
<span class="title">"Diffusion Magnetic Resonance Imaging (dMRI)"</span>,<br>
<span class="publisher">Handbook of Neuroimaging Data Analysis</span>, <span class="year">2016</span>.
<br><a target="_self" href="https://hal.archives-ouvertes.fr/hal-01430475/document"><img src="Icons/pdf.gif" alt="[pdf]" style="width: 16px; height: 16px;"></a>&nbsp;
<a target="_self" href="https://hal.archives-ouvertes.fr/hal-01430475"><img src="Icons/www.gif" alt="[www]" style="width: 16px; height: 16px;"></a>&nbsp;
[<a type="button" data-toggle="modal" data-target="#bib-cheng-handbook2016">bibtex</a>]&nbsp;
[<a type="button" data-toggle="modal" data-target="#abs-cheng-handbook2016">abstract</a>]&nbsp;
[citations: <a target="_blank" href="https://scholar.google.com//citations?view_op=view_citation&hl=en&oe=ASCII&user=BARqXQ0AAAAJ&pagesize=100&sortby=pubdate&citation_for_view=BARqXQ0AAAAJ:QYdC8u9Cj1oC">5</a>]&nbsp;

<div class="modal fade" id="bib-cheng-handbook2016" role="dialog"><div class="modal-dialog"><div class="modal-content"><div class="modal-header"><button type="button" class="close" data-dismiss="modal">&times;</button><h4 class="modal-title">Bibtex</h4></div><div class="modal-body">
<pre>@inbook{cheng:handbook2016,
 author = {Jian Cheng and Hongtu Zhu},
 booktitle = {Handbook of Neuroimaging Data Analysis},
 hal_id = {hal-01430475},
 pdf = {https://hal.archives-ouvertes.fr/hal-01430475/document},
 title = {Diffusion Magnetic Resonance Imaging (dMRI)},
 url = {https://hal.archives-ouvertes.fr/hal-01430475},
 year = {2016}
}</pre> </div><div class="modal-footer"><button type="button" class="btn btn-default" data-dismiss="modal">Close</button></div></div></div></div>
<div class="modal fade" id="abs-cheng-handbook2016" role="dialog"><div class="modal-dialog"><div class="modal-content"><div class="modal-header"><button type="button" class="close" data-dismiss="modal">&times;</button><h4 class="modal-title">Abstract</h4></div><div class="modal-body">
<pre>Since the 1980s, diffusion magnetic resonance imaging (dMRI) as a
magnetic resonance imaging (MRI) technique has been widely used to
track the effective diffusion of water molecules, which is hindered
by many obstacles (e.g., fibers or membranes), in the human brain in
vivo. Because water molecules tend to diffuse slowly across white
matter fibers and diffuse fast along such fibers, the use of dMRI to
track water diffusion allows one to map the microstructure and
organization of those white matter pathways (17). Quantitatively
measuring the diffusion process is critical for a quantitative
assessment of the integrity of anatomical connectivity in white
matter and its association with brain functional connectivity. Its
clinical applications include normal brain maturation and aging,
cerebral ischemia, multiple sclerosis, epilepsy, metabolic
disorders, and brain tumors, among many others. Although there are
several nice review papers and monographies on dMRI (15, 5, 101,
81), this chapter was written for the readers who are interested in
the theoretical underpinning of various mathematical and statistical
methods associated with dMRI. Due to limitations of space, we are
unable to cite all important papers in the dMRI literature.</pre> </div><div class="modal-footer"><button type="button" class="btn btn-default" data-dismiss="modal">Close</button></div></div></div></div>
</li><br>

<li>
<span class="author">Ruiliang Bai, Dan Benjamini, <b>Jian Cheng</b>, Peter J. Basser</span>,<br>
<span class="title">"Fast, accurate 2D-MR relaxation exchange spectroscopy (REXSY): Beyond compressed sensing"</span>,<br>
<span class="publisher">The Journal of Chemical Physics</span>, vol. 145, no. 15, pp. 154202, <span class="year">2016</span>.
<br><a target="_self" href="https://www.researchgate.net/profile/Ruiliang_Bai2/publication/309463759_14964144/links/5811be3008ae009606be8db0.pdf"><img src="Icons/pdf.gif" alt="[pdf]" style="width: 16px; height: 16px;"></a>&nbsp;
<a target="_self" href="https://dx.doi.org/10.1063/1.4964144"><img src="Icons/www.gif" alt="[www]" style="width: 16px; height: 16px;"></a>&nbsp;
[<a type="button" data-toggle="modal" data-target="#bib-bai_JCP2016">bibtex</a>]&nbsp;
[<a type="button" data-toggle="modal" data-target="#abs-bai_JCP2016">abstract</a>]&nbsp;
[citations: <a target="_blank" href="https://scholar.google.com//citations?view_op=view_citation&hl=en&oe=ASCII&user=BARqXQ0AAAAJ&pagesize=100&sortby=pubdate&citation_for_view=BARqXQ0AAAAJ:9c2xU6iGI7YC">31</a>]&nbsp;

<div class="modal fade" id="bib-bai_JCP2016" role="dialog"><div class="modal-dialog"><div class="modal-content"><div class="modal-header"><button type="button" class="close" data-dismiss="modal">&times;</button><h4 class="modal-title">Bibtex</h4></div><div class="modal-body">
<pre>@article{bai_JCP2016,
 author = {Ruiliang Bai and Dan Benjamini and Jian Cheng and Peter J. Basser},
 doi = {10.1063/1.4964144},
 journal = {The Journal of Chemical Physics},
 number = {15},
 pages = {154202},
 pdf = {https://www.researchgate.net/profile/Ruiliang_Bai2/publication/309463759_14964144/links/5811be3008ae009606be8db0.pdf},
 publisher = {AIP Publishing},
 title = {Fast, accurate 2D-MR relaxation exchange spectroscopy (REXSY): Beyond compressed sensing},
 url = {https://dx.doi.org/10.1063/1.4964144},
 volume = {145},
 year = {2016}
}</pre> </div><div class="modal-footer"><button type="button" class="btn btn-default" data-dismiss="modal">Close</button></div></div></div></div>
<div class="modal fade" id="abs-bai_JCP2016" role="dialog"><div class="modal-dialog"><div class="modal-content"><div class="modal-header"><button type="button" class="close" data-dismiss="modal">&times;</button><h4 class="modal-title">Abstract</h4></div><div class="modal-body">
<pre>Previously, we showed that compressive or compressed sensing (CS)
can be used to reduce significantly the data required to obtain
2D-NMR relaxation and diffusion spectra when they are sparse or well
localized. In some cases, an order of magnitude fewer uniformly
sampled data were required to reconstruct 2D-MR spectra of
comparable quality. Nonetheless, this acceleration may still not be
sufficient to make 2D-MR spectroscopy practicable for many important
applications, such as studying time-varying exchange processes in
swelling gels or drying paints, in living tissue in response to
various biological or biochemical challenges, and particularly for
in vivo MRI applications. A recently introduced framework, marginal
distributions constrained optimization (MADCO), tremendously
accelerates such 2D acquisitions by using a priori obtained 1D
marginal distribution as powerful constraints when 2D spectra are
reconstructed. Here we exploit one important intrinsic property of
the 2D-MR relaxation exchange spectra: the fact that the 1D marginal
distributions of each 2D-MR relaxation exchange spectrum in both
dimensions are equal and can be rapidly estimated from a single
Carr-Purcell-Meiboom-Gill (CPMG) or inversion recovery prepared CPMG
measurement. We extend the MADCO framework by further proposing to
use the 1D marginal distributions to inform the subsequent 2D data-
sampling scheme, concentrating measurements where spectral peaks are
present and reducing them where they are not. In this way we achieve
compression or acceleration that is an order of magnitude greater
than that in our previous CS method while providing data in
reconstructed 2D-MR spectral maps of comparable quality,
demonstrated using several simulated and real 2D T2-T2 experimental
data. This method, which can be called "informed compressed
sensing", is extendable to other 2D- and even ND-MR exchange
spectroscopy.</pre> </div><div class="modal-footer"><button type="button" class="btn btn-default" data-dismiss="modal">Close</button></div></div></div></div>
</li><br>

<li>
<span class="author">Shangbang Rao, Joseph G. Ibrahim, <b>Jian Cheng</b>, Pew-Thian Yap, Hongtu Zhu*</span>,<br>
<span class="title">"SR-HARDI: Spatially Regularizing High Angular Resolution Diffusion Imaging"</span>,<br>
<span class="publisher">Journal of Computational and Graphical Statistics</span>, vol. 25, no. 4, pp. 1195&ndash;1211, <span class="year">2016</span>.
<br><a target="_self" href="https://www.researchgate.net/profile/Jian_Cheng2/publication/283828288_SR-HARDI_Spatially_Regularizing_High_Angular_Resolution_Diffusion_Imaging/links/584bc62408aecb6bd8c285a1.pdf"><img src="Icons/pdf.gif" alt="[pdf]" style="width: 16px; height: 16px;"></a>&nbsp;
<a target="_self" href="https://dx.doi.org/10.1080/10618600.2015.1105750"><img src="Icons/www.gif" alt="[www]" style="width: 16px; height: 16px;"></a>&nbsp;
[<a type="button" data-toggle="modal" data-target="#bib-rao_2015">bibtex</a>]&nbsp;
[<a type="button" data-toggle="modal" data-target="#abs-rao_2015">abstract</a>]&nbsp;
[citations: <a target="_blank" href="https://scholar.google.com//citations?view_op=view_citation&hl=en&oe=ASCII&user=BARqXQ0AAAAJ&pagesize=100&sortby=pubdate&citation_for_view=BARqXQ0AAAAJ:IUKN3-7HHlwC">7</a>]&nbsp;

<div class="modal fade" id="bib-rao_2015" role="dialog"><div class="modal-dialog"><div class="modal-content"><div class="modal-header"><button type="button" class="close" data-dismiss="modal">&times;</button><h4 class="modal-title">Bibtex</h4></div><div class="modal-body">
<pre>@article{rao_2015,
 author = {Shangbang Rao and Joseph G. Ibrahim and Jian Cheng and Pew-Thian Yap and Hongtu Zhu},
 doi = {10.1080/10618600.2015.1105750},
 journal = {Journal of Computational and Graphical Statistics},
 number = {4},
 pages = {1195-1211},
 pdf = {https://www.researchgate.net/profile/Jian_Cheng2/publication/283828288_SR-HARDI_Spatially_Regularizing_High_Angular_Resolution_Diffusion_Imaging/links/584bc62408aecb6bd8c285a1.pdf},
 title = {SR-HARDI: Spatially Regularizing High Angular Resolution Diffusion Imaging},
 url = {https://dx.doi.org/10.1080/10618600.2015.1105750},
 volume = {25},
 year = {2016}
}</pre> </div><div class="modal-footer"><button type="button" class="btn btn-default" data-dismiss="modal">Close</button></div></div></div></div>
<div class="modal fade" id="abs-rao_2015" role="dialog"><div class="modal-dialog"><div class="modal-content"><div class="modal-header"><button type="button" class="close" data-dismiss="modal">&times;</button><h4 class="modal-title">Abstract</h4></div><div class="modal-body">
<pre>High angular resolution diffusion imaging (HARDI) has recently been
of great interest in mapping the orientation of intravoxel crossing
fibers, and such orientation information allows one to infer the
connectivity patterns prevalent among different brain regions and
possible changes in such connectivity over time for various
neurodegenerative and neuropsychiatric diseases. The aim of this
article is to propose a penalized multiscale adaptive regression
model (PMARM) framework to spatially and adaptively infer the
orientation distribution function (ODF) of water diffusion in
regions with complex fiber configurations. In PMARM, we reformulate
the HARDI imaging reconstruction as a weighted regularized least-
square regression (WRLSR) problem. Similarity and distance weights
are introduced to account for spatial smoothness of HARDI, while
preserving the unknown discontinuities (e.g., edges between white
matter and gray matter) of HARDI. The L1 penalty function is
introduced to ensure the sparse solutions of ODFs, while a scaled L1
weighted estimator is calculated to correct the bias introduced by
the L1 penalty at each voxel. In PMARM, we integrate the multiscale
adaptive regression models, the propagation-separation method, and
Lasso (least absolute shrinkage and selection operator) to
adaptively estimate ODFs across voxels. Experimental results
indicate that PMARM can reduce the angle detection errors on fiber
crossing area and provide more accurate reconstruction than standard
voxel-wise methods. Supplementary materials for this article are
available online.</pre> </div><div class="modal-footer"><button type="button" class="btn btn-default" data-dismiss="modal">Close</button></div></div></div></div>
</li><br>

</ol><h2><a name="year2015"></a>2015</h2>

<ol reversed>

<li>
<span class="author">Feng Shi#, <b>Jian Cheng</b>#, Li Wang, Pew-Thian Yap, Dinggang Shen*</span>,<br>
<span class="title">"LRTV: MR image super-resolution with low-rank and total variation regularizations"</span>,<br>
<span class="publisher">IEEE Transactions on Medical Imaging (<b>TMI</b>)</span>, vol. 34, no. 12, pp. 2459&ndash;2466, <span class="year">2015</span>.
<br><a target="_self" href="https://hal.archives-ouvertes.fr/hal-01154770/document"><img src="Icons/pdf.gif" alt="[pdf]" style="width: 16px; height: 16px;"></a>&nbsp;
<a target="_self" href="https://dx.doi.org/10.1109/TMI.2015.2437894"><img src="Icons/www.gif" alt="[www]" style="width: 16px; height: 16px;"></a>&nbsp;
[<a type="button" data-toggle="modal" data-target="#bib-shi_TMI2015">bibtex</a>]&nbsp;
[<a type="button" data-toggle="modal" data-target="#abs-shi_TMI2015">abstract</a>]&nbsp;
[<a target="_self" href="https://bitbucket.org/fengshi421/superresolutiontoolkit">code</a>]&nbsp;
[citations: <a target="_blank" href="https://scholar.google.com//citations?view_op=view_citation&hl=en&oe=ASCII&user=BARqXQ0AAAAJ&pagesize=100&sortby=pubdate&citation_for_view=BARqXQ0AAAAJ:XvxMoLDsR5gC">358</a>]&nbsp;
(<span class="hlnote0">F. Shi and J. Cheng contributed equally.</span>)&nbsp;

<div class="modal fade" id="bib-shi_TMI2015" role="dialog"><div class="modal-dialog"><div class="modal-content"><div class="modal-header"><button type="button" class="close" data-dismiss="modal">&times;</button><h4 class="modal-title">Bibtex</h4></div><div class="modal-body">
<pre>@article{shi_TMI2015,
 author = {Feng Shi and Jian Cheng and Li Wang and Pew-Thian Yap and Dinggang Shen},
 doi = {10.1109/TMI.2015.2437894},
 hal_id = {hal-01154770},
 journal = {IEEE Transactions on Medical Imaging},
 number = {12},
 pages = {2459-2466},
 pdf = {https://hal.archives-ouvertes.fr/hal-01154770/document},
 publisher = {IEEE},
 title = {LRTV: MR image super-resolution with low-rank and total variation regularizations},
 url = {https://dx.doi.org/10.1109/TMI.2015.2437894},
 volume = {34},
 year = {2015}
}</pre> </div><div class="modal-footer"><button type="button" class="btn btn-default" data-dismiss="modal">Close</button></div></div></div></div>
<div class="modal fade" id="abs-shi_TMI2015" role="dialog"><div class="modal-dialog"><div class="modal-content"><div class="modal-header"><button type="button" class="close" data-dismiss="modal">&times;</button><h4 class="modal-title">Abstract</h4></div><div class="modal-body">
<pre>Image super-resolution (SR) aims to recover high-resolution images
from their low-resolution counterparts for improving image analysis
and visualization. Interpolation methods, widely used for this
purpose, often result in images with blurred edges and blocking
effects. More advanced methods such as total variation (TV) retain
edge sharpness during image recovery. However, these methods only
utilize information from local neighborhoods, neglecting useful
information from remote voxels. In this paper, we propose a novel
image SR method that integrates both local and global information
for effective image recovery. This is achieved by, in addition to
TV, low-rank regularization that enables utilization of information
throughout the image. The optimization problem can be solved
effectively via alternating direction method of multipliers (ADMM).
Experiments on MR images of both adult and pediatric subjects
demonstrate that the proposed method enhances the details in the
recovered high-resolution images, and outperforms methods such as
the nearest-neighbor interpolation, cubic interpolation, iterative
back projection (IBP), non-local means (NLM), and TV-based up-
sampling.</pre> </div><div class="modal-footer"><button type="button" class="btn btn-default" data-dismiss="modal">Close</button></div></div></div></div>
</li><br>

<li>
<span class="author"><b>Jian Cheng</b>, Dinggang Shen, Pew-Thian Yap, Peter J. Basser</span>,<br>
<span class="title">"Tensorial Spherical Polar Fourier diffusion MRI with Optimal Dictionary Learning"</span>,<br>
<span class="publisher">International Conference on Medical Image Computing and Computer-Assisted Intervention (<b>MICCAI</b>'15)</span>, vol. 9349, pp. 174&ndash;182, <span class="year">2015</span>.
<br><a target="_self" href="https://hal.archives-ouvertes.fr/hal-01164809/document"><img src="Icons/pdf.gif" alt="[pdf]" style="width: 16px; height: 16px;"></a>&nbsp;
<a target="_self" href="https://dx.doi.org/10.1007/978-3-319-24553-9_22"><img src="Icons/www.gif" alt="[www]" style="width: 16px; height: 16px;"></a>&nbsp;
[<a type="button" data-toggle="modal" data-target="#bib-cheng_MICCAI2015">bibtex</a>]&nbsp;
[<a type="button" data-toggle="modal" data-target="#abs-cheng_MICCAI2015">abstract</a>]&nbsp;
[<a target="_self" href="https://diffusionmritool.github.io/tutorial_spfi.html">project</a>]&nbsp;
[citations: <a target="_blank" href="https://scholar.google.com//citations?view_op=view_citation&hl=en&oe=ASCII&user=BARqXQ0AAAAJ&pagesize=100&sortby=pubdate&citation_for_view=BARqXQ0AAAAJ:OTTXONDVkokC">13</a>]&nbsp;

<div class="modal fade" id="bib-cheng_MICCAI2015" role="dialog"><div class="modal-dialog"><div class="modal-content"><div class="modal-header"><button type="button" class="close" data-dismiss="modal">&times;</button><h4 class="modal-title">Bibtex</h4></div><div class="modal-body">
<pre>@inproceedings{cheng_MICCAI2015,
 author = {Jian Cheng and Dinggang Shen and Pew-Thian Yap and Peter J. Basser},
 booktitle = {International Conference on Medical Image Computing and Computer-Assisted Intervention (MICCAI'15)},
 doi = {10.1007/978-3-319-24553-9_22},
 organization = {Springer},
 pages = {174-182},
 pdf = {https://hal.archives-ouvertes.fr/hal-01164809/document},
 series = {LNCS},
 title = {Tensorial Spherical Polar Fourier diffusion MRI with Optimal Dictionary Learning},
 url = {https://dx.doi.org/10.1007/978-3-319-24553-9_22},
 volume = {9349},
 year = {2015}
}</pre> </div><div class="modal-footer"><button type="button" class="btn btn-default" data-dismiss="modal">Close</button></div></div></div></div>
<div class="modal fade" id="abs-cheng_MICCAI2015" role="dialog"><div class="modal-dialog"><div class="modal-content"><div class="modal-header"><button type="button" class="close" data-dismiss="modal">&times;</button><h4 class="modal-title">Abstract</h4></div><div class="modal-body">
<pre>High Angular Resolution Diffusion Imaging (HARDI) can characterize
complex white matter micro-structure, avoiding the Gaussian
diffusion assumption inherent in Diffusion Tensor Imaging (DTI).
However, HARDI methods normally require significantly more signal
measurements and a longer scan time than DTI, which limits its
clinical utility. By considering sparsity of the diffusion signal,
Compressed Sensing (CS) allows robust signal reconstruction from
relatively fewer samples, reducing the scanning time. A good
dictionary that sparsifies the signal is crucial for CS
reconstruction. In this paper, we propose a novel method called
Tensorial Spherical Polar Fourier Imaging (TSPFI) to recover
continuous diffusion signal and diffusion propagator by representing
the diffusion signal using an orthonormal TSPF basis. TSPFI is a
generalization of the existing model-based method DTI and the model-
free method SPFI. We also propose dictionary learning TSPFI (DL-
TSPFI) to learn an even sparser dictionary represented as a linear
combination of TSPF basis from continuous mixture of Gaussian
signals. The learning process is efficiently performed in a small
subspace of SPF coefficients, and the learned dictionary is proved
to be sparse for all mixture of Gaussian signals by adaptively
setting the tensor in TSPF basis. Then the learned DL-TSPF
dictionary is optimally and adaptively applied to different voxels
using DTI and a weighted LASSO for CS reconstruction. DL-TSPFI is a
generalization of DL-SPFI, by considering general adaptive tensor
setting instead of a scale value. The experiments demonstrated that
the learned DL-TSPF dictionary has a sparser representation and
lower reconstruction Root-Mean-Squared-Error (RMSE) than both the
original SPF basis and the DL-SPF dictionary.</pre> </div><div class="modal-footer"><button type="button" class="btn btn-default" data-dismiss="modal">Close</button></div></div></div></div>
</li><br>

<li>
<span class="author"><b>Jian Cheng</b>, Dinggang Shen, Pew-Thian Yap, Peter J. Basser</span>,<br>
<span class="title">"Novel single and multiple shell uniform sampling schemes for diffusion MRI using spherical codes"</span>,<br>
<span class="publisher">International Conference on Medical Image Computing and Computer-Assisted Intervention (<b>MICCAI</b>'15)</span>, vol. 9349, pp. 28&ndash;36, <span class="year">2015</span>.
<br><a target="_self" href="https://hal.archives-ouvertes.fr/hal-01154774/document"><img src="Icons/pdf.gif" alt="[pdf]" style="width: 16px; height: 16px;"></a>&nbsp;
<a target="_self" href="https://dx.doi.org/10.1007/978-3-319-24553-9_4"><img src="Icons/www.gif" alt="[www]" style="width: 16px; height: 16px;"></a>&nbsp;
[<a type="button" data-toggle="modal" data-target="#bib-cheng_MICCAI2015_sampling">bibtex</a>]&nbsp;
[<a type="button" data-toggle="modal" data-target="#abs-cheng_MICCAI2015_sampling">abstract</a>]&nbsp;
[<a target="_self" href="https://diffusionmritool.github.io/tutorial_qspacesampling.html">project</a>]&nbsp;
[citations: <a target="_blank" href="https://scholar.google.com//citations?view_op=view_citation&hl=en&oe=ASCII&user=BARqXQ0AAAAJ&pagesize=100&sortby=pubdate&citation_for_view=BARqXQ0AAAAJ:IRz6iEL74y4C">9</a>]&nbsp;
(<span class="hlnote">early accepted</span>)&nbsp;

<div class="modal fade" id="bib-cheng_MICCAI2015_sampling" role="dialog"><div class="modal-dialog"><div class="modal-content"><div class="modal-header"><button type="button" class="close" data-dismiss="modal">&times;</button><h4 class="modal-title">Bibtex</h4></div><div class="modal-body">
<pre>@inproceedings{cheng_MICCAI2015_sampling,
 author = {Jian Cheng and Dinggang Shen and Pew-Thian Yap and Peter J. Basser},
 booktitle = {International Conference on Medical Image Computing and Computer-Assisted Intervention (MICCAI'15)},
 doi = {10.1007/978-3-319-24553-9_4},
 organization = {Springer},
 pages = {28-36},
 pdf = {https://hal.archives-ouvertes.fr/hal-01154774/document},
 title = {Novel single and multiple shell uniform sampling schemes for diffusion MRI using spherical codes},
 url = {https://dx.doi.org/10.1007/978-3-319-24553-9_4},
 volume = {9349},
 year = {2015}
}</pre> </div><div class="modal-footer"><button type="button" class="btn btn-default" data-dismiss="modal">Close</button></div></div></div></div>
<div class="modal fade" id="abs-cheng_MICCAI2015_sampling" role="dialog"><div class="modal-dialog"><div class="modal-content"><div class="modal-header"><button type="button" class="close" data-dismiss="modal">&times;</button><h4 class="modal-title">Abstract</h4></div><div class="modal-body">
<pre>A good data sampling scheme is important for diffusion MRI
acquisition and reconstruction. Diffusion Weighted Imaging (DWI)
data is normally acquired on single or multiple shells in q-space.
The samples in different shells are typically distributed uniformly,
because they should be invariant to the orientation of structures
within tissue, or the laboratory coordinate frame. The Electrostatic
Energy Minimization (EEM) method, originally proposed for single
shell sampling scheme in dMRI by Jones et al., was recently
generalized to the multi-shell case, called generalized EEM (GEEM).
GEEM has been successfully used in the Human Connectome Project
(HCP). Recently, the Spherical Code (SC) concept was proposed to
maximize the minimal angle between different samples in single or
multiple shells, producing a larger angular separation and better
rotational invariance than the GEEM method. In this paper, we
propose two novel algorithms based on the SC concept: 1) an
efficient incremental constructive method, called Iterative Maximum
Overlap Construction (IMOC), to generate a sampling scheme on a
discretized sphere; 2) a constrained non-linear optimization (CNLO)
method to update a given initial scheme on the continuous sphere.
Compared to existing incremental estimation methods, IMOC obtains
schemes with much larger separation angles between samples, which
are very close to the best known solutions in single shell case.
Compared to the existing Riemannian gradient descent method, CNLO is
more robust and stable. Experiments demonstrated that the two
proposed methods provide larger separation angles and better
rotational invariance than the state-of-the-art GEEM and methods
based on the SC concept.</pre> </div><div class="modal-footer"><button type="button" class="btn btn-default" data-dismiss="modal">Close</button></div></div></div></div>
</li><br>

<li>
<span class="author"><b>Jian Cheng</b>, Dinggang Shen, Peter J. Basser, Pew-Thian Yap</span>,<br>
<span class="title">"Joint 6D k-q Space Compressed Sensing for Accelerated High Angular Resolution Diffusion MRI"</span>,<br>
<span class="publisher">International Conference on Information Processing in Medical Imaging (<b>IPMI</b>'15)</span>, vol. 9123, pp. 782&ndash;793, <span class="year">2015</span>.
<br><a target="_self" href="https://hal.archives-ouvertes.fr/hal-01356133/document"><img src="Icons/pdf.gif" alt="[pdf]" style="width: 16px; height: 16px;"></a>&nbsp;
<a target="_self" href="https://dx.doi.org/10.1007/978-3-319-19992-4_62"><img src="Icons/www.gif" alt="[www]" style="width: 16px; height: 16px;"></a>&nbsp;
[<a type="button" data-toggle="modal" data-target="#bib-cheng_IPMI2015">bibtex</a>]&nbsp;
[<a type="button" data-toggle="modal" data-target="#abs-cheng_IPMI2015">abstract</a>]&nbsp;
[citations: <a target="_blank" href="https://scholar.google.com//citations?view_op=view_citation&hl=en&oe=ASCII&user=BARqXQ0AAAAJ&pagesize=100&sortby=pubdate&citation_for_view=BARqXQ0AAAAJ:_5tno0g5mFcC">32</a>]&nbsp;

<div class="modal fade" id="bib-cheng_IPMI2015" role="dialog"><div class="modal-dialog"><div class="modal-content"><div class="modal-header"><button type="button" class="close" data-dismiss="modal">&times;</button><h4 class="modal-title">Bibtex</h4></div><div class="modal-body">
<pre>@inproceedings{cheng_IPMI2015,
 author = {Jian Cheng and Dinggang Shen and Peter J. Basser and Pew-Thian Yap},
 booktitle = {International Conference on Information Processing in Medical Imaging (IPMI'15)},
 doi = {10.1007/978-3-319-19992-4_62},
 organization = {Springer},
 pages = {782-793},
 pdf = {https://hal.archives-ouvertes.fr/hal-01356133/document},
 title = {Joint 6D k-q Space Compressed Sensing for Accelerated High Angular Resolution Diffusion MRI},
 url = {https://dx.doi.org/10.1007/978-3-319-19992-4_62},
 volume = {9123},
 year = {2015}
}</pre> </div><div class="modal-footer"><button type="button" class="btn btn-default" data-dismiss="modal">Close</button></div></div></div></div>
<div class="modal fade" id="abs-cheng_IPMI2015" role="dialog"><div class="modal-dialog"><div class="modal-content"><div class="modal-header"><button type="button" class="close" data-dismiss="modal">&times;</button><h4 class="modal-title">Abstract</h4></div><div class="modal-body">
<pre>High Angular Resolution Diffusion Imaging (HARDI) avoids the
Gaussian diffusion assumption that is inherent in Diffusion Tensor
Imaging (DTI), and is capable of characterizing complex white matter
micro-structure with greater precision. However, HARDI methods such
as Diffusion Spectrum Imaging (DSI) typically require significantly
more signal measurements than DTI, resulting in prohibitively long
scanning times. One of the goals in HARDI research is therefore to
improve estimation of quantities such as the Ensemble Average
Propagator (EAP) and the Orientation Distribution Function (ODF)
with a limited number of diffusion-weighted measurements. A popular
approach to this problem, Compressed Sensing (CS), affords highly
accurate signal reconstruction using significantly fewer (sub-
Nyquist) data points than required traditionally. Existing
approaches to CS diffusion MRI (CS-dMRI) mainly focus on applying CS
in the q-space of diffusion signal measurements and fail to take
into consideration information redundancy in the k-space. In this
paper, we propose a framework, called 6-Dimensional Compressed
Sensing diffusion MRI (6D-CS-dMRI), for reconstruction of the
diffusion signal and the EAP from data sub-sampled in both 3D
k-space and 3D q-space. To our knowledge, 6D-CS-dMRI is the first
work that applies compressed sensing in the full 6D k-q space and
reconstructs the diffusion signal in the full continuous q-space and
the EAP in continuous displacement space. Experimental results on
synthetic and real data demonstrate that, compared with full DSI
sampling in k-q space, 6D-CS-dMRI yields excellent diffusion signal
and EAP reconstruction with low root-mean-square error (RMSE) using
11 times less samples (3-fold reduction in k-space and 3.7-fold
reduction in q-space).</pre> </div><div class="modal-footer"><button type="button" class="btn btn-default" data-dismiss="modal">Close</button></div></div></div></div>
</li><br>

<li>
<span class="author">Feng Shi, <b>Jian Cheng</b>, Li Wang, Pew-Thian Yap, Dinggang Shen</span>,<br>
<span class="title">"Super-Resolution Reconstruction of Diffusion-Weighted Images Using 4D Low-Rank and Total Variation"</span>,<br>
<span class="publisher">MICCAI workshop on Computational Diffusion MRI (CDMRI'15)</span>, pp. 15&ndash;25, <span class="year">2015</span>.
<br><a target="_self" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5102512/pdf/nihms772323.pdf"><img src="Icons/pdf.gif" alt="[pdf]" style="width: 16px; height: 16px;"></a>&nbsp;
<a target="_self" href="https://dx.doi.org/10.1007/978-3-319-28588-7_2"><img src="Icons/www.gif" alt="[www]" style="width: 16px; height: 16px;"></a>&nbsp;
[<a type="button" data-toggle="modal" data-target="#bib-shi-CDMRI2015">bibtex</a>]&nbsp;
[<a type="button" data-toggle="modal" data-target="#abs-shi-CDMRI2015">abstract</a>]&nbsp;
[citations: <a target="_blank" href="https://scholar.google.com//citations?view_op=view_citation&hl=en&oe=ASCII&user=BARqXQ0AAAAJ&pagesize=100&sortby=pubdate&citation_for_view=BARqXQ0AAAAJ:HIFyuExEbWQC">17</a>]&nbsp;
(<span class="hlnote">Oral presentation</span>)&nbsp;

<div class="modal fade" id="bib-shi-CDMRI2015" role="dialog"><div class="modal-dialog"><div class="modal-content"><div class="modal-header"><button type="button" class="close" data-dismiss="modal">&times;</button><h4 class="modal-title">Bibtex</h4></div><div class="modal-body">
<pre>@inproceedings{shi:CDMRI2015,
 author = {Feng Shi and Jian Cheng and Li Wang and Pew-Thian Yap and Dinggang Shen},
 booktitle = {MICCAI workshop on Computational Diffusion MRI (CDMRI'15)},
 doi = {10.1007/978-3-319-28588-7_2},
 pages = {15-25},
 pdf = {https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5102512/pdf/nihms772323.pdf},
 publisher = {Springer},
 title = {Super-Resolution Reconstruction of Diffusion-Weighted Images Using 4D Low-Rank and Total Variation},
 url = {https://dx.doi.org/10.1007/978-3-319-28588-7_2},
 year = {2015}
}</pre> </div><div class="modal-footer"><button type="button" class="btn btn-default" data-dismiss="modal">Close</button></div></div></div></div>
<div class="modal fade" id="abs-shi-CDMRI2015" role="dialog"><div class="modal-dialog"><div class="modal-content"><div class="modal-header"><button type="button" class="close" data-dismiss="modal">&times;</button><h4 class="modal-title">Abstract</h4></div><div class="modal-body">
<pre>Diffusion-weighted imaging (DWI) provides invaluable information in
white matter microstructure and is widely applied in neurological
applications. However, DWI is largely limited by its relatively low
spatial resolution. In this paper, we propose an image post-
processing method, referred to as super-resolution reconstruction,
to estimate a high spatial resolution DWI from the input low-
resolution DWI, e.g., at a factor of 2. Instead of requiring
specially designed DWI acquisition of multiple shifted or orthogonal
scans, our method needs only a single DWI scan. To do that, we
propose to model both the blurring and downsampling effects in the
image degradation process where the low-resolution image is observed
from the latent high-resolution image, and recover the latent high-
resolution image with the help of two regularizations. The first
regularization is four-dimensional (4D) low-rank, proposed to gather
self-similarity information from both the spatial domain and the
diffusion domain of 4D DWI. The second regularization is total
variation, proposed to depress noise and preserve local structures
such as edges in the image recovery process. Extensive experiments
were performed on 20 subjects, and results show that the proposed
method is able to recover the fine details of white matter
structures, and outperform other approaches such as interpolation
methods, non-local means based upsampling, and total variation based
upsampling.</pre> </div><div class="modal-footer"><button type="button" class="btn btn-default" data-dismiss="modal">Close</button></div></div></div></div>
</li><br>

<li>
<span class="author"><b>Jian Cheng</b>, Dinggang Shen, Pew-Thian Yap, Peter J. Basser</span>,<br>
<span class="title">"Novel Single and Multiple Shell Gradient Sampling Schemes for Diffusion MRI Using Spherical Codes"</span>,<br>
<span class="publisher">23rd Scientific Meeting and Exhibition of the ISMRM (ISMRM'15)</span>, <span class="year">2015</span>.
<br><a target="_self" href="https://hal.archives-ouvertes.fr/hal-01154773/document"><img src="Icons/pdf.gif" alt="[pdf]" style="width: 16px; height: 16px;"></a>&nbsp;
<a target="_self" href="https://hal.archives-ouvertes.fr/hal-01154773"><img src="Icons/www.gif" alt="[www]" style="width: 16px; height: 16px;"></a>&nbsp;
[<a type="button" data-toggle="modal" data-target="#bib-cheng-ISMRM2015">bibtex</a>]&nbsp;

<div class="modal fade" id="bib-cheng-ISMRM2015" role="dialog"><div class="modal-dialog"><div class="modal-content"><div class="modal-header"><button type="button" class="close" data-dismiss="modal">&times;</button><h4 class="modal-title">Bibtex</h4></div><div class="modal-body">
<pre>@conference{cheng:ISMRM2015,
 author = {Jian Cheng and Dinggang Shen and Pew-Thian Yap and Peter J. Basser},
 booktitle = {23rd Scientific Meeting and Exhibition of the ISMRM (ISMRM'15)},
 hal_id = {hal-01154773},
 pdf = {https://hal.archives-ouvertes.fr/hal-01154773/document},
 title = {Novel Single and Multiple Shell Gradient Sampling Schemes for Diffusion MRI Using Spherical Codes},
 url = {https://hal.archives-ouvertes.fr/hal-01154773},
 year = {2015}
}</pre> </div><div class="modal-footer"><button type="button" class="btn btn-default" data-dismiss="modal">Close</button></div></div></div></div>
</li><br>

</ol><h2><a name="year2014"></a>2014</h2>

<ol reversed>

<li>
<span class="author"><b>Jian Cheng</b>*, Rachid Deriche, Tianzi Jiang, Dinggang Shen, Pew-Thian Yap*</span>,<br>
<span class="title">"Non-Negative Spherical Deconvolution (NNSD) for estimation of fiber Orientation Distribution Function in single-/multi-shell diffusion MRI"</span>,<br>
<span class="publisher"><b>NeuroImage</b></span>, vol. 101, pp. 750&ndash;764, <span class="year">2014</span>.
<br><a target="_self" href="https://pdfs.semanticscholar.org/fc76/327ac26d573283db98939119660588c2af6f.pdf"><img src="Icons/pdf.gif" alt="[pdf]" style="width: 16px; height: 16px;"></a>&nbsp;
<a target="_self" href="https://dx.doi.org/10.1016/j.neuroimage.2014.07.062"><img src="Icons/www.gif" alt="[www]" style="width: 16px; height: 16px;"></a>&nbsp;
[<a type="button" data-toggle="modal" data-target="#bib-cheng_NI2014">bibtex</a>]&nbsp;
[<a type="button" data-toggle="modal" data-target="#abs-cheng_NI2014">abstract</a>]&nbsp;
[citations: <a target="_blank" href="https://scholar.google.com//citations?view_op=view_citation&hl=en&oe=ASCII&user=BARqXQ0AAAAJ&pagesize=100&sortby=pubdate&citation_for_view=BARqXQ0AAAAJ:FAceZFleit8C">44</a>]&nbsp;
(<span class="hlnote2">Honorable Mention in ISBI HARDI Reconstruction Challenge in 2013; the best technique in terms of local comparison.</span>)&nbsp;

<div class="modal fade" id="bib-cheng_NI2014" role="dialog"><div class="modal-dialog"><div class="modal-content"><div class="modal-header"><button type="button" class="close" data-dismiss="modal">&times;</button><h4 class="modal-title">Bibtex</h4></div><div class="modal-body">
<pre>@article{cheng_NI2014,
 author = {Jian Cheng and Rachid Deriche and Tianzi Jiang and Dinggang Shen and Pew-Thian Yap},
 doi = {10.1016/j.neuroimage.2014.07.062},
 journal = {NeuroImage},
 pages = {750-764},
 pdf = {https://pdfs.semanticscholar.org/fc76/327ac26d573283db98939119660588c2af6f.pdf},
 publisher = {Elsevier},
 title = {Non-Negative Spherical Deconvolution (NNSD) for estimation of fiber Orientation Distribution Function in single-/multi-shell diffusion MRI},
 url = {https://dx.doi.org/10.1016/j.neuroimage.2014.07.062},
 volume = {101},
 year = {2014}
}</pre> </div><div class="modal-footer"><button type="button" class="btn btn-default" data-dismiss="modal">Close</button></div></div></div></div>
<div class="modal fade" id="abs-cheng_NI2014" role="dialog"><div class="modal-dialog"><div class="modal-content"><div class="modal-header"><button type="button" class="close" data-dismiss="modal">&times;</button><h4 class="modal-title">Abstract</h4></div><div class="modal-body">
<pre>Spherical Deconvolution (SD) is commonly used for estimating fiber
Orientation Distribution Functions (fODFs) from diffusion-weighted
signals. Existing SD methods can be classified into two categories:
1) Continuous Representation based SD (CR-SD), where typically
Spherical Harmonic (SH) representation is used for convenient
analytical solutions, and 2) Discrete Representation based SD (DR-
SD), where the signal profile is represented by a discrete set of
basis functions uniformly oriented on the unit sphere. A feasible
fODF should be non-negative and should integrate to unity throughout
the unit sphere S 2. However, to our knowledge, most existing SH-
based SD methods enforce non-negativity only on discretized points
and not the whole continuum of S 2. Maximum Entropy SD (MESD) and
Cartesian Tensor Fiber Orientation Distributions (CT-FOD) are the
only SD methods that ensure non-negativity throughout the unit
sphere. They are however computational intensive and are susceptible
to errors caused by numerical spherical integration. Existing SD
methods are also known to overestimate the number of fiber
directions, especially in regions with low anisotropy. DR-SD
introduces additional error in peak detection owing to the angular
discretization of the unit sphere. This paper proposes a SD
framework, called Non-Negative SD (NNSD), to overcome all the
limitations above. NNSD is significantly less susceptible to the
false-positive peaks, uses SH representation for efficient
analytical spherical deconvolution, and allows accurate peak
detection throughout the whole unit sphere. We further show that
NNSD and most existing SD methods can be extended to work on multi-
shell data by introducing a three-dimensional fiber response
function. We evaluated NNSD in comparison with Constrained SD (CSD),
a quadratic programming variant of CSD, MESD, and an L1-norm
regularized non-negative least-squares DR-SD. Experiments on
synthetic and real single-/multi-shell data indicate that NNSD
improves estimation performance in terms of mean difference of
angles, peak detection consistency, and anisotropy contrast between
isotropic and anisotropic regions.</pre> </div><div class="modal-footer"><button type="button" class="btn btn-default" data-dismiss="modal">Close</button></div></div></div></div>
</li><br>

<li>
<span class="author">Rui Min, Guorong Wu, <b>Jian Cheng</b>, Qian Wang, Dinggang Shen*</span>,<br>
<span class="title">"Multi-atlas based representations for Alzheimer's disease diagnosis"</span>,<br>
<span class="publisher">Human Brain Mapping (<b>HBM</b>)</span>, vol. 35, no. 10, pp. 5052&ndash;5070, <span class="year">2014</span>.
<br><a target="_self" href="https://www.researchgate.net/profile/Jian_Cheng2/publication/261772139_Multi-Atlas_Based_Representations_for_Alzheimer%27s_Disease_Diagnosis/links/584bc35808aed95c24fbf813/Multi-Atlas-Based-Representations-for-Alzheimers-Disease-Diagnosis.pdf"><img src="Icons/pdf.gif" alt="[pdf]" style="width: 16px; height: 16px;"></a>&nbsp;
<a target="_self" href="https://dx.doi.org/10.1002/hbm.22531"><img src="Icons/www.gif" alt="[www]" style="width: 16px; height: 16px;"></a>&nbsp;
[<a type="button" data-toggle="modal" data-target="#bib-min_HBM2014">bibtex</a>]&nbsp;
[<a type="button" data-toggle="modal" data-target="#abs-min_HBM2014">abstract</a>]&nbsp;
[citations: <a target="_blank" href="https://scholar.google.com//citations?view_op=view_citation&hl=en&oe=ASCII&user=BARqXQ0AAAAJ&pagesize=100&sortby=pubdate&citation_for_view=BARqXQ0AAAAJ:4fGpz3EwCPoC">93</a>]&nbsp;

<div class="modal fade" id="bib-min_HBM2014" role="dialog"><div class="modal-dialog"><div class="modal-content"><div class="modal-header"><button type="button" class="close" data-dismiss="modal">&times;</button><h4 class="modal-title">Bibtex</h4></div><div class="modal-body">
<pre>@article{min_HBM2014,
 author = {Rui Min and Guorong Wu and Jian Cheng and Qian Wang and Dinggang Shen},
 doi = {10.1002/hbm.22531},
 journal = {Human Brain Mapping},
 number = {10},
 pages = {5052-5070},
 pdf = {https://www.researchgate.net/profile/Jian_Cheng2/publication/261772139_Multi-Atlas_Based_Representations_for_Alzheimer%27s_Disease_Diagnosis/links/584bc35808aed95c24fbf813/Multi-Atlas-Based-Representations-for-Alzheimers-Disease-Diagnosis.pdf},
 publisher = {Wiley Online Library},
 title = {Multi-atlas based representations for Alzheimer's disease diagnosis},
 url = {https://dx.doi.org/10.1002/hbm.22531},
 volume = {35},
 year = {2014}
}</pre> </div><div class="modal-footer"><button type="button" class="btn btn-default" data-dismiss="modal">Close</button></div></div></div></div>
<div class="modal fade" id="abs-min_HBM2014" role="dialog"><div class="modal-dialog"><div class="modal-content"><div class="modal-header"><button type="button" class="close" data-dismiss="modal">&times;</button><h4 class="modal-title">Abstract</h4></div><div class="modal-body">
<pre>Brain morphometry based classification from magnetic resonance (MR)
acquisitions has been widely investigated in the diagnosis of
Alzheimer's disease (AD) and its prodromal stage, i.e., mild
cognitive impairment (MCI). In the literature, a morphometric
representation of brain structures is obtained by spatial
normalization of each image into a common space (i.e., a pre-defined
atlas) via non-linear registration, thus the corresponding regions
in different brains can be compared. However, representations
generated from one single atlas may not be sufficient to reveal the
underlying anatomical differences between the groups of disease-
affected patients and normal controls (NC). In this article, we
propose a different methodology, namely the multi-atlas based
morphometry, which measures morphometric representations of the same
image in different spaces of multiple atlases. Representations
generated from different atlases can thus provide the complementary
information to discriminate different groups, and also reduce the
negative impacts from registration errors. Specifically, each
studied subject is registered to multiple atlases, where adaptive
regional features are extracted. Then, all features from different
atlases are jointly selected by a correlation and relevance based
scheme, followed by final classification with the support vector
machine (SVM). We have evaluated the proposed method on 459 subjects
(97 AD, 117 progressive-MCI (p-MCI), 117 stable-MCI (s-MCI), and 128
NC) from the Alzheimer's Disease Neuroimaging Initiative (ADNI)
database, and achieved 91.64% for AD/NC classification and 72.41%
for p-MCI/s-MCI classification. Our results clearly demonstrate that
the proposed multi-atlas based method can significantly outperform
the previous single-atlas based methods.</pre> </div><div class="modal-footer"><button type="button" class="btn btn-default" data-dismiss="modal">Close</button></div></div></div></div>
</li><br>

<li>
<span class="author"><b>Jian Cheng</b>, Dinggang Shen, Pew-Thian Yap</span>,<br>
<span class="title">"Designing single- and multiple-shell sampling schemes for diffusion MRI using spherical code"</span>,<br>
<span class="publisher">Medical Image Computing and Computer-Assisted Intervention (<b>MICCAI</b>'14)</span>, vol. 8675, pp. 281&ndash;288, <span class="year">2014</span>.
<br><a target="_self" href="https://hal.archives-ouvertes.fr/hal-01011897/document"><img src="Icons/pdf.gif" alt="[pdf]" style="width: 16px; height: 16px;"></a>&nbsp;
<a target="_self" href="https://dx.doi.org/10.1007/978-3-319-10443-0_36"><img src="Icons/www.gif" alt="[www]" style="width: 16px; height: 16px;"></a>&nbsp;
[<a type="button" data-toggle="modal" data-target="#bib-cheng_MICCAI2014">bibtex</a>]&nbsp;
[<a type="button" data-toggle="modal" data-target="#abs-cheng_MICCAI2014">abstract</a>]&nbsp;
[<a target="_self" href="https://diffusionmritool.github.io/tutorial_qspacesampling.html">project</a>]&nbsp;
[citations: <a target="_blank" href="https://scholar.google.com//citations?view_op=view_citation&hl=en&oe=ASCII&user=BARqXQ0AAAAJ&pagesize=100&sortby=pubdate&citation_for_view=BARqXQ0AAAAJ:9pM33mqn1YgC">22</a>]&nbsp;
(<span class="hlnote">early accepted</span>)&nbsp;

<div class="modal fade" id="bib-cheng_MICCAI2014" role="dialog"><div class="modal-dialog"><div class="modal-content"><div class="modal-header"><button type="button" class="close" data-dismiss="modal">&times;</button><h4 class="modal-title">Bibtex</h4></div><div class="modal-body">
<pre>@inproceedings{cheng_MICCAI2014,
 author = {Jian Cheng and Dinggang Shen and Pew-Thian Yap},
 booktitle = {Medical Image Computing and Computer-Assisted Intervention (MICCAI'14)},
 doi = {10.1007/978-3-319-10443-0_36},
 hal_id = {hal-01011897},
 pages = {281-288},
 pdf = {https://hal.archives-ouvertes.fr/hal-01011897/document},
 publisher = {Springer},
 title = {Designing single- and multiple-shell sampling schemes for diffusion MRI using spherical code},
 url = {https://dx.doi.org/10.1007/978-3-319-10443-0_36},
 volume = {8675},
 year = {2014}
}</pre> </div><div class="modal-footer"><button type="button" class="btn btn-default" data-dismiss="modal">Close</button></div></div></div></div>
<div class="modal fade" id="abs-cheng_MICCAI2014" role="dialog"><div class="modal-dialog"><div class="modal-content"><div class="modal-header"><button type="button" class="close" data-dismiss="modal">&times;</button><h4 class="modal-title">Abstract</h4></div><div class="modal-body">
<pre>In diffusion MRI (dMRI), determining an appropriate sampling scheme
is crucial for acquiring the maximal amount of information for data
reconstruction and analysis using the minimal amount of time. For
single-shell acquisition, uniform sampling without directional
preference is usually favored. To achieve this, a commonly used
approach is the Electrostatic Energy Minimization (EEM) method
introduced in dMRI by Jones et al. However, the electrostatic energy
formulation in EEM is not directly related to the goal of optimal
sampling-scheme design, i.e., achieving large angular separation
between sampling points. A mathematically more natural approach is
to consider the Spherical Code (SC) formulation, which aims to
achieve uniform sampling by maximizing the minimal angular
difference between sampling points on the unit sphere. Although SC
is well studied in the mathematical literature, its current
formulation is limited to a single shell and is not applicable to
multiple shells. Moreover, SC, or more precisely continuous SC
(CSC), currently can only be applied on the continuous unit sphere
and hence cannot be used in situations where one or several subsets
of sampling points need to be determined from an existing sampling
scheme. In this case, discrete SC (DSC) is required. In this paper,
we propose novel DSC and CSC methods for designing uniform
single-/multi-shell sampling schemes. The DSC and CSC formulations
are solved respectively by Mixed Integer Linear Programming (MILP)
and a gradient descent approach. A fast greedy incremental solution
is also provided for both DSC and CSC. To our knowledge, this is the
first work to use SC formulation for designing sampling schemes in
dMRI. Experimental results indicate that our methods obtain larger
angular separation and better rotational invariance than the
generalized EEM (gEEM) method currently used in the Human Connectome
Project (HCP).</pre> </div><div class="modal-footer"><button type="button" class="btn btn-default" data-dismiss="modal">Close</button></div></div></div></div>
</li><br>

<li>
<span class="author">Rui Min, <b>Jian Cheng</b>, True Price, Guorong Wu, Dinggang Shen</span>,<br>
<span class="title">"Maximum-margin based representation learning from multiple atlases for Alzheimer's disease classification"</span>,<br>
<span class="publisher">International Conference on Medical Image Computing and Computer-Assisted Intervention (<b>MICCAI</b>'14)</span>, vol. 8674, pp. 212&ndash;219, <span class="year">2014</span>.
<br><a target="_self" href="https://www.researchgate.net/profile/Jian_Cheng2/publication/269284266_Maximum-Margin_Based_Representation_Learning_from_Multiple_Atlases_for_Alzheimer's_Disease_Classification/links/584bc76608aed95c24fbf847.pdf"><img src="Icons/pdf.gif" alt="[pdf]" style="width: 16px; height: 16px;"></a>&nbsp;
<a target="_self" href="https://dx.doi.org/10.1007/978-3-319-10470-6_27"><img src="Icons/www.gif" alt="[www]" style="width: 16px; height: 16px;"></a>&nbsp;
[<a type="button" data-toggle="modal" data-target="#bib-min-MICCAI2014">bibtex</a>]&nbsp;
[<a type="button" data-toggle="modal" data-target="#abs-min-MICCAI2014">abstract</a>]&nbsp;
[citations: <a target="_blank" href="https://scholar.google.com//citations?view_op=view_citation&hl=en&oe=ASCII&user=BARqXQ0AAAAJ&pagesize=100&sortby=pubdate&citation_for_view=BARqXQ0AAAAJ:8d8msizDQcsC">15</a>]&nbsp;

<div class="modal fade" id="bib-min-MICCAI2014" role="dialog"><div class="modal-dialog"><div class="modal-content"><div class="modal-header"><button type="button" class="close" data-dismiss="modal">&times;</button><h4 class="modal-title">Bibtex</h4></div><div class="modal-body">
<pre>@inproceedings{min:MICCAI2014,
 author = {Rui Min and Jian Cheng and True Price and Guorong Wu and Dinggang Shen},
 booktitle = {International Conference on Medical Image Computing and Computer-Assisted Intervention (MICCAI'14)},
 doi = {10.1007/978-3-319-10470-6_27},
 organization = {Springer},
 pages = {212-219},
 pdf = {https://www.researchgate.net/profile/Jian_Cheng2/publication/269284266_Maximum-Margin_Based_Representation_Learning_from_Multiple_Atlases_for_Alzheimer's_Disease_Classification/links/584bc76608aed95c24fbf847.pdf},
 title = {Maximum-margin based representation learning from multiple atlases for Alzheimer's disease classification},
 url = {https://dx.doi.org/10.1007/978-3-319-10470-6_27},
 volume = {8674},
 year = {2014}
}</pre> </div><div class="modal-footer"><button type="button" class="btn btn-default" data-dismiss="modal">Close</button></div></div></div></div>
<div class="modal fade" id="abs-min-MICCAI2014" role="dialog"><div class="modal-dialog"><div class="modal-content"><div class="modal-header"><button type="button" class="close" data-dismiss="modal">&times;</button><h4 class="modal-title">Abstract</h4></div><div class="modal-body">
<pre>In order to establish the correspondences between different brains
for comparison, spatial normalization based morphometric
measurements have been widely used in the analysis of Alzheimer's
disease (AD). In the literature, different subjects are often
compared in one atlas space, which may be insufficient in revealing
complex brain changes. In this paper, instead of deploying one atlas
for feature extraction and classification, we propose a maximum-
margin based representation learning (MMRL) method to learn the
optimal representation from multiple atlases. Unlike traditional
methods that perform the representation learning separately from the
classification, we propose to learn the new representation jointly
with the classification model, which is more powerful in
discriminating AD patients from normal controls (NC). We evaluated
the proposed method on the ADNI database, and achieved 90.69% for
AD/NC classification and 73.69% for p-MCI/s-MCI classification.</pre> </div><div class="modal-footer"><button type="button" class="btn btn-default" data-dismiss="modal">Close</button></div></div></div></div>
</li><br>

<li>
<span class="author">Feng Shi#, <b>Jian Cheng</b>#, Li Wang, Pew-Thian Yap, Dinggang Shen</span>,<br>
<span class="title">"Longitudinal guided super-resolution reconstruction of neonatal brain MR images"</span>,<br>
<span class="publisher">MICCAI Workshop on Spatio-temporal Image Analysis for Longitudinal and Time-Series Image Data</span>, vol. 8682, pp. 67&ndash;76, <span class="year">2014</span>.
<br><a target="_self" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4669195/pdf/nihms691856.pdf"><img src="Icons/pdf.gif" alt="[pdf]" style="width: 16px; height: 16px;"></a>&nbsp;
<a target="_self" href="https://dx.doi.org/10.1007/978-3-319-14905-9_6"><img src="Icons/www.gif" alt="[www]" style="width: 16px; height: 16px;"></a>&nbsp;
[<a type="button" data-toggle="modal" data-target="#bib-shi-2014">bibtex</a>]&nbsp;
[<a type="button" data-toggle="modal" data-target="#abs-shi-2014">abstract</a>]&nbsp;
[citations: <a target="_blank" href="https://scholar.google.com//citations?view_op=view_citation&hl=en&oe=ASCII&user=BARqXQ0AAAAJ&pagesize=100&sortby=pubdate&citation_for_view=BARqXQ0AAAAJ:LO7wyVUgiFcC">7</a>]&nbsp;
(<span class="hlnote0">F. Shi and J. Cheng contributed equally.</span>)&nbsp;

<div class="modal fade" id="bib-shi-2014" role="dialog"><div class="modal-dialog"><div class="modal-content"><div class="modal-header"><button type="button" class="close" data-dismiss="modal">&times;</button><h4 class="modal-title">Bibtex</h4></div><div class="modal-body">
<pre>@inproceedings{shi:2014,
 author = {Feng Shi and Jian Cheng and Li Wang and Pew-Thian Yap and Dinggang Shen},
 booktitle = {MICCAI Workshop on Spatio-temporal Image Analysis for Longitudinal and Time-Series Image Data},
 doi = {10.1007/978-3-319-14905-9_6},
 organization = {Springer},
 pages = {67-76},
 pdf = {https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4669195/pdf/nihms691856.pdf},
 title = {Longitudinal guided super-resolution reconstruction of neonatal brain MR images},
 url = {https://dx.doi.org/10.1007/978-3-319-14905-9_6},
 volume = {8682},
 year = {2014}
}</pre> </div><div class="modal-footer"><button type="button" class="btn btn-default" data-dismiss="modal">Close</button></div></div></div></div>
<div class="modal fade" id="abs-shi-2014" role="dialog"><div class="modal-dialog"><div class="modal-content"><div class="modal-header"><button type="button" class="close" data-dismiss="modal">&times;</button><h4 class="modal-title">Abstract</h4></div><div class="modal-body">
<pre>Neonatal images have low spatial resolution and insufficient tissue
contrast. Generally, interpolation methods are used to upsample
neonatal images to a higher resolution for more effective image
analysis. However, the resulting images are often blurry and are
susceptible to partial volume effect. In this paper, we propose an
algorithm that utilizes longitudinal prior information for effective
super-resolution reconstruction of neonatal images. We use a non-
local approach to learn the spatial relationships of brain
structures in high-resolution longitudinal images and apply this
information to the super-resolution reconstruction of the neonatal
image. In other words, the recurring patterns throughout the
longitudinal scans are leveraged for reconstructing the neonatal
image with high resolution. To solve this otherwise ill-posed
inverse problem, low-rank and total-variation regularizations are
enforced. Experiments performed on both T1- and T2-weighted MR
images of 28 neonates demonstrate that the proposed method is
capable of recovering more structural details and outperforms
methods such as nearest neighbor interpolation, spline-based
interpolation, non-local means upsampling, and both low-rank and
total variation based super-resolution</pre> </div><div class="modal-footer"><button type="button" class="btn btn-default" data-dismiss="modal">Close</button></div></div></div></div>
</li><br>

<li>
<span class="author"><b>Jian Cheng</b>, Dinggang Shen, Pew-Thian Yap</span>,<br>
<span class="title">"Joint k-q Space Compressed Sensing for Accelerated Multi-Shell Acquisition and Reconstruction of the diffusion signal and Ensemble Average Propagator"</span>,<br>
<span class="publisher">ISMRM'14</span>, <span class="year">2014</span>.
<br><a target="_self" href="https://hal.archives-ouvertes.fr/hal-01011893/document"><img src="Icons/pdf.gif" alt="[pdf]" style="width: 16px; height: 16px;"></a>&nbsp;
<a target="_self" href="https://hal.archives-ouvertes.fr/hal-01011893"><img src="Icons/www.gif" alt="[www]" style="width: 16px; height: 16px;"></a>&nbsp;
[<a type="button" data-toggle="modal" data-target="#bib-cheng_ISMRM2014">bibtex</a>]&nbsp;
(<span class="hlnote">Oral presentation</span>)&nbsp;

<div class="modal fade" id="bib-cheng_ISMRM2014" role="dialog"><div class="modal-dialog"><div class="modal-content"><div class="modal-header"><button type="button" class="close" data-dismiss="modal">&times;</button><h4 class="modal-title">Bibtex</h4></div><div class="modal-body">
<pre>@conference{cheng_ISMRM2014,
 author = {Jian Cheng and Dinggang Shen and Pew-Thian Yap},
 booktitle = {ISMRM'14},
 hal_id = {hal-01011893},
 pdf = {https://hal.archives-ouvertes.fr/hal-01011893/document},
 title = {Joint k-q Space Compressed Sensing for Accelerated Multi-Shell Acquisition and Reconstruction of the diffusion signal and Ensemble Average Propagator},
 url = {https://hal.archives-ouvertes.fr/hal-01011893},
 year = {2014}
}</pre> </div><div class="modal-footer"><button type="button" class="btn btn-default" data-dismiss="modal">Close</button></div></div></div></div>
</li><br>

<li>
<span class="author"><b>Jian Cheng</b>, Pew-Thian Yap, Dinggang Shen</span>,<br>
<span class="title">"Single and Multiple Shell Sampling Design in dMRI Using Spherical code and Mixed Integer Linear Programming"</span>,<br>
<span class="publisher">ISMRM'14</span>, pp. 2558, May, <span class="year">2014</span>.
<br><a target="_self" href="https://hal.archives-ouvertes.fr/hal-01011892/document"><img src="Icons/pdf.gif" alt="[pdf]" style="width: 16px; height: 16px;"></a>&nbsp;
<a target="_self" href="https://hal.archives-ouvertes.fr/hal-01011892"><img src="Icons/www.gif" alt="[www]" style="width: 16px; height: 16px;"></a>&nbsp;
[<a type="button" data-toggle="modal" data-target="#bib-cheng-ISMRM2014-sampling">bibtex</a>]&nbsp;

<div class="modal fade" id="bib-cheng-ISMRM2014-sampling" role="dialog"><div class="modal-dialog"><div class="modal-content"><div class="modal-header"><button type="button" class="close" data-dismiss="modal">&times;</button><h4 class="modal-title">Bibtex</h4></div><div class="modal-body">
<pre>@conference{cheng:ISMRM2014:sampling,
 address = {Italy},
 author = {Jian Cheng and Pew-Thian Yap and Dinggang Shen},
 booktitle = {ISMRM'14},
 hal_id = {hal-01011892},
 month = {May},
 pages = {2558},
 pdf = {https://hal.archives-ouvertes.fr/hal-01011892/document},
 title = {Single and Multiple Shell Sampling Design in dMRI Using Spherical code and Mixed Integer Linear Programming},
 url = {https://hal.archives-ouvertes.fr/hal-01011892},
 year = {2014}
}</pre> </div><div class="modal-footer"><button type="button" class="btn btn-default" data-dismiss="modal">Close</button></div></div></div></div>
</li><br>

</ol><h2><a name="year2013"></a>2013</h2>

<ol reversed>

<li>
<span class="author"><b>Jian Cheng</b>, Tianzi Jiang, Rachid Deriche, Dinggang Shen, Pew-Thian Yap</span>,<br>
<span class="title">"Regularized Spherical Polar Fourier Diffusion MRI with Optimal Dictionary Learning"</span>,<br>
<span class="publisher">The 16th International Conference on Medical Image Computing and Computer Assisted Intervention (<b>MICCAI</b>'13)</span>, vol. 8149, pp. 639&ndash;646, <span class="year">2013</span>.
<br><a target="_self" href="https://hal.inria.fr/hal-00824507/document"><img src="Icons/pdf.gif" alt="[pdf]" style="width: 16px; height: 16px;"></a>&nbsp;
<a target="_self" href="https://dx.doi.org/10.1007/978-3-642-40811-3_80"><img src="Icons/www.gif" alt="[www]" style="width: 16px; height: 16px;"></a>&nbsp;
[<a type="button" data-toggle="modal" data-target="#bib-cheng_DL-SPFI_MICCAI13">bibtex</a>]&nbsp;
[<a type="button" data-toggle="modal" data-target="#abs-cheng_DL-SPFI_MICCAI13">abstract</a>]&nbsp;
[<a target="_self" href="https://diffusionmritool.github.io/tutorial_spfi.html">project</a>]&nbsp;
[citations: <a target="_blank" href="https://scholar.google.com//citations?view_op=view_citation&hl=en&oe=ASCII&user=BARqXQ0AAAAJ&pagesize=100&sortby=pubdate&citation_for_view=BARqXQ0AAAAJ:rmuvC79q63oC">16</a>]&nbsp;

<div class="modal fade" id="bib-cheng_DL-SPFI_MICCAI13" role="dialog"><div class="modal-dialog"><div class="modal-content"><div class="modal-header"><button type="button" class="close" data-dismiss="modal">&times;</button><h4 class="modal-title">Bibtex</h4></div><div class="modal-body">
<pre>@inproceedings{cheng_DL-SPFI_MICCAI13,
 author = {Jian Cheng and Tianzi Jiang and Rachid Deriche and Dinggang Shen and Pew-Thian Yap},
 booktitle = {The 16th International Conference on Medical Image Computing and Computer Assisted Intervention (MICCAI'13)},
 doi = {10.1007/978-3-642-40811-3_80},
 hal_id = {hal-00824507},
 pages = {639-646},
 pdf = {https://hal.inria.fr/hal-00824507/document},
 title = {Regularized Spherical Polar Fourier Diffusion MRI with Optimal Dictionary Learning},
 url = {https://dx.doi.org/10.1007/978-3-642-40811-3_80},
 volume = {8149},
 year = {2013}
}</pre> </div><div class="modal-footer"><button type="button" class="btn btn-default" data-dismiss="modal">Close</button></div></div></div></div>
<div class="modal fade" id="abs-cheng_DL-SPFI_MICCAI13" role="dialog"><div class="modal-dialog"><div class="modal-content"><div class="modal-header"><button type="button" class="close" data-dismiss="modal">&times;</button><h4 class="modal-title">Abstract</h4></div><div class="modal-body">
<pre>Compressed Sensing (CS) takes advantage of signal sparsity or
compressibility and allows superb signal reconstruction from
relatively few measurements. Based on CS theory, a suitable
dictionary for sparse representation of the signal is required. In
diffusion MRI (dMRI), CS methods proposed for reconstruction of
diffusion-weighted signal and the Ensemble Average Propagator (EAP)
utilize two kinds of Dictionary Learning (DL) methods: 1) Discrete
Representation DL (DR-DL), and 2) Continuous Representation DL (CR-
DL). DR-DL is susceptible to numerical inaccuracy owing to
interpolation and regridding errors in a discretized q-space. In
this paper, we propose a novel CR-DL approach, called Dictionary
Learning - Spherical Polar Fourier Imaging (DL-SPFI) for effective
compressed-sensing reconstruction of the q-space diffusion-weighted
signal and the EAP. In DL-SPFI, a dictionary that sparsifies the
signal is learned from the space of continuous Gaussian diffusion
signals. The learned dictionary is then adaptively applied to
different voxels using a weighted LASSO framework for robust signal
reconstruction. Compared with the start-of-the-art CR-DL and DR-DL
methods proposed by Merlet et al. and Bilgic et al., respectively,
our work offers the following advantages. First, the learned
dictionary is proved to be optimal for Gaussian diffusion signals.
Second, to our knowledge, this is the first work to learn a voxel-
adaptive dictionary. The importance of the adaptive dictionary in
EAP reconstruction will be demonstrated theoretically and
empirically. Third, optimization in DL-SPFI is only performed in a
small subspace resided by the SPF coefficients, as opposed to the
q-space approach utilized by Merlet et al. We experimentally
evaluated DL-SPFI with respect to L1-norm regularized SPFI
(L1-SPFI), which uses the original SPF basis, and the DR-DL method
proposed by Bilgic et al. The experiment results on synthetic and
real data indicate that the learned dictionary produces sparser
coefficients than the original SPF basis and results in
significantly lower reconstruction error than Bilgic et al.'s
method.</pre> </div><div class="modal-footer"><button type="button" class="btn btn-default" data-dismiss="modal">Close</button></div></div></div></div>
</li><br>

<li>
<span class="author">Feng Shi#, <b>Jian Cheng</b>#, Li Wang, Pew-Thian Yap, Dinggang Shen</span>,<br>
<span class="title">"Low-Rank Total Variation for Image Super-Resolution"</span>,<br>
<span class="publisher">Medical Image Computing and Computer-Assisted Intervention (<b>MICCAI</b>'13)</span>, vol. 8149, pp. 155&ndash;162, <span class="year">2013</span>.
<br><a target="_self" href="https://www.researchgate.net/profile/Jian_Cheng2/publication/260127836_Low-Rank_Total_Variation_for_Image_Super-Resolution/links/0a85e52f69f43abff6000000.pdf"><img src="Icons/pdf.gif" alt="[pdf]" style="width: 16px; height: 16px;"></a>&nbsp;
<a target="_self" href="https://dx.doi.org/10.1007/978-3-642-40811-3_20"><img src="Icons/www.gif" alt="[www]" style="width: 16px; height: 16px;"></a>&nbsp;
[<a type="button" data-toggle="modal" data-target="#bib-shi_MICCAI13">bibtex</a>]&nbsp;
[<a type="button" data-toggle="modal" data-target="#abs-shi_MICCAI13">abstract</a>]&nbsp;
[<a target="_self" href="https://bitbucket.org/fengshi421/superresolutiontoolkit">code</a>]&nbsp;
[citations: <a target="_blank" href="https://scholar.google.com//citations?view_op=view_citation&hl=en&oe=ASCII&user=BARqXQ0AAAAJ&pagesize=100&sortby=pubdate&citation_for_view=BARqXQ0AAAAJ:YohjEiUPhakC">56</a>]&nbsp;
(<span class="hlnote0">F. Shi and J. Cheng contributed equally.</span>)&nbsp;

<div class="modal fade" id="bib-shi_MICCAI13" role="dialog"><div class="modal-dialog"><div class="modal-content"><div class="modal-header"><button type="button" class="close" data-dismiss="modal">&times;</button><h4 class="modal-title">Bibtex</h4></div><div class="modal-body">
<pre>@inproceedings{shi_MICCAI13,
 author = {Feng Shi and Jian Cheng and Li Wang and Pew-Thian Yap and Dinggang Shen},
 booktitle = {Medical Image Computing and Computer-Assisted Intervention (MICCAI'13)},
 doi = {10.1007/978-3-642-40811-3_20},
 pages = {155-162},
 pdf = {https://www.researchgate.net/profile/Jian_Cheng2/publication/260127836_Low-Rank_Total_Variation_for_Image_Super-Resolution/links/0a85e52f69f43abff6000000.pdf},
 publisher = {Springer},
 title = {Low-Rank Total Variation for Image Super-Resolution},
 url = {https://dx.doi.org/10.1007/978-3-642-40811-3_20},
 volume = {8149},
 year = {2013}
}</pre> </div><div class="modal-footer"><button type="button" class="btn btn-default" data-dismiss="modal">Close</button></div></div></div></div>
<div class="modal fade" id="abs-shi_MICCAI13" role="dialog"><div class="modal-dialog"><div class="modal-content"><div class="modal-header"><button type="button" class="close" data-dismiss="modal">&times;</button><h4 class="modal-title">Abstract</h4></div><div class="modal-body">
<pre>Most natural images can be approximated using their low-rank
components. This fact has been successfully exploited in recent
advancements of matrix completion algorithms for image recovery.
However, a major limitation of low-rank matrix completion algorithms
is that they cannot recover the case where a whole row or column is
missing. The missing row or column will be simply filled as an
arbitrary combination of other rows or columns with known values.
This precludes the application of matrix completion to problems such
as super-resolution (SR) where missing values in many rows and
columns need to be recovered in the process of up-sampling a low-
resolution image. Moreover, low-rank regularization considers
information globally from the whole image and does not take proper
consideration of local spatial consistency. Accordingly, we propose
in this paper a solution to the SR problem via simultaneous (global)
low-rank and (local) total variation (TV) regularization. We solve
the respective cost function using the alternating direction method
of multipliers (ADMM). Experiments on MR images of adults and
pediatric subjects demonstrate that the proposed method enhances the
details of the recovered high-resolution images, and outperforms the
nearest-neighbor interpolation, cubic interpolation, non-local
means, and TV-based up-sampling.</pre> </div><div class="modal-footer"><button type="button" class="btn btn-default" data-dismiss="modal">Close</button></div></div></div></div>
</li><br>

<li>
<span class="author"><b>Jian Cheng</b>, Rachid Deriche, Tianzi Jiang, Dinggang Shen, Pew-Thian Yap</span>,<br>
<span class="title">"Non-Negative Spherical Deconvolution (NNSD) for Fiber Orientation Distribution Function Estimation"</span>,<br>
<span class="publisher">MICCAI Workshop on Computational Diffusion MRI (CDMRI'13)</span>, pp. 81&ndash;93, <span class="year">2013</span>.
<br><a target="_self" href="https://hal.archives-ouvertes.fr/hal-00967829/document"><img src="Icons/pdf.gif" alt="[pdf]" style="width: 16px; height: 16px;"></a>&nbsp;
<a target="_self" href="https://dx.doi.org/10.1007/978-3-319-02475-2_8"><img src="Icons/www.gif" alt="[www]" style="width: 16px; height: 16px;"></a>&nbsp;
[<a type="button" data-toggle="modal" data-target="#bib-cheng_NNSD_CDMRI2013">bibtex</a>]&nbsp;
[<a type="button" data-toggle="modal" data-target="#abs-cheng_NNSD_CDMRI2013">abstract</a>]&nbsp;
[citations: <a target="_blank" href="https://scholar.google.com//citations?view_op=view_citation&hl=en&oe=ASCII&user=BARqXQ0AAAAJ&pagesize=100&sortby=pubdate&citation_for_view=BARqXQ0AAAAJ:86PQX7AUzd4C">4</a>]&nbsp;
(<span class="hlnote">Oral presentation</span>)&nbsp;

<div class="modal fade" id="bib-cheng_NNSD_CDMRI2013" role="dialog"><div class="modal-dialog"><div class="modal-content"><div class="modal-header"><button type="button" class="close" data-dismiss="modal">&times;</button><h4 class="modal-title">Bibtex</h4></div><div class="modal-body">
<pre>@inproceedings{cheng_NNSD_CDMRI2013,
 author = {Jian Cheng and Rachid Deriche and Tianzi Jiang and Dinggang Shen and Pew-Thian Yap},
 booktitle = {MICCAI Workshop on Computational Diffusion MRI (CDMRI'13)},
 doi = {10.1007/978-3-319-02475-2_8},
 hal_id = {hal-00967829},
 pages = {81-93},
 pdf = {https://hal.archives-ouvertes.fr/hal-00967829/document},
 title = {Non-Negative Spherical Deconvolution (NNSD) for Fiber Orientation Distribution Function Estimation},
 url = {https://dx.doi.org/10.1007/978-3-319-02475-2_8},
 year = {2013}
}</pre> </div><div class="modal-footer"><button type="button" class="btn btn-default" data-dismiss="modal">Close</button></div></div></div></div>
<div class="modal fade" id="abs-cheng_NNSD_CDMRI2013" role="dialog"><div class="modal-dialog"><div class="modal-content"><div class="modal-header"><button type="button" class="close" data-dismiss="modal">&times;</button><h4 class="modal-title">Abstract</h4></div><div class="modal-body">
<pre>In diffusion Magnetic Resonance Imaging (dMRI), Spherical
Deconvolution (SD) is a commonly used approach for estimating the
fiber Orientation Distribution Function (fODF). As a Probability
Density Function (PDF) that characterizes the distribution of fiber
orientations, the fODF is expected to be non-negative and to
integrate to unity on the continuous unit sphere $\mathbb{S}^2$.
However, many existing approaches, despite using continuous
representation such as Spherical Harmonics (SH), impose non-
negativity only on discretized points of $\mathbb{S}^2$. Therefore,
non-negativity is not guaranteed on the whole $\mathbb{S}^2$.
Existing approaches are also known to exhibit false positive fODF
peaks, especially in regions with low anisotropy, causing an over-
estimation of the number of fascicles that traverse each voxel. This
paper proposes a novel approach, called Non-Negative SD (NNSD), to
overcome the above limitations. NNSD offers the following
advantages. First, NNSD is the first SH based method that guarantees
non-negativity of the fODF throughout the unit sphere. Second,
unlike approaches such as Maximum Entropy SD (MESD), Cartesian
Tensor Fiber Orientation Distribution (CT-FOD), and discrete
representation based SD (DR-SD) techniques, the SH representation
allows closed form of spherical integration, efficient computation
in a low dimensional space resided by the SH coefficients, and
accurate peak detection on the continuous domain defined by the unit
sphere. Third, NNSD is significantly less susceptible to producing
false positive peaks in regions with low anisotropy. Evaluations of
NNSD in comparison with Constrained SD (CSD), MESD, and DR-SD
(implemented using L1-regularized least-squares with non-negative
constraint), indicate that NNSD yields improved performance for both
synthetic and real data. The performance gain is especially
prominent for high resolution (1.25 mm)^3 data.</pre> </div><div class="modal-footer"><button type="button" class="btn btn-default" data-dismiss="modal">Close</button></div></div></div></div>
</li><br>

<li>
<span class="author"><b>Jian Cheng</b>, Rachid Deriche, Tianzi Jiang, Dinggang Shen, Pew-Thian Yap</span>,<br>
<span class="title">"Non-Local Non-Negative Spherical Deconvolution for Single and Multiple Shell Diffusion MRI"</span>,<br>
<span class="publisher">HARDI Reconstruction Challenge, International Symposium on Biomedical Imaging (ISBI'13)</span>, <span class="year">2013</span>.
<br><a target="_self" href="https://hal.archives-ouvertes.fr/hal-00967830/document"><img src="Icons/pdf.gif" alt="[pdf]" style="width: 16px; height: 16px;"></a>&nbsp;
<a target="_self" href="https://hal.archives-ouvertes.fr/hal-00967830"><img src="Icons/www.gif" alt="[www]" style="width: 16px; height: 16px;"></a>&nbsp;
[<a type="button" data-toggle="modal" data-target="#bib-JianCheng_NNSD_ISBI13">bibtex</a>]&nbsp;
(<span class="hlnote2">Honorable Mention; the best technique in terms of local comparison.</span>)&nbsp;

<div class="modal fade" id="bib-JianCheng_NNSD_ISBI13" role="dialog"><div class="modal-dialog"><div class="modal-content"><div class="modal-header"><button type="button" class="close" data-dismiss="modal">&times;</button><h4 class="modal-title">Bibtex</h4></div><div class="modal-body">
<pre>@conference{JianCheng_NNSD_ISBI13,
 author = {Jian Cheng and Rachid Deriche and Tianzi Jiang and Dinggang Shen and Pew-Thian Yap},
 booktitle = {HARDI Reconstruction Challenge, International Symposium on Biomedical Imaging (ISBI'13)},
 hal_id = {hal-00967830},
 pdf = {https://hal.archives-ouvertes.fr/hal-00967830/document},
 title = {Non-Local Non-Negative Spherical Deconvolution for Single and Multiple Shell Diffusion MRI},
 url = {https://hal.archives-ouvertes.fr/hal-00967830},
 year = {2013}
}</pre> </div><div class="modal-footer"><button type="button" class="btn btn-default" data-dismiss="modal">Close</button></div></div></div></div>
</li><br>

<li>
<span class="author"><b>Jian Cheng</b>, Dinggang Shen, Pew-Thian Yap</span>,<br>
<span class="title">"Non-Negative Spherical Deconvolution for Fiber Orientation Distribution Estimation"</span>,<br>
<span class="publisher">ISMRM'13</span>, Apr, <span class="year">2013</span>.
<br><a target="_self" href="https://hal.archives-ouvertes.fr/hal-00967831/document"><img src="Icons/pdf.gif" alt="[pdf]" style="width: 16px; height: 16px;"></a>&nbsp;
<a target="_self" href="https://hal.archives-ouvertes.fr/hal-00967831"><img src="Icons/www.gif" alt="[www]" style="width: 16px; height: 16px;"></a>&nbsp;
[<a type="button" data-toggle="modal" data-target="#bib-cheng-ISMRM2013">bibtex</a>]&nbsp;

<div class="modal fade" id="bib-cheng-ISMRM2013" role="dialog"><div class="modal-dialog"><div class="modal-content"><div class="modal-header"><button type="button" class="close" data-dismiss="modal">&times;</button><h4 class="modal-title">Bibtex</h4></div><div class="modal-body">
<pre>@conference{cheng:ISMRM2013,
 address = {United States},
 author = {Jian Cheng and Dinggang Shen and Pew-Thian Yap},
 booktitle = {ISMRM'13},
 hal_id = {hal-00967831},
 month = {Apr},
 pdf = {https://hal.archives-ouvertes.fr/hal-00967831/document},
 title = {Non-Negative Spherical Deconvolution for Fiber Orientation Distribution Estimation},
 url = {https://hal.archives-ouvertes.fr/hal-00967831},
 year = {2013}
}</pre> </div><div class="modal-footer"><button type="button" class="btn btn-default" data-dismiss="modal">Close</button></div></div></div></div>
</li><br>

</ol><h2><a name="year2012"></a>2012</h2>

<ol reversed>

<li>
<span class="author"><b>Jian Cheng</b></span>,<br>
<span class="title">"Estimation and Processing of Ensemble Average Propagator and Its Features in Diffusion MRI"</span>,<br>
PhD thesis, Universit&eacute; Nice Sophia Antipolis, and Chinese Academy of Sciences, May, <span class="year">2012</span>.
<br><a target="_self" href="https://tel.archives-ouvertes.fr/tel-00759048/document"><img src="Icons/pdf.gif" alt="[pdf]" style="width: 16px; height: 16px;"></a>&nbsp;
<a target="_self" href="https://hal.archives-ouvertes.fr/tel-00759048"><img src="Icons/www.gif" alt="[www]" style="width: 16px; height: 16px;"></a>&nbsp;
[<a type="button" data-toggle="modal" data-target="#bib-thesis_jiancheng2012">bibtex</a>]&nbsp;
[<a type="button" data-toggle="modal" data-target="#abs-thesis_jiancheng2012">abstract</a>]&nbsp;
[<a target="_self" href="https://speakerdeck.com/jiancheng/estimation-and-processing-of-ensemble-average-propagator-and-its-features-in-diffusion-mri">slides</a>]&nbsp;
[citations: <a target="_blank" href="https://scholar.google.com//citations?view_op=view_citation&hl=en&oe=ASCII&user=BARqXQ0AAAAJ&pagesize=100&sortby=pubdate&citation_for_view=BARqXQ0AAAAJ:_axFR9aDTf0C">10</a>]&nbsp;

<div class="modal fade" id="bib-thesis_jiancheng2012" role="dialog"><div class="modal-dialog"><div class="modal-content"><div class="modal-header"><button type="button" class="close" data-dismiss="modal">&times;</button><h4 class="modal-title">Bibtex</h4></div><div class="modal-body">
<pre>@phdthesis{thesis_jiancheng2012,
 author = {Jian Cheng},
 hal_id = {tel-00759048},
 month = {May},
 pdf = {https://tel.archives-ouvertes.fr/tel-00759048/document},
 school = {Universit&eacute; Nice Sophia Antipolis, and Chinese Academy of Sciences},
 title = {Estimation and Processing of Ensemble Average Propagator and Its Features in Diffusion MRI},
 url = {https://hal.archives-ouvertes.fr/tel-00759048},
 year = {2012}
}</pre> </div><div class="modal-footer"><button type="button" class="btn btn-default" data-dismiss="modal">Close</button></div></div></div></div>
<div class="modal fade" id="abs-thesis_jiancheng2012" role="dialog"><div class="modal-dialog"><div class="modal-content"><div class="modal-header"><button type="button" class="close" data-dismiss="modal">&times;</button><h4 class="modal-title">Abstract</h4></div><div class="modal-body">
<pre>Diffusion MRI (dMRI) is the unique technique to infer the
microstructure of the white matter in vivo and noninvasively, by
modeling the diffusion of water molecules. Ensemble Average
Propagator (EAP) and Orientation Distribution Function (ODF) are two
important Probability Density Functions (PDFs) which reflect the
water diffusion. Estimation and processing of EAP and ODF is the
central problem in dMRI, and is also the first step for
tractography. Diffusion Tensor Imaging (DTI) is the most widely used
estimation method which assumes EAP as a Gaussian distribution
parameterized by a tensor. Riemannian framework for tensors has been
proposed successfully in tensor estimation and processing. However,
since the Gaussian EAP assumption is oversimplified, DTI can not
reflect complex microstructure like fiber crossing. High Angular
Resolution Diffusion Imaging (HARDI) is a category of methods
proposed to avoid the limitations of DTI. Most HARDI methods like
Q-Ball Imaging (QBI) need some assumptions and only can handle the
data from single shell (single $b$ value), which are called as
single shell HARDI (sHARDI) methods. However, with the development
of scanners and acquisition methods, multiple shell data becomes
more and more practical and popular. This thesis focuses on the
estimation and processing methods in multiple shell HARDI (mHARDI)
which can handle the diffusion data from arbitrary sampling scheme.
There are many original contributions in this thesis. -First, we
develop the analytical Spherical Polar Fourier Imaging (SPFI), which
represents the signal using SPF basis and obtains EAP and its
various features including ODFs and some scalar indices like
Generalized Fractional Anisotropy (GFA) from analytical linear
transforms. In the implementation of SPFI, we present two ways for
scale estimation and propose to consider the prior $E(0)=1$ in
estimation process. -Second, a novel Analytical Fourier Transform in
Spherical Coordinate (AFT-SC) framework is proposed to incorporate
many sHARDI and mHARDI methods, explore their relation and devise
new analytical EAP/ODF estimation methods. -Third, we present some
important criteria to compare different HARDI methods and illustrate
their advantages and limitations. -Fourth, we propose a novel
diffeomorphism invariant Riemannian framework for ODF and EAP
processing, which is a natural generalization of previous Riemannian
framework for tensors, and can be used for general PDF computing by
representing the square root of the PDF called wavefunction with
orthonormal basis. In this Riemannian framework, the exponential
map, logarithmic map and geodesic have closed forms, the weighted
Riemannian mean and median uniquely exist and can be estimated from
an efficient gradient descent. Log-Euclidean framework and Affine-
Euclidean framework are developed for fast data processing. -Fifth,
we theoretically and experimentally compare the Euclidean metric and
Riemannian metric for tensors, ODFs and EAPs. -Finally, we propose
the Geodesic Anisotropy (GA) to measure the anisotropy of EAPs,
Square Root Parameterized Estimation (SRPE) for nonnegative definite
ODF/EAP estimation, weighted Riemannian mean/median for ODF/EAP
interpolation, smoothing, atlas estimation. The concept of
\emph{reasonable mean value interpolation} is presented for
interpolation of general PDF data.</pre> </div><div class="modal-footer"><button type="button" class="btn btn-default" data-dismiss="modal">Close</button></div></div></div></div>
</li><br>

<li>
<span class="author">Nianming Zuo, <b>Jian Cheng</b>, Tianzi Jiang*</span>,<br>
<span class="title">"Diffusion Magnetic Resonance Imaging for Brainnetome: A Critical Review"</span>,<br>
<span class="publisher">Neuroscience Bulletin</span>, vol. 28, no. 4, pp. 375&ndash;388, <span class="year">2012</span>.
<br><a target="_self" href="http://www.nlpr.ia.ac.cn/2012papers/gjkw/gk84.pdf"><img src="Icons/pdf.gif" alt="[pdf]" style="width: 16px; height: 16px;"></a>&nbsp;
<a target="_self" href="https://dx.doi.org/10.1007/s12264-012-1245-3"><img src="Icons/www.gif" alt="[www]" style="width: 16px; height: 16px;"></a>&nbsp;
[<a type="button" data-toggle="modal" data-target="#bib-zuo-2012">bibtex</a>]&nbsp;
[<a type="button" data-toggle="modal" data-target="#abs-zuo-2012">abstract</a>]&nbsp;
[citations: <a target="_blank" href="https://scholar.google.com//citations?view_op=view_citation&hl=en&oe=ASCII&user=BARqXQ0AAAAJ&pagesize=100&sortby=pubdate&citation_for_view=BARqXQ0AAAAJ:_FxGoFyzp5QC">17</a>]&nbsp;

<div class="modal fade" id="bib-zuo-2012" role="dialog"><div class="modal-dialog"><div class="modal-content"><div class="modal-header"><button type="button" class="close" data-dismiss="modal">&times;</button><h4 class="modal-title">Bibtex</h4></div><div class="modal-body">
<pre>@article{zuo:2012,
 author = {Nianming Zuo and Jian Cheng and Tianzi Jiang},
 doi = {10.1007/s12264-012-1245-3},
 journal = {Neuroscience Bulletin},
 number = {4},
 pages = {375-388},
 pdf = {http://www.nlpr.ia.ac.cn/2012papers/gjkw/gk84.pdf},
 title = {Diffusion Magnetic Resonance Imaging for Brainnetome: A Critical Review},
 url = {https://dx.doi.org/10.1007/s12264-012-1245-3},
 volume = {28},
 year = {2012}
}</pre> </div><div class="modal-footer"><button type="button" class="btn btn-default" data-dismiss="modal">Close</button></div></div></div></div>
<div class="modal fade" id="abs-zuo-2012" role="dialog"><div class="modal-dialog"><div class="modal-content"><div class="modal-header"><button type="button" class="close" data-dismiss="modal">&times;</button><h4 class="modal-title">Abstract</h4></div><div class="modal-body">
<pre>Increasing evidence shows that the human brain is a highly self-
organized system that shows attributes of smallworldness, hierarchy
and modularity. The "connectome" was conceived several years ago to
identify the underpinning physical connectivities of brain networks.
The need for an integration of multi-spatial and -temporal
approaches is becoming apparent. Therefore, the "Brainnetome"
(brain-net-ome) project was proposed. Diffusion magnetic resonance
imaging (dMRI) is a non-invasive way to study the anatomy of brain
networks. Here, we review the principles of dMRI, its methodologies,
and some of its clinical applications for the Brainnetome. Future
research in this field is discussed.</pre> </div><div class="modal-footer"><button type="button" class="btn btn-default" data-dismiss="modal">Close</button></div></div></div></div>
</li><br>

<li>
<span class="author"><b>Jian Cheng</b>, Tianzi Jiang, Rachid Deriche</span>,<br>
<span class="title">"Nonnegative Definite EAP and ODF Estimation via a Unified Multi-Shell HARDI Reconstruction"</span>,<br>
<span class="publisher">Medical Image Computing and Computer-Assisted Intervention (<b>MICCAI</b>'12)</span>, vol. 6892, pp. 98&ndash;106, <span class="year">2012</span>.
<br><a target="_self" href="https://hal.archives-ouvertes.fr/hal-00759014/document"><img src="Icons/pdf.gif" alt="[pdf]" style="width: 16px; height: 16px;"></a>&nbsp;
<a target="_self" href="https://dx.doi.org/10.1007/978-3-642-33418-4_39"><img src="Icons/www.gif" alt="[www]" style="width: 16px; height: 16px;"></a>&nbsp;
[<a type="button" data-toggle="modal" data-target="#bib-cheng_MICCAI2012">bibtex</a>]&nbsp;
[<a type="button" data-toggle="modal" data-target="#abs-cheng_MICCAI2012">abstract</a>]&nbsp;
[citations: <a target="_blank" href="https://scholar.google.com//citations?view_op=view_citation&hl=en&oe=ASCII&user=BARqXQ0AAAAJ&pagesize=100&sortby=pubdate&citation_for_view=BARqXQ0AAAAJ:ufrVoPGSRksC">15</a>]&nbsp;
(<span class="hlnote2">MICCAI Student Travel Award</span>)&nbsp;

<div class="modal fade" id="bib-cheng_MICCAI2012" role="dialog"><div class="modal-dialog"><div class="modal-content"><div class="modal-header"><button type="button" class="close" data-dismiss="modal">&times;</button><h4 class="modal-title">Bibtex</h4></div><div class="modal-body">
<pre>@inproceedings{cheng_MICCAI2012,
 author = {Jian Cheng and Tianzi Jiang and Rachid Deriche},
 booktitle = {Medical Image Computing and Computer-Assisted Intervention (MICCAI'12)},
 doi = {10.1007/978-3-642-33418-4_39},
 pages = {98-106},
 pdf = {https://hal.archives-ouvertes.fr/hal-00759014/document},
 publisher = {Springer Berlin / Heidelberg},
 series = {LNCS},
 title = {Nonnegative Definite EAP and ODF Estimation via a Unified Multi-Shell HARDI Reconstruction},
 url = {https://dx.doi.org/10.1007/978-3-642-33418-4_39},
 volume = {6892},
 year = {2012}
}</pre> </div><div class="modal-footer"><button type="button" class="btn btn-default" data-dismiss="modal">Close</button></div></div></div></div>
<div class="modal fade" id="abs-cheng_MICCAI2012" role="dialog"><div class="modal-dialog"><div class="modal-content"><div class="modal-header"><button type="button" class="close" data-dismiss="modal">&times;</button><h4 class="modal-title">Abstract</h4></div><div class="modal-body">
<pre>In High Angular Resolution Diffusion Imaging (HARDI), Orientation
Distribution Function (ODF) and Ensemble Average Propagator (EAP)
are two important Probability Density Functions (PDFs) which reflect
the water diffusion and fiber orientations. Spherical Polar Fourier
Imaging (SPFI) is a recent model-free multi-shell HARDI method which
estimates both EAP and ODF from the diffusion signals with multiple
b values. As physical PDFs, ODFs and EAPs are nonnegative definite
respectively in their domains $\mathbb{S}^2$ and $\mathbb{R}^3$.
However, existing ODF/EAP estimation methods like SPFI seldom
consider this natural constraint. Although some works considered the
nonnegative constraint on the given discrete samples of ODF/EAP, the
estimated ODF/EAP is not guaranteed to be nonnegative definite in
the whole continuous domain. The Riemannian framework for ODFs and
EAPs has been proposed via the square root parameterization based on
pre-estimated ODFs and EAPs by other methods like SPFI. However,
there is no work on how to estimate the square root of ODF/EAP
called as the wavefuntion directly from diffusion signals. In this
paper, based on the Riemannian framework for ODFs/EAPs and Spherical
Polar Fourier (SPF) basis representation, we propose a unified
model-free multi-shell HARDI method, named as Square Root
Parameterized Estimation (SRPE), to simultaneously estimate both the
wavefunction of EAPs and the nonnegative definite ODFs and EAPs from
diffusion signals. The experiments on synthetic data and real data
showed SRPE is more robust to noise and has better EAP
reconstruction than SPFI, especially for EAP profiles at large
radius.</pre> </div><div class="modal-footer"><button type="button" class="btn btn-default" data-dismiss="modal">Close</button></div></div></div></div>
</li><br>

</ol><h2><a name="year2011"></a>2011</h2>

<ol reversed>

<li>
<span class="author"><b>Jian Cheng</b>, Aurobrata Ghosh, Tianzi Jiang, Rachid Deriche</span>,<br>
<span class="title">"Diffeomorphism Invariant Riemannian Framework for Ensemble Average Propagator Computing"</span>,<br>
<span class="publisher">Medical Image Computing and Computer-Assisted Intervention (<b>MICCAI</b>'11)</span>, vol. 6892, pp. 98&ndash;106, <span class="year">2011</span>.
<br><a target="_self" href="https://hal.inria.fr/inria-00615431/document"><img src="Icons/pdf.gif" alt="[pdf]" style="width: 16px; height: 16px;"></a>&nbsp;
<a target="_self" href="https://dx.doi.org/10.1007/978-3-642-23629-7_13"><img src="Icons/www.gif" alt="[www]" style="width: 16px; height: 16px;"></a>&nbsp;
[<a type="button" data-toggle="modal" data-target="#bib-cheng_MICCAI2011">bibtex</a>]&nbsp;
[<a type="button" data-toggle="modal" data-target="#abs-cheng_MICCAI2011">abstract</a>]&nbsp;
[citations: <a target="_blank" href="https://scholar.google.com//citations?view_op=view_citation&hl=en&oe=ASCII&user=BARqXQ0AAAAJ&pagesize=100&sortby=pubdate&citation_for_view=BARqXQ0AAAAJ:qjMakFHDy7sC">7</a>]&nbsp;
(<span class="hlnote2">MICCAI Student Travel Award</span>)&nbsp;

<div class="modal fade" id="bib-cheng_MICCAI2011" role="dialog"><div class="modal-dialog"><div class="modal-content"><div class="modal-header"><button type="button" class="close" data-dismiss="modal">&times;</button><h4 class="modal-title">Bibtex</h4></div><div class="modal-body">
<pre>@inproceedings{cheng_MICCAI2011,
 author = {Jian Cheng and Aurobrata Ghosh and Tianzi Jiang and Rachid Deriche},
 booktitle = {Medical Image Computing and Computer-Assisted Intervention (MICCAI'11)},
 doi = {10.1007/978-3-642-23629-7_13},
 hal_id = {inria-00615431},
 pages = {98-106},
 pdf = {https://hal.inria.fr/inria-00615431/document},
 publisher = {Springer Berlin / Heidelberg},
 series = {LNCS},
 title = {Diffeomorphism Invariant Riemannian Framework for Ensemble Average Propagator Computing},
 url = {https://dx.doi.org/10.1007/978-3-642-23629-7_13},
 volume = {6892},
 year = {2011}
}</pre> </div><div class="modal-footer"><button type="button" class="btn btn-default" data-dismiss="modal">Close</button></div></div></div></div>
<div class="modal fade" id="abs-cheng_MICCAI2011" role="dialog"><div class="modal-dialog"><div class="modal-content"><div class="modal-header"><button type="button" class="close" data-dismiss="modal">&times;</button><h4 class="modal-title">Abstract</h4></div><div class="modal-body">
<pre>Background: In Diffusion Tensor Imaging (DTI), Riemannian framework
based on Information Geometry theory has been proposed for
processing tensors on estimation, interpolation, smoothing,
regularization, segmentation, statistical test and so on. Recently
Riemannian framework has been generalized to Orientation
Distribution Function (ODF) and it is applicable to any Probability
Density Function (PDF) under orthonormal basis representation.
Spherical Polar Fourier Imaging (SPFI) was proposed for ODF and
Ensemble Average Propagator (EAP) estimation from arbitrary sampled
signals without any assumption. Purpose: Tensors only can represent
Gaussian EAP and ODF is the radial integration of EAP, while EAP has
full information for diffusion process. To our knowledge, so far
there is no work on how to process EAP data. In this paper, we
present a Riemannian framework as a mathematical tool for such task.
Method: We propose a state-of-the-art Riemannian framework for EAPs
by representing the square root of EAP, called wavefunction based on
quantum mechanics, with the Fourier dual Spherical Polar Fourier
(dSPF) basis. In this framework, the exponential map, logarithmic
map and geodesic have closed forms, and weighted Riemannian mean and
median uniquely exist. We analyze theoretically the similarities and
differences between Riemannian frameworks for EAPs and for ODFs and
tensors. The Riemannian metric for EAPs is diffeomorphism invariant,
which is the natural extension of the affine-invariant metric for
tensors. We propose Log-Euclidean framework to fast process EAPs,
and Geodesic Anisotropy (GA) to measure the anisotropy of EAPs. With
this framework, many important data processing operations, such as
interpolation, smoothing, atlas estimation, Principal Geodesic
Analysis (PGA), can be performed on EAP data. Results and
Conclusions: The proposed Riemannian framework was validated in
synthetic data for interpolation, smoothing, PGA and in real data
for GA and atlas estimation. Riemannian median is much robust for
atlas estimation.</pre> </div><div class="modal-footer"><button type="button" class="btn btn-default" data-dismiss="modal">Close</button></div></div></div></div>
</li><br>

<li>
<span class="author"><b>Jian Cheng</b>, Tianzi Jiang, Rachid Deriche</span>,<br>
<span class="title">"Theoretical Analysis and Practical Insights on EAP Estimation via a Unified HARDI Framework"</span>,<br>
<span class="publisher">MICCAI Workshop on Computational Diffusion MRI (CDMRI'11)</span>, <span class="year">2011</span>.
<br><a target="_self" href="https://hal.inria.fr/inria-00615430/document"><img src="Icons/pdf.gif" alt="[pdf]" style="width: 16px; height: 16px;"></a>&nbsp;
<a target="_self" href="https://hal.archives-ouvertes.fr/inria-00615430"><img src="Icons/www.gif" alt="[www]" style="width: 16px; height: 16px;"></a>&nbsp;
[<a type="button" data-toggle="modal" data-target="#bib-cheng_AFTSC_CDMRI2011">bibtex</a>]&nbsp;
[<a type="button" data-toggle="modal" data-target="#abs-cheng_AFTSC_CDMRI2011">abstract</a>]&nbsp;
[<a target="_self" href="https://diffusionmritool.github.io/tutorial_spfi.html">project</a>]&nbsp;
[citations: <a target="_blank" href="https://scholar.google.com//citations?view_op=view_citation&hl=en&oe=ASCII&user=BARqXQ0AAAAJ&pagesize=100&sortby=pubdate&citation_for_view=BARqXQ0AAAAJ:UeHWp8X0CEIC">24</a>]&nbsp;
(<span class="hlnote">Oral presentation</span>)&nbsp;

<div class="modal fade" id="bib-cheng_AFTSC_CDMRI2011" role="dialog"><div class="modal-dialog"><div class="modal-content"><div class="modal-header"><button type="button" class="close" data-dismiss="modal">&times;</button><h4 class="modal-title">Bibtex</h4></div><div class="modal-body">
<pre>@inproceedings{cheng_AFTSC_CDMRI2011,
 author = {Jian Cheng and Tianzi Jiang and Rachid Deriche},
 booktitle = {MICCAI Workshop on Computational Diffusion MRI (CDMRI'11)},
 hal_id = {inria-00615430},
 pdf = {https://hal.inria.fr/inria-00615430/document},
 title = {Theoretical Analysis and Practical Insights on EAP Estimation via a Unified HARDI Framework},
 url = {https://hal.archives-ouvertes.fr/inria-00615430},
 year = {2011}
}</pre> </div><div class="modal-footer"><button type="button" class="btn btn-default" data-dismiss="modal">Close</button></div></div></div></div>
<div class="modal fade" id="abs-cheng_AFTSC_CDMRI2011" role="dialog"><div class="modal-dialog"><div class="modal-content"><div class="modal-header"><button type="button" class="close" data-dismiss="modal">&times;</button><h4 class="modal-title">Abstract</h4></div><div class="modal-body">
<pre>Since Diffusion Tensor Imaging (DTI) cannot describe complex non-
Gaussian diffusion process, many techniques, called as single shell
High Angular Resolution Diffusion Imaging (sHARDI) methods,
reconstruct the Ensemble Average Propagator (EAP) or its feature
Orientation Distribution Function (ODF) from diffusion weighted
signals only in single shell. Q-Ball Imaging (QBI) and Diffusion
Orientation Transform (DOT) are two famous sHARDI methods. However,
these sHARDI methods have some intrinsic modeling errors or need
some unreal assumptions. Moreover they are hard to deal with signals
from different q-shells. Most recently several novel multiple shell
HARDI (mHARDI) methods, including Diffusion Propagator Imaging
(DPI), Spherical Polar Fourier Imaging (SPFI) and Simple Harmonic
Oscillator based Reconstruction and Estimation (SHORE), were
proposed to analytically estimate EAP or ODF from multiple shell (or
arbitrarily sampled) signals. These three methods all represent
diffusion signal with some basis functions in spherical coordinate
and use plane wave formula to analytically solve the Fourier
transform. To our knowledge, there is no theoretical analysis and
practical comparison among these sHARDI and mHARDI methods. In this
paper, we propose a unified computational framework, named
Analytical Fourier Transform in Spherical Coordinate (AFT-SC), to
perform such theoretical analysis and practical comparison among all
these five state-of-the-art diffusion MRI methods. We compare these
five methods in both theoretical and experimental aspects. With
respect to the theoretical aspect, some criteria are proposed for
evaluation and some differences together with some similarities
among the methods are highlighted. Regarding the experimental
aspect, all the methods are compared in synthetic, phantom and real
data. The shortcomings and advantages of each method are highlighted
from which SPFI appears to be among the best because it uses an
orthonormal basis that completely separates the spherical and radial
information.</pre> </div><div class="modal-footer"><button type="button" class="btn btn-default" data-dismiss="modal">Close</button></div></div></div></div>
</li><br>

<li>
<span class="author"><b>Jian Cheng</b>, Sylvain Merlet, Emmanuel Caruyer, Aurobrata Ghosh, Tianzi Jiang, Rachid Deriche</span>,<br>
<span class="title">"Compressive Sensing Ensemble Average Propagator Estimation via L1 Spherical Polar Fourier Imaging"</span>,<br>
<span class="publisher">MICCAI Workshop on Computational Diffusion MRI (CDMRI'11)</span>, pp. 108&ndash;118, <span class="year">2011</span>.
<br><a target="_self" href="https://hal.inria.fr/inria-00615437/document"><img src="Icons/pdf.gif" alt="[pdf]" style="width: 16px; height: 16px;"></a>&nbsp;
<a target="_self" href="https://hal.archives-ouvertes.fr/inria-00615437"><img src="Icons/www.gif" alt="[www]" style="width: 16px; height: 16px;"></a>&nbsp;
[<a type="button" data-toggle="modal" data-target="#bib-cheng_L1SPFI_CDMRI2011">bibtex</a>]&nbsp;
[<a type="button" data-toggle="modal" data-target="#abs-cheng_L1SPFI_CDMRI2011">abstract</a>]&nbsp;
[<a target="_self" href="https://diffusionmritool.github.io/tutorial_spfi.html">project</a>]&nbsp;
[citations: <a target="_blank" href="https://scholar.google.com//citations?view_op=view_citation&hl=en&oe=ASCII&user=BARqXQ0AAAAJ&pagesize=100&sortby=pubdate&citation_for_view=BARqXQ0AAAAJ:IjCSPb-OGe4C">22</a>]&nbsp;
(<span class="hlnote">Oral presentation</span>)&nbsp;

<div class="modal fade" id="bib-cheng_L1SPFI_CDMRI2011" role="dialog"><div class="modal-dialog"><div class="modal-content"><div class="modal-header"><button type="button" class="close" data-dismiss="modal">&times;</button><h4 class="modal-title">Bibtex</h4></div><div class="modal-body">
<pre>@inproceedings{cheng_L1SPFI_CDMRI2011,
 author = {Jian Cheng and Sylvain Merlet and Emmanuel Caruyer and Aurobrata Ghosh and Tianzi Jiang and Rachid Deriche},
 booktitle = {MICCAI Workshop on Computational Diffusion MRI (CDMRI'11)},
 hal_id = {inria-00615437},
 pages = {108-118},
 pdf = {https://hal.inria.fr/inria-00615437/document},
 title = {Compressive Sensing Ensemble Average Propagator Estimation via L1 Spherical Polar Fourier Imaging},
 url = {https://hal.archives-ouvertes.fr/inria-00615437},
 year = {2011}
}</pre> </div><div class="modal-footer"><button type="button" class="btn btn-default" data-dismiss="modal">Close</button></div></div></div></div>
<div class="modal fade" id="abs-cheng_L1SPFI_CDMRI2011" role="dialog"><div class="modal-dialog"><div class="modal-content"><div class="modal-header"><button type="button" class="close" data-dismiss="modal">&times;</button><h4 class="modal-title">Abstract</h4></div><div class="modal-body">
<pre>Since Diffusion Tensor Imaging (DTI) cannot detect the fiber
crossing, many new works beyond DTI has been proposed to explore the
q-space. Most works, known as single shell High Angular Resolution
Imaging (sHARDI), focus on single shell sampling and reconstruct the
Orientation Distribution Function (ODF). The ODF, which has no
radial information at all, is just one of features of Ensemble
Average Propagator (EAP). Diffusion Spectrum Imaging (DSI) is a
standard method to estimate EAP via numerical Fourier Transform
(FT), which needs lots of samples and is impractical for clinical
study. Spherical Polar Fourier Imaging (SPFI) [1,2] was proposed to
represent the signal using SPF basis, then the EAP and the ODF have
analytical closed forms. So the estimation of the coefficients under
SPF basis is very important. In [1,2], the coefficients are
estimated based on a standard Least Square (LS) with L2 norm
regularization (L2-L2). In this paper, we propose to estimate the
coefficients using LS with L1 norm regularization (L2-L1), also
named as Least Absolute Selection and Shrinkage Operator (LASSO).
And we prove that the L2-L1 estimation of the coefficients is
actually the well known Compressive Sensing (CS) method to estimate
EAP, which brings lots of Mathematical tools and possibility to
improve the sampling scheme in q-space.</pre> </div><div class="modal-footer"><button type="button" class="btn btn-default" data-dismiss="modal">Close</button></div></div></div></div>
</li><br>

<li>
<span class="author">Emmanuel Caruyer, <b>Jian Cheng</b>, Christophe Lenglet, Guillermo Sapiro, Tianzi Jiang, Rachid Deriche</span>,<br>
<span class="title">"Optimal Design of Multiple Q-shells experiments for Diffusion MRI"</span>,<br>
<span class="publisher">MICCAI Workshop on Computational Diffusion MRI (CDMRI'11)</span>, pp. 45&ndash;53, <span class="year">2011</span>.
<br><a target="_self" href="https://hal.inria.fr/inria-00617663/document"><img src="Icons/pdf.gif" alt="[pdf]" style="width: 16px; height: 16px;"></a>&nbsp;
<a target="_self" href="https://hal.archives-ouvertes.fr/inria-00617663"><img src="Icons/www.gif" alt="[www]" style="width: 16px; height: 16px;"></a>&nbsp;
[<a type="button" data-toggle="modal" data-target="#bib-caruyer_CDMRI11">bibtex</a>]&nbsp;
[<a type="button" data-toggle="modal" data-target="#abs-caruyer_CDMRI11">abstract</a>]&nbsp;
[citations: <a target="_blank" href="https://scholar.google.com//citations?view_op=view_citation&hl=en&oe=ASCII&user=BARqXQ0AAAAJ&pagesize=100&sortby=pubdate&citation_for_view=BARqXQ0AAAAJ:W7OEmFMy1HYC">49</a>]&nbsp;
(<span class="hlnote">Oral presentation</span>)&nbsp;
(<span class="hlnote2">Used in <a href=https://en.wikipedia.org/wiki/Human_Connectome_Project>Human Connectome Project (HCP)</a> for multi-shell diffusion MRI data acquisition.</span>)&nbsp;

<div class="modal fade" id="bib-caruyer_CDMRI11" role="dialog"><div class="modal-dialog"><div class="modal-content"><div class="modal-header"><button type="button" class="close" data-dismiss="modal">&times;</button><h4 class="modal-title">Bibtex</h4></div><div class="modal-body">
<pre>@inproceedings{caruyer_CDMRI11,
 author = {Emmanuel Caruyer and Jian Cheng and Christophe Lenglet and Guillermo Sapiro and Tianzi Jiang and Rachid Deriche},
 booktitle = {MICCAI Workshop on Computational Diffusion MRI (CDMRI'11)},
 hal_id = {inria-00617663},
 pages = {45-53},
 pdf = {https://hal.inria.fr/inria-00617663/document},
 title = {Optimal Design of Multiple Q-shells experiments for Diffusion MRI},
 url = {https://hal.archives-ouvertes.fr/inria-00617663},
 year = {2011}
}</pre> </div><div class="modal-footer"><button type="button" class="btn btn-default" data-dismiss="modal">Close</button></div></div></div></div>
<div class="modal fade" id="abs-caruyer_CDMRI11" role="dialog"><div class="modal-dialog"><div class="modal-content"><div class="modal-header"><button type="button" class="close" data-dismiss="modal">&times;</button><h4 class="modal-title">Abstract</h4></div><div class="modal-body">
<pre>Recent advances in diffusion MRI make use of the diffusion signal
sampled on the whole Q-space, rather than on a single sphere. While
much effort has been done to design uniform sampling schemes for
single shell experiment, it is yet not clear how to build a strategy
to sample the diffusion signal in the whole Fourier domain. In this
article, we propose a method to generate acquisition schemes for
multiple Q-shells experiment in diffusion MRI. The acquisition
protocols we design are incremental, which means they remain
approximately optimal when truncated before the acquisition is
complete. Our method is fast, incremental, and we can generate
diffusion gradients schemes for any number of acquisitions, any
number of shells, and any number of points per shell. The samples
arranged on different shells do not share the same directions. The
method is tested for Spherical Polar Fourier reconstruction of the
diffusion signal, and based on Monte-Carlo simulations, several
preferred acquisition parameters are identified.</pre> </div><div class="modal-footer"><button type="button" class="btn btn-default" data-dismiss="modal">Close</button></div></div></div></div>
</li><br>

<li>
<span class="author">Sylvain Merlet, <b>Jian Cheng</b>, Aurobrata Ghosh, Rachid Deriche</span>,<br>
<span class="title">"Spherical Polar Fourier EAP and ODF Reconstruction via Compressed Sensing in Diffusion MRI"</span>,<br>
<span class="publisher">IEEE International Symposium on Biomedical Imaging: From Nano to Macro (ISBI'11)</span>, pp. 365&ndash;371, <span class="year">2011</span>.
<br><a target="_self" href="https://hal.inria.fr/inria-00585694/document"><img src="Icons/pdf.gif" alt="[pdf]" style="width: 16px; height: 16px;"></a>&nbsp;
<a target="_self" href="https://dx.doi.org/10.1109/ISBI.2011.5872425"><img src="Icons/www.gif" alt="[www]" style="width: 16px; height: 16px;"></a>&nbsp;
[<a type="button" data-toggle="modal" data-target="#bib-merlet_ISBI2011">bibtex</a>]&nbsp;
[<a type="button" data-toggle="modal" data-target="#abs-merlet_ISBI2011">abstract</a>]&nbsp;
[citations: <a target="_blank" href="https://scholar.google.com//citations?view_op=view_citation&hl=en&oe=ASCII&user=BARqXQ0AAAAJ&pagesize=100&sortby=pubdate&citation_for_view=BARqXQ0AAAAJ:2osOgNQ5qMEC">23</a>]&nbsp;

<div class="modal fade" id="bib-merlet_ISBI2011" role="dialog"><div class="modal-dialog"><div class="modal-content"><div class="modal-header"><button type="button" class="close" data-dismiss="modal">&times;</button><h4 class="modal-title">Bibtex</h4></div><div class="modal-body">
<pre>@inproceedings{merlet_ISBI2011,
 author = {Sylvain Merlet and Jian Cheng and Aurobrata Ghosh and Rachid Deriche},
 booktitle = {IEEE International Symposium on Biomedical Imaging: From Nano to Macro (ISBI'11)},
 doi = {10.1109/ISBI.2011.5872425},
 pages = {365-371},
 pdf = {https://hal.inria.fr/inria-00585694/document},
 title = {Spherical Polar Fourier EAP and ODF Reconstruction via Compressed Sensing in Diffusion MRI},
 url = {https://dx.doi.org/10.1109/ISBI.2011.5872425},
 year = {2011}
}</pre> </div><div class="modal-footer"><button type="button" class="btn btn-default" data-dismiss="modal">Close</button></div></div></div></div>
<div class="modal fade" id="abs-merlet_ISBI2011" role="dialog"><div class="modal-dialog"><div class="modal-content"><div class="modal-header"><button type="button" class="close" data-dismiss="modal">&times;</button><h4 class="modal-title">Abstract</h4></div><div class="modal-body">
<pre>In diffusion magnetic resonance imaging (dMRI), the Ensemble Average
Propagator (EAP), also known as the propagator, describes completely
the water molecule diffusion in the brain white matter without any
prior knowledge about the tissue shape. In this paper, we describe a
new and efficient method to accurately reconstruct the EAP in terms
of the Spherical Polar Fourier (SPF) basis from very few diffusion
weighted magnetic resonance images (DW-MRI). This approach nicely
exploits the duality between SPF and a closely related basis in
which one can respectively represent the EAP and the diffusion
signal using the same coefficients, and efficiently combines it to
the recent acquisition and reconstruction technique called
Compressed Sensing (CS). Our work provides an efficient analytical
solution to estimate, from few measurements, the diffusion
propagator at any radius. We also provide a new analytical solution
to extract an important feature characterising the tissue
microstructure: the Orientation Distribution Function (ODF). We
illustrate and prove the effectiveness of our method in
reconstructing the propagator and the ODF on both noisy multiple
q-shell synthetic and phantom data.</pre> </div><div class="modal-footer"><button type="button" class="btn btn-default" data-dismiss="modal">Close</button></div></div></div></div>
</li><br>

<li>
<span class="author"><b>Jian Cheng</b>, Sylvain Merlet, Emmanuel Caruyer, Aurobrata Ghosh, Tianzi Jiang, Rachid Deriche</span>,<br>
<span class="title">"Compressive Sensing Ensemble Average Propagator Estimation via L1 Spherical Polar Fourier Imaging"</span>,<br>
<span class="publisher">ISMRM'11</span>, May, <span class="year">2011</span>.
<br><a target="_self" href="https://hal.inria.fr/inria-00615434/document"><img src="Icons/pdf.gif" alt="[pdf]" style="width: 16px; height: 16px;"></a>&nbsp;
<a target="_self" href="https://hal.archives-ouvertes.fr/inria-00615434"><img src="Icons/www.gif" alt="[www]" style="width: 16px; height: 16px;"></a>&nbsp;
[<a type="button" data-toggle="modal" data-target="#bib-cheng-ISMRM2011">bibtex</a>]&nbsp;

<div class="modal fade" id="bib-cheng-ISMRM2011" role="dialog"><div class="modal-dialog"><div class="modal-content"><div class="modal-header"><button type="button" class="close" data-dismiss="modal">&times;</button><h4 class="modal-title">Bibtex</h4></div><div class="modal-body">
<pre>@conference{cheng:ISMRM2011,
 address = {Montr&eacute;al, Canada},
 author = {Jian Cheng and Sylvain Merlet and Emmanuel Caruyer and Aurobrata Ghosh and Tianzi Jiang and Rachid Deriche},
 booktitle = {ISMRM'11},
 hal_id = {inria-00615434},
 month = {May},
 pdf = {https://hal.inria.fr/inria-00615434/document},
 title = {Compressive Sensing Ensemble Average Propagator Estimation via L1 Spherical Polar Fourier Imaging},
 url = {https://hal.archives-ouvertes.fr/inria-00615434},
 year = {2011}
}</pre> </div><div class="modal-footer"><button type="button" class="btn btn-default" data-dismiss="modal">Close</button></div></div></div></div>
</li><br>

<li>
<span class="author"><b>Jian Cheng</b>, Aurobrata Ghosh, Tianzi Jiang, Rachid Deriche</span>,<br>
<span class="title">"A Riemannian Framework for Ensemble Average Propagator Computing"</span>,<br>
<span class="publisher">ISMRM'11</span>, <span class="year">2011</span>.
<br><a target="_self" href="https://hal.inria.fr/inria-00615436/document"><img src="Icons/pdf.gif" alt="[pdf]" style="width: 16px; height: 16px;"></a>&nbsp;
<a target="_self" href="https://hal.archives-ouvertes.fr/inria-00615436"><img src="Icons/www.gif" alt="[www]" style="width: 16px; height: 16px;"></a>&nbsp;
[<a type="button" data-toggle="modal" data-target="#bib-cheng-ISMRM2011-EAP">bibtex</a>]&nbsp;

<div class="modal fade" id="bib-cheng-ISMRM2011-EAP" role="dialog"><div class="modal-dialog"><div class="modal-content"><div class="modal-header"><button type="button" class="close" data-dismiss="modal">&times;</button><h4 class="modal-title">Bibtex</h4></div><div class="modal-body">
<pre>@conference{cheng:ISMRM2011:EAP,
 author = {Jian Cheng and Aurobrata Ghosh and Tianzi Jiang and Rachid Deriche},
 booktitle = {ISMRM'11},
 hal_id = {inria-00615436},
 pdf = {https://hal.inria.fr/inria-00615436/document},
 title = {A Riemannian Framework for Ensemble Average Propagator Computing},
 url = {https://hal.archives-ouvertes.fr/inria-00615436},
 year = {2011}
}</pre> </div><div class="modal-footer"><button type="button" class="btn btn-default" data-dismiss="modal">Close</button></div></div></div></div>
</li><br>

</ol><h2><a name="year2010"></a>2010</h2>

<ol reversed>

<li>
<span class="author"><b>Jian Cheng</b>, Aurobrata Ghosh, Rachid Deriche, Tianzi Jiang</span>,<br>
<span class="title">"Model-Free, Regularized, Fast, and Robust Analytical Orientation Distribution Function Estimation"</span>,<br>
<span class="publisher">Medical Image Computing and Computer-Assisted Intervention (<b>MICCAI</b>'10)</span>, vol. 6361, pp. 648&ndash;656, sep, <span class="year">2010</span>.
<br><a target="_self" href="https://hal.inria.fr/inria-00496929/document"><img src="Icons/pdf.gif" alt="[pdf]" style="width: 16px; height: 16px;"></a>&nbsp;
<a target="_self" href="https://dx.doi.org/10.1007/978-3-642-15705-9_79"><img src="Icons/www.gif" alt="[www]" style="width: 16px; height: 16px;"></a>&nbsp;
[<a type="button" data-toggle="modal" data-target="#bib-Cheng_ODF_MICCAI2010">bibtex</a>]&nbsp;
[<a type="button" data-toggle="modal" data-target="#abs-Cheng_ODF_MICCAI2010">abstract</a>]&nbsp;
[<a target="_self" href="https://diffusionmritool.github.io/tutorial_spfi.html">project</a>]&nbsp;
[<a target="_self" href="https://www.dropbox.com/s/z28dckp6gv6lhf2/ODF_MICCAI2010_poster.pdf">poster</a>]&nbsp;
[citations: <a target="_blank" href="https://scholar.google.com//citations?view_op=view_citation&hl=en&oe=ASCII&user=BARqXQ0AAAAJ&pagesize=100&sortby=pubdate&citation_for_view=BARqXQ0AAAAJ:d1gkVwhDpl0C">46</a>]&nbsp;

<div class="modal fade" id="bib-Cheng_ODF_MICCAI2010" role="dialog"><div class="modal-dialog"><div class="modal-content"><div class="modal-header"><button type="button" class="close" data-dismiss="modal">&times;</button><h4 class="modal-title">Bibtex</h4></div><div class="modal-body">
<pre>@inproceedings{Cheng_ODF_MICCAI2010,
 author = {Jian Cheng and Aurobrata Ghosh and Rachid Deriche and Tianzi Jiang},
 booktitle = {Medical Image Computing and Computer-Assisted Intervention (MICCAI'10)},
 doi = {10.1007/978-3-642-15705-9_79},
 hal_id = {inria-00496929},
 month = {sep},
 pages = {648-656},
 pdf = {https://hal.inria.fr/inria-00496929/document},
 title = {Model-Free, Regularized, Fast, and Robust Analytical Orientation Distribution Function Estimation},
 url = {https://dx.doi.org/10.1007/978-3-642-15705-9_79},
 volume = {6361},
 year = {2010}
}</pre> </div><div class="modal-footer"><button type="button" class="btn btn-default" data-dismiss="modal">Close</button></div></div></div></div>
<div class="modal fade" id="abs-Cheng_ODF_MICCAI2010" role="dialog"><div class="modal-dialog"><div class="modal-content"><div class="modal-header"><button type="button" class="close" data-dismiss="modal">&times;</button><h4 class="modal-title">Abstract</h4></div><div class="modal-body">
<pre>High Angular Resolution Imaging (HARDI) can better explore the
complex micro-structure of white matter compared to Diffusion Tensor
Imaging (DTI). Orientation Distribution Function (ODF) in HARDI is
used to describe the probability of the fiber direction. There are
two type definitions of the ODF, which were respectively proposed in
Q-Ball Imaging (QBI) and Diffusion Spectrum Imaging (DSI). Some
analytical reconstructions methods have been proposed to estimate
these two type of ODFs from single shell HARDI data. However they
all have some assumptions and intrinsic modeling errors. In this
article, we propose, almost without any assumption, a uniform
analytical method to estimate these two ODFs from DWI signals in q
space, which is based on Spherical Polar Fourier Expression (SPFE)
of signals. The solution is analytical and is a linear
transformation from the q-space signal to the ODF represented by
Spherical Harmonics (SH). It can naturally combines the DWI signals
in different Q-shells. Moreover It can avoid the intrinsic Funk-
Radon Transform (FRT) blurring error in QBI and it does not need any
assumption of the signals, such as the multiple tensor model and
mono/multi-exponential decay. We validate our method using synthetic
data, phantom data and real data. Our method works well in all
experiments, especially for the data with low SNR, low anisotropy
and non-exponential decay.</pre> </div><div class="modal-footer"><button type="button" class="btn btn-default" data-dismiss="modal">Close</button></div></div></div></div>
</li><br>

<li>
<span class="author"><b>Jian Cheng</b>, Aurobrata Ghosh, Tianzi Jiang, Rachid Deriche</span>,<br>
<span class="title">"Model-free and Analytical EAP Reconstruction via Spherical Polar Fourier Diffusion MRI"</span>,<br>
<span class="publisher">Medical Image Computing and Computer-Assisted Intervention (<b>MICCAI</b>'10)</span>, vol. 6361, pp. 590&ndash;597, sep, <span class="year">2010</span>.
<br><a target="_self" href="https://hal.inria.fr/inria-00496932/document"><img src="Icons/pdf.gif" alt="[pdf]" style="width: 16px; height: 16px;"></a>&nbsp;
<a target="_self" href="https://dx.doi.org/10.1007/978-3-642-15705-9_72"><img src="Icons/www.gif" alt="[www]" style="width: 16px; height: 16px;"></a>&nbsp;
[<a type="button" data-toggle="modal" data-target="#bib-Cheng_PDF_MICCAI2010">bibtex</a>]&nbsp;
[<a type="button" data-toggle="modal" data-target="#abs-Cheng_PDF_MICCAI2010">abstract</a>]&nbsp;
[<a target="_self" href="https://diffusionmritool.github.io/tutorial_spfi.html">project</a>]&nbsp;
[<a target="_self" href="https://www.dropbox.com/s/d9ea28w15hxpcgl/EAP_MICCAI2010_poster.pdf">poster</a>]&nbsp;
[citations: <a target="_blank" href="https://scholar.google.com//citations?view_op=view_citation&hl=en&oe=ASCII&user=BARqXQ0AAAAJ&pagesize=100&sortby=pubdate&citation_for_view=BARqXQ0AAAAJ:u-x6o8ySG0sC">92</a>]&nbsp;

<div class="modal fade" id="bib-Cheng_PDF_MICCAI2010" role="dialog"><div class="modal-dialog"><div class="modal-content"><div class="modal-header"><button type="button" class="close" data-dismiss="modal">&times;</button><h4 class="modal-title">Bibtex</h4></div><div class="modal-body">
<pre>@inproceedings{Cheng_PDF_MICCAI2010,
 author = {Jian Cheng and Aurobrata Ghosh and Tianzi Jiang and Rachid Deriche},
 booktitle = {Medical Image Computing and Computer-Assisted Intervention (MICCAI'10)},
 doi = {10.1007/978-3-642-15705-9_72},
 hal_id = {inria-00496932},
 month = {sep},
 pages = {590-597},
 pdf = {https://hal.inria.fr/inria-00496932/document},
 title = {Model-free and Analytical EAP Reconstruction via Spherical Polar Fourier Diffusion MRI},
 url = {https://dx.doi.org/10.1007/978-3-642-15705-9_72},
 volume = {6361},
 year = {2010}
}</pre> </div><div class="modal-footer"><button type="button" class="btn btn-default" data-dismiss="modal">Close</button></div></div></div></div>
<div class="modal fade" id="abs-Cheng_PDF_MICCAI2010" role="dialog"><div class="modal-dialog"><div class="modal-content"><div class="modal-header"><button type="button" class="close" data-dismiss="modal">&times;</button><h4 class="modal-title">Abstract</h4></div><div class="modal-body">
<pre>How to estimate the diffusion Ensemble Average Propagator (EAP) from
the DWI signals in q-space is an open problem in diffusion MRI
field. Many methods were proposed to estimate the Orientation
Distribution Function (ODF) that is used to describe the fiber
direction. However, ODF is just one of the features of the EAP.
Compared with ODF, EAP has the full information about the diffusion
process which reflects the complex tissue micro-structure. Diffusion
Orientation Transform (DOT) and Diffusion Spectrum Imaging (DSI) are
two important methods to estimate the EAP from the signal. However,
DOT is based on mono-exponential assumption and DSI needs a lot of
samplings and very large b values. In this paper, we propose
Spherical Polar Fourier Imaging (SPFI), a novel model-free fast
robust analytical EAP reconstruction method, which almost does not
need any assumption of data and does not need too many samplings.
SPFI naturally combines the DWI signals with different b-values. It
is an analytical linear transformation from the q-space signal to
the EAP profile represented by Spherical Harmonics (SH). We
validated the proposed methods in synthetic data, phantom data and
real data. It works well in all experiments, especially for the data
with low SNR, low anisotropy, and non-exponential decay.</pre> </div><div class="modal-footer"><button type="button" class="btn btn-default" data-dismiss="modal">Close</button></div></div></div></div>
</li><br>

<li>
<span class="author"><b>Jian Cheng</b>, Aurobrata Ghosh, Tianzi Jiang, Rachid Deriche</span>,<br>
<span class="title">"Fast, Model-Free, Analytical ODF Reconstruction from the Q-Space Signal"</span>,<br>
<span class="publisher">Proceedings of the Sixteenth Annual Meeting of the Organization for Human Brain Mapping (OHBM'10)</span>, <span class="year">2010</span>.
<br><a target="_self" href="https://hal.inria.fr/inria-00497243/document"><img src="Icons/pdf.gif" alt="[pdf]" style="width: 16px; height: 16px;"></a>&nbsp;
<a target="_self" href="https://hal.archives-ouvertes.fr/inria-00497243"><img src="Icons/www.gif" alt="[www]" style="width: 16px; height: 16px;"></a>&nbsp;
[<a type="button" data-toggle="modal" data-target="#bib-Cheng_ODF_OHBM2010">bibtex</a>]&nbsp;

<div class="modal fade" id="bib-Cheng_ODF_OHBM2010" role="dialog"><div class="modal-dialog"><div class="modal-content"><div class="modal-header"><button type="button" class="close" data-dismiss="modal">&times;</button><h4 class="modal-title">Bibtex</h4></div><div class="modal-body">
<pre>@conference{Cheng_ODF_OHBM2010,
 author = {Jian Cheng and Aurobrata Ghosh and Tianzi Jiang and Rachid Deriche},
 booktitle = {Proceedings of the Sixteenth Annual Meeting of the Organization for Human Brain Mapping (OHBM'10)},
 hal_id = {inria-00497243},
 pdf = {https://hal.inria.fr/inria-00497243/document},
 title = {Fast, Model-Free, Analytical ODF Reconstruction from the Q-Space Signal},
 url = {https://hal.archives-ouvertes.fr/inria-00497243},
 year = {2010}
}</pre> </div><div class="modal-footer"><button type="button" class="btn btn-default" data-dismiss="modal">Close</button></div></div></div></div>
</li><br>

<li>
<span class="author"><b>Jian Cheng</b>, Aurobrata Ghosh, Tianzi Jiang, Rachid Deriche</span>,<br>
<span class="title">"Fast, model-free, analytical diffusion PDF profile estimation from the DWI signals"</span>,<br>
<span class="publisher">Proceedings of the Sixteenth Annual Meeting of the Organization for Human Brain Mapping (OHBM'10)</span>, <span class="year">2010</span>.
<br><a target="_self" href="https://hal.inria.fr/inria-00497244/document"><img src="Icons/pdf.gif" alt="[pdf]" style="width: 16px; height: 16px;"></a>&nbsp;
<a target="_self" href="https://hal.archives-ouvertes.fr/inria-00497244"><img src="Icons/www.gif" alt="[www]" style="width: 16px; height: 16px;"></a>&nbsp;
[<a type="button" data-toggle="modal" data-target="#bib-Cheng_PDF_OHBM2010">bibtex</a>]&nbsp;

<div class="modal fade" id="bib-Cheng_PDF_OHBM2010" role="dialog"><div class="modal-dialog"><div class="modal-content"><div class="modal-header"><button type="button" class="close" data-dismiss="modal">&times;</button><h4 class="modal-title">Bibtex</h4></div><div class="modal-body">
<pre>@conference{Cheng_PDF_OHBM2010,
 author = {Jian Cheng and Aurobrata Ghosh and Tianzi Jiang and Rachid Deriche},
 booktitle = {Proceedings of the Sixteenth Annual Meeting of the Organization for Human Brain Mapping (OHBM'10)},
 hal_id = {inria-00497244},
 pdf = {https://hal.inria.fr/inria-00497244/document},
 title = {Fast, model-free, analytical diffusion PDF profile estimation from the DWI signals},
 url = {https://hal.archives-ouvertes.fr/inria-00497244},
 year = {2010}
}</pre> </div><div class="modal-footer"><button type="button" class="btn btn-default" data-dismiss="modal">Close</button></div></div></div></div>
</li><br>

<li>
<span class="author"><b>Jian Cheng</b>, Aurobrata Ghosh, Tianzi Jiang, Rachid Deriche</span>,<br>
<span class="title">"Riemannian Median and Its Applications for Orientation Distribution Function Computing"</span>,<br>
<span class="publisher">ISMRM'10</span>, <span class="year">2010</span>.
<br><a target="_self" href="https://hal.inria.fr/inria-00497246/document"><img src="Icons/pdf.gif" alt="[pdf]" style="width: 16px; height: 16px;"></a>&nbsp;
<a target="_self" href="https://hal.archives-ouvertes.fr/inria-00497246"><img src="Icons/www.gif" alt="[www]" style="width: 16px; height: 16px;"></a>&nbsp;
[<a type="button" data-toggle="modal" data-target="#bib-JianChengISMRM2010">bibtex</a>]&nbsp;

<div class="modal fade" id="bib-JianChengISMRM2010" role="dialog"><div class="modal-dialog"><div class="modal-content"><div class="modal-header"><button type="button" class="close" data-dismiss="modal">&times;</button><h4 class="modal-title">Bibtex</h4></div><div class="modal-body">
<pre>@conference{JianChengISMRM2010,
 author = {Jian Cheng and Aurobrata Ghosh and Tianzi Jiang and Rachid Deriche},
 booktitle = {ISMRM'10},
 hal_id = {inria-00497246},
 pdf = {https://hal.inria.fr/inria-00497246/document},
 title = {Riemannian Median and Its Applications for Orientation Distribution Function Computing},
 url = {https://hal.archives-ouvertes.fr/inria-00497246},
 year = {2010}
}</pre> </div><div class="modal-footer"><button type="button" class="btn btn-default" data-dismiss="modal">Close</button></div></div></div></div>
</li><br>

</ol><h2><a name="year2009"></a>2009</h2>

<ol reversed>

<li>
<span class="author"><b>Jian Cheng</b>, Aurobrata Ghosh, Tianzi Jiang, Rachid Deriche</span>,<br>
<span class="title">"A Riemannian Framework for Orientation Distribution Function Computing"</span>,<br>
<span class="publisher">Medical Image Computing and Computer-Assisted Intervention (<b>MICCAI</b>'09)</span>, vol. 5761, pp. 911&ndash;918, <span class="year">2009</span>.
<br><a target="_self" href="https://hal.inria.fr/inria-00424764/document"><img src="Icons/pdf.gif" alt="[pdf]" style="width: 16px; height: 16px;"></a>&nbsp;
<a target="_self" href="https://dx.doi.org/10.1007/978-3-642-04268-3_112"><img src="Icons/www.gif" alt="[www]" style="width: 16px; height: 16px;"></a>&nbsp;
[<a type="button" data-toggle="modal" data-target="#bib-JianChengMICCAI2009">bibtex</a>]&nbsp;
[<a type="button" data-toggle="modal" data-target="#abs-JianChengMICCAI2009">abstract</a>]&nbsp;
[citations: <a target="_blank" href="https://scholar.google.com//citations?view_op=view_citation&hl=en&oe=ASCII&user=BARqXQ0AAAAJ&pagesize=100&sortby=pubdate&citation_for_view=BARqXQ0AAAAJ:u5HHmVD_uO8C">51</a>]&nbsp;

<div class="modal fade" id="bib-JianChengMICCAI2009" role="dialog"><div class="modal-dialog"><div class="modal-content"><div class="modal-header"><button type="button" class="close" data-dismiss="modal">&times;</button><h4 class="modal-title">Bibtex</h4></div><div class="modal-body">
<pre>@inproceedings{JianChengMICCAI2009,
 author = {Jian Cheng and Aurobrata Ghosh and Tianzi Jiang and Rachid Deriche},
 booktitle = {Medical Image Computing and Computer-Assisted Intervention (MICCAI'09)},
 doi = {10.1007/978-3-642-04268-3_112},
 hal_id = {inria-00424764},
 pages = {911-918},
 pdf = {https://hal.inria.fr/inria-00424764/document},
 title = {A Riemannian Framework for Orientation Distribution Function Computing},
 url = {https://dx.doi.org/10.1007/978-3-642-04268-3_112},
 volume = {5761},
 year = {2009}
}</pre> </div><div class="modal-footer"><button type="button" class="btn btn-default" data-dismiss="modal">Close</button></div></div></div></div>
<div class="modal fade" id="abs-JianChengMICCAI2009" role="dialog"><div class="modal-dialog"><div class="modal-content"><div class="modal-header"><button type="button" class="close" data-dismiss="modal">&times;</button><h4 class="modal-title">Abstract</h4></div><div class="modal-body">
<pre>Compared with Diffusion Tensor Imaging (DTI), High Angular
Resolution Imaging (HARDI) can better explore the complex
microstructure of white matter. Orientation Distribution Function
(ODF) is used to describe the probability of the fiber direction.
Fisher information metric has been constructed for probability
density family in Information Geometry theory and it has been
successfully applied for tensor computing in DTI. In this paper, we
present a state of the art Riemannian framework for ODF computing
based on Information Geometry and sparse representation of
orthonormal bases. In this Riemannian framework, the exponential
map, logarithmic map and geodesic have closed forms. And the
weighted Frechet mean exists uniquely on this manifold. We also
propose a novel scalar measurement, named Geometric Anisotropy (GA),
which is the Riemannian geodesic distance between the ODF and the
isotropic ODF. The Renyi entropy $H_{1/2}$ of the ODF can be
computed from the GA. Moreover, we present an Affine-Euclidean
framework and a Log-Euclidean framework so that we can work in an
Euclidean space. As an application, Lagrange interpolation on ODF
field is proposed based on weighted Frechet mean. We validate our
methods on synthetic and real data experiments. Compared with
existing Riemannian frameworks on ODF, our framework is model-free.
The estimation of the parameters, i.e. Riemannian coordinates, is
robust and linear. Moreover it should be noted that our theoretical
results can be used for any probability density function (PDF) under
an orthonormal basis representation.</pre> </div><div class="modal-footer"><button type="button" class="btn btn-default" data-dismiss="modal">Close</button></div></div></div></div>
</li><br>

<li>
<span class="author"><b>Jian Cheng</b>, Feng Shi, Kun Wang, Ming Song, Jiefeng Jiang, Lijuan Xu, Tianzi Jiang</span>,<br>
<span class="title">"Nonparametric Mean Shift Functional Detection in the Functional Space for Task and Resting-state fMRI"</span>,<br>
<span class="publisher">Workshop on fMRI data analysis: statistical modeling and detection issues in intra- and inter-subject functional MRI data analysis in conjunction with the MICCAI</span>, <span class="year">2009</span>.
<br><a target="_self" href="https://hal.inria.fr/inria-00424765/document"><img src="Icons/pdf.gif" alt="[pdf]" style="width: 16px; height: 16px;"></a>&nbsp;
<a target="_self" href="https://hal.archives-ouvertes.fr/inria-00424765"><img src="Icons/www.gif" alt="[www]" style="width: 16px; height: 16px;"></a>&nbsp;
[<a type="button" data-toggle="modal" data-target="#bib-Cheng_workshopMICCAI2009">bibtex</a>]&nbsp;
[<a type="button" data-toggle="modal" data-target="#abs-Cheng_workshopMICCAI2009">abstract</a>]&nbsp;
[citations: <a target="_blank" href="https://scholar.google.com//citations?view_op=view_citation&hl=en&oe=ASCII&user=BARqXQ0AAAAJ&pagesize=100&sortby=pubdate&citation_for_view=BARqXQ0AAAAJ:eQOLeE2rZwMC">3</a>]&nbsp;
(<span class="hlnote">Oral presentation</span>)&nbsp;

<div class="modal fade" id="bib-Cheng_workshopMICCAI2009" role="dialog"><div class="modal-dialog"><div class="modal-content"><div class="modal-header"><button type="button" class="close" data-dismiss="modal">&times;</button><h4 class="modal-title">Bibtex</h4></div><div class="modal-body">
<pre>@inproceedings{Cheng_workshopMICCAI2009,
 author = {Jian Cheng and Feng Shi and Kun Wang and Ming Song and Jiefeng Jiang and Lijuan Xu and Tianzi Jiang},
 booktitle = {Workshop on fMRI data analysis: statistical modeling and detection issues in intra- and inter-subject functional MRI data analysis in conjunction with the MICCAI},
 hal_id = {inria-00424765},
 pdf = {https://hal.inria.fr/inria-00424765/document},
 title = {Nonparametric Mean Shift Functional Detection in the Functional Space for Task and Resting-state fMRI},
 url = {https://hal.archives-ouvertes.fr/inria-00424765},
 year = {2009}
}</pre> </div><div class="modal-footer"><button type="button" class="btn btn-default" data-dismiss="modal">Close</button></div></div></div></div>
<div class="modal fade" id="abs-Cheng_workshopMICCAI2009" role="dialog"><div class="modal-dialog"><div class="modal-content"><div class="modal-header"><button type="button" class="close" data-dismiss="modal">&times;</button><h4 class="modal-title">Abstract</h4></div><div class="modal-body">
<pre>In functional Magnetic Resonance Imaging (fMRI) data analysis,
normalization of time series is an important and sometimes necessary
preprocessing step in many widely used methods. The space of
normalized time series with n time points is the unit sphere
$S^{n-2}$, named the functional space. Riemannian framework on the
sphere, including the geodesic, the exponential map, and the
logarithmic map, has been well studied in Riemannian geometry. In
this paper, by introducing the Riemannian framework in the
functional space, we propose a novel nonparametric robust method,
namely Mean Shift Functional Detection (MSFD), to explore the
functional space. The first merit of the MSFD is that it does not
need many assumptions on data which are assumed in many existing
method, e.g. linear addition (GLM, PCA, ICA), uncorrelation (PCA),
independence (ICA), the number and the shape of clusters (FCM).
Second, MSFD takes into account the spatial information and can be
seen as a multivariate extension of the functional connectivity
analysis method. It is robust and works well for activation
detection in task study even with a biased activation reference. It
is also able to find the functional networks in resting-state study
without a user-selected "seed" region. Third, it can enhance the
boundary between different functional networks. Experiments were
conducted on synthetic and real data to compare the performance of
the proposed method with GLM and ICA. The experimental results
validated the accuracy and robustness of MSFD, not only for
activation detection in task study but also for functional network
exploration in resting-state study.</pre> </div><div class="modal-footer"><button type="button" class="btn btn-default" data-dismiss="modal">Close</button></div></div></div></div>
</li><br>

</ol><h2><a name="year2008"></a>2008</h2>

<ol reversed>

<li>
<span class="author">Xi Li, Weiming Hu, Zhongfei Zhang, Xiaoqin Zhang, Mingliang Zhu, <b>Jian Cheng</b></span>,<br>
<span class="title">"Visual tracking via incremental log-Euclidean Riemannian subspace learning"</span>,<br>
<span class="publisher">IEEE Conference on Computer Vision and Pattern Recognition (<b>CVPR</b>'08)</span>, <span class="year">2008</span>.
<br><a target="_self" href="http://www.nlpr.ia.ac.cn/2008papers/gjhy/gh9.pdf"><img src="Icons/pdf.gif" alt="[pdf]" style="width: 16px; height: 16px;"></a>&nbsp;
<a target="_self" href="https://dx.doi.org/10.1109/CVPR.2008.4587516"><img src="Icons/www.gif" alt="[www]" style="width: 16px; height: 16px;"></a>&nbsp;
[<a type="button" data-toggle="modal" data-target="#bib-li_cheng_CVPR2008">bibtex</a>]&nbsp;
[<a type="button" data-toggle="modal" data-target="#abs-li_cheng_CVPR2008">abstract</a>]&nbsp;
[citations: <a target="_blank" href="https://scholar.google.com//citations?view_op=view_citation&hl=en&oe=ASCII&user=BARqXQ0AAAAJ&pagesize=100&sortby=pubdate&citation_for_view=BARqXQ0AAAAJ:WF5omc3nYNoC">174</a>]&nbsp;

<div class="modal fade" id="bib-li_cheng_CVPR2008" role="dialog"><div class="modal-dialog"><div class="modal-content"><div class="modal-header"><button type="button" class="close" data-dismiss="modal">&times;</button><h4 class="modal-title">Bibtex</h4></div><div class="modal-body">
<pre>@inproceedings{li_cheng_CVPR2008,
 author = {Xi Li and Weiming Hu and Zhongfei Zhang and Xiaoqin Zhang and Mingliang Zhu and Jian Cheng},
 booktitle = {IEEE Conference on Computer Vision and Pattern Recognition (CVPR'08)},
 doi = {10.1109/CVPR.2008.4587516},
 organization = {IEEE},
 pdf = {http://www.nlpr.ia.ac.cn/2008papers/gjhy/gh9.pdf},
 title = {Visual tracking via incremental log-Euclidean Riemannian subspace learning},
 url = {https://dx.doi.org/10.1109/CVPR.2008.4587516},
 year = {2008}
}</pre> </div><div class="modal-footer"><button type="button" class="btn btn-default" data-dismiss="modal">Close</button></div></div></div></div>
<div class="modal fade" id="abs-li_cheng_CVPR2008" role="dialog"><div class="modal-dialog"><div class="modal-content"><div class="modal-header"><button type="button" class="close" data-dismiss="modal">&times;</button><h4 class="modal-title">Abstract</h4></div><div class="modal-body">
<pre>Recently, a novel Log-Euclidean Riemannian metric [28] is proposed
for statistics on symmetric positive definite (SPD) matrices. Under
this metric, distances and Riemannian means take a much simpler form
than the widely used affine-invariant Riemannian metric. Based on
the Log-Euclidean Riemannian metric, we develop a tracking framework
in this paper. In the framework, the covariance matrices of image
features in the five modes are used to represent object appearance.
Since a nonsingular covariance matrix is a SPD matrix lying on a
connected Riemannian manifold, the Log-Euclidean Riemannian metric
is used for statistics on the covariance matrices of image features.
Further, we present an effective online Log-Euclidean Riemannian
subspace learning algorithm which models the appearance changes of
an object by incrementally learning a low-order Log-Euclidean
eigenspace representation through adaptively updating the sample
mean and eigenbasis. Tracking is then led by the Bayesian state
inference framework in which a particle filter is used for
propagating sample distributions over the time. Theoretic analysis
and experimental evaluations demonstrate the promise and
effectiveness of the proposed framework.</pre> </div><div class="modal-footer"><button type="button" class="btn btn-default" data-dismiss="modal">Close</button></div></div></div></div>
</li><br>

</ol>



        <br></section>


           </div>
           <div class="articleComments">
            
           </div>
          </div>
          <footer>
  
    <div class="rst-footer-buttons" role="navigation" aria-label="footer navigation">
      
        <a href="publications_by_venue.html" class="btn btn-neutral float-right" title="Publications By Selected Venues" accesskey="n">Next <span class="fa fa-arrow-circle-right"></span></a>
      
      
        <a href="publications_by_type.html" class="btn btn-neutral" title="Publications By Type" accesskey="p"><span class="fa fa-arrow-circle-left"></span> Previous</a>
      
    </div>
  

  <hr/>

  <div role="contentinfo">
    <p>
        &copy; Copyright <a class="reference external" href="mailto:jiancheng@ieee.org">Jian Cheng</a>.
  <!-- Hosted by <a href="https://pages.coding.me">Coding Pages</a>. -->
   <!-- <a class="reference external" href="https://pages.coding.me"> -->
   <!--     <img alt="" src="https://dn-coding-net-production-static.qbox.me/static/9348be98ed46c1cd972743184b76fe73.png" style="width: 80px; height: 16px;"/>  -->
   <!-- </a> -->
    

    </p>
  </div> 

</footer>

        </div>
      </div>

    </section>

  </div>
  


  

    <script type="text/javascript">
        var DOCUMENTATION_OPTIONS = {
            URL_ROOT:'./',
            VERSION:'',
            COLLAPSE_INDEX:false,
            FILE_SUFFIX:'.html',
            HAS_SOURCE:  true,
            SOURCELINK_SUFFIX: '.txt'
        };
    </script>
      <script type="text/javascript" src="_static/documentation_options.js"></script>
      <script type="text/javascript" src="_static/jquery.js"></script>
      <script type="text/javascript" src="_static/underscore.js"></script>
      <script type="text/javascript" src="_static/doctools.js"></script>

  

  
  
    <script type="text/javascript" src="_static/js/theme.js"></script>
  

  
  
  <script type="text/javascript">
      jQuery(function () {
          SphinxRtdTheme.StickyNav.enable();
      });
  </script>
  
 
<script>
  (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
  (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
  m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
  })(window,document,'script','//www.google-analytics.com/analytics.js','ga');

  ga('create', 'UA-60718632-1', 'auto');
  ga('send', 'pageview');

</script>

<!-- Start of StatCounter Code for Default Guide -->
<script type="text/javascript">
var sc_project=11048629; 
var sc_invisible=0; 
var sc_security="d518e60a"; 
var scJsHost = (("https:" == document.location.protocol) ?
"https://secure." : "http://www.");
document.write("<sc"+"ript type='text/javascript' src='" +
scJsHost+
"statcounter.com/counter/counter.js'></"+"script>");
</script>
<noscript><div class="statcounter"><a title="shopify
analytics tool" href="https://statcounter.com/shopify/"
target="_blank"><img class="statcounter"
src="//c.statcounter.com/11048629/0/d518e60a/0/"
alt="shopify analytics tool"></a></div></noscript>
<!-- End of StatCounter Code for Default Guide -->



</body>
</html>